# Molecular Basis of Inheritance - Assorted Question Paper

## Section A: Multiple Choice Questions (100 × 1 = 100 Marks)

1. In Griffith’s experiment, which combination led to death in mice?  
    - (a) Live R strain  
    - (b) Live S strain  
    - (c) Heat-killed S strain  
    - (d) Heat-killed S strain + Live R strain  

2. The definitive proof that DNA is the genetic material was provided by:  
    - (a) Hershey and Chase  
    - (b) Watson and Crick  
    - (c) Avery, MacLeod, McCarty  
    - (d) Franklin and Wilkins  

3. Which base pairs with guanine in DNA?  
    - (a) Uracil  
    - (b) Thymine  
    - (c) Cytosine  
    - (d) Adenine  

4. What is the pitch of the DNA helix?  
    - (a) 0.34 nm  
    - (b) 3.4 nm  
    - (c) 10 nm  
    - (d) 5 nm  

5. Which RNA has an anticodon loop?  
    - (a) mRNA  
    - (b) rRNA  
    - (c) tRNA  
    - (d) snRNA  

6. The start codon in mRNA is:  
    - (a) UAA  
    - (b) AUG  
    - (c) UAG  
    - (d) UGA  

7. Which component of the nucleosome is not a core histone?  
    - (a) H1  
    - (b) H2A  
    - (c) H3  
    - (d) H4  

8. Which enzyme joins Okazaki fragments?  
    - (a) Helicase  
    - (b) Ligase  
    - (c) Polymerase  
    - (d) Topoisomerase  

9. Reverse transcription is found in:  
    - (a) All bacteria  
    - (b) Retroviruses  
    - (c) Archaea  
    - (d) Fungi  

10. The lac operon is switched ON when:  
    - (a) Lactose is absent  
    - (b) Glucose is present  
    - (c) Lactose is present  
    - (d) None of the above  

11. Who proposed the central dogma of molecular biology?  
    - (a) Watson  
    - (b) Crick  
    - (c) Chargaff  
    - (d) Meselson  

12. Which RNA carries genetic code from nucleus to ribosome?  
    - (a) rRNA  
    - (b) tRNA  
    - (c) mRNA  
    - (d) snRNA  

13. Which enzyme relieves supercoiling during replication?  
    - (a) Helicase  
    - (b) Ligase  
    - (c) Topoisomerase  
    - (d) Polymerase  

14. Which base is replaced by uracil in RNA?  
    - (a) Adenine  
    - (b) Guanine  
    - (c) Thymine  
    - (d) Cytosine  

15. Which bond holds nitrogen bases in DNA?  
    - (a) Phosphodiester bond  
    - (b) Ionic bond  
    - (c) Hydrogen bond  
    - (d) Peptide bond  

16. Which RNA is involved in protein synthesis site structure?  
    - (a) mRNA  
    - (b) tRNA  
    - (c) rRNA  
    - (d) snRNA  

17. Which histone binds at entry/exit of DNA in nucleosome?  
    - (a) H1  
    - (b) H2A  
    - (c) H2B  
    - (d) H4  

18. How many base pairs are there per turn in DNA?  
    - (a) 8  
    - (b) 10  
    - (c) 12  
    - (d) 15  

19. What is the distance between two consecutive base pairs?  
    - (a) 3.4 nm  
    - (b) 0.34 nm  
    - (c) 2.0 nm  
    - (d) 1.5 nm  

20. What is the function of DNA polymerase?  
    - (a) Joining DNA fragments  
    - (b) Unwinding DNA  
    - (c) Synthesizing DNA  
    - (d) Breaking hydrogen bonds  

21. Which enzyme initiates synthesis of RNA primer?  
    - (a) Ligase  
    - (b) Primase  
    - (c) Helicase  
    - (d) Polymerase  

22. Which RNA is first formed in eukaryotic transcription?  
    - (a) mRNA  
    - (b) hnRNA  
    - (c) tRNA  
    - (d) snRNA  

23. What is the role of snRNA?  
    - (a) Codes for protein  
    - (b) Forms ribosomes  
    - (c) Involved in splicing  
    - (d) Carries amino acids  

24. In which organelle does transcription occur in eukaryotes?  
    - (a) Cytoplasm  
    - (b) Ribosome  
    - (c) Nucleus  
    - (d) Mitochondria  

25. What is the function of the lacZ gene?  
    - (a) Codes for permease  
    - (b) Codes for β-galactosidase  
    - (c) Codes for repressor  
    - (d) Codes for transacetylase  

26. What is the full form of rRNA?  
    - (a) Resistant RNA  
    - (b) Ribosomal RNA  
    - (c) Repeated RNA  
    - (d) Regulatory RNA  

27. Which scientist discovered that A = T and G = C?  
    - (a) Franklin  
    - (b) Chargaff  
    - (c) Crick  
    - (d) Wilkins  

28. What is the nature of DNA strands in replication?  
    - (a) Conservative  
    - (b) Semi-conservative  
    - (c) Dispersive  
    - (d) Random  

29. DNA polymerase works in which direction?  
    - (a) 3' → 5'  
    - (b) 5' → 3'  
    - (c) Both directions  
    - (d) Random  

30. Which virus was used by Hershey and Chase?  
    - (a) Adenovirus  
    - (b) Bacteriophage  
    - (c) Retrovirus  
    - (d) Influenza  

31. Who proposed the central dogma?
    a) Watson  b) Crick  c) Meselson  d) Avery

32. The flow of genetic information according to central dogma is:
    a) RNA→DNA→Protein  b) DNA→Protein→RNA  c) DNA→RNA→Protein  d) Protein→RNA→DNA

33. Reverse transcription is catalyzed by:
    a) DNA polymerase  b) RNA polymerase  c) Reverse transcriptase  d) Ligase

34. Retroviruses follow which pattern of genetic information flow?
    a) DNA→RNA→Protein  b) RNA→DNA→RNA→Protein  c) DNA→Protein→RNA  d) RNA→Protein→DNA

35. Semi-conservative replication was experimentally proven by:
    a) Watson and Crick  b) Meselson and Stahl  c) Hershey and Chase  d) Avery

36. In semi-conservative replication, each new DNA molecule contains:
    a) Two new strands  b) Two old strands  c) One old and one new strand  d) Mixed old and new segments

37. Meselson and Stahl used which isotopes in their experiment?
    a) ¹⁴C and ¹²C  b) ¹⁵N and ¹⁴N  c) ³²P and ³¹P  d) ³⁵S and ³²S

38. DNA helicase is responsible for:
    a) Joining DNA fragments  b) Unwinding DNA  c) Synthesizing DNA  d) Proofreading DNA

39. DNA polymerase works in which direction?
    a) 3'→5'  b) 5'→3'  c) Both directions  d) Random direction

40. Okazaki fragments are joined by:
    a) DNA polymerase  b) DNA helicase  c) DNA ligase  d) Primase

41. RNA primers are synthesized by:
    a) DNA polymerase  b) RNA polymerase  c) Primase  d) Ligase

42. Topoisomerase relieves:
    a) Tension in DNA  b) Supercoiling in DNA  c) Breaks in DNA  d) Mutations in DNA

43. In prokaryotes, transcription occurs in:
    a) Nucleus  b) Cytoplasm  c) Mitochondria  d) Ribosomes

44. RNA polymerase I transcribes:
    a) mRNA  b) tRNA  c) rRNA  d) hnRNA

45. RNA polymerase II transcribes:
    a) rRNA  b) tRNA  c) mRNA precursors  d) snRNA

46. The primary transcript in eukaryotes is:
    a) mRNA  b) tRNA  c) rRNA  d) hnRNA

47. Splicing is carried out by:
    a) Ribosomes  b) Spliceosomes  c) Nucleosomes  d) Lysosomes

48. Capping occurs at which end of hnRNA?
    a) 3' end  b) 5' end  c) Both ends  d) Middle

49. Poly-A tail is added to:
    a) 3' end  b) 5' end  c) Both ends  d) Middle

50. Introns are:
    a) Coding sequences  b) Non-coding sequences  c) Regulatory sequences  d) Structural sequences

51. Exons are:
    a) Non-coding sequences  b) Coding sequences  c) Regulatory sequences  d) Structural sequences

52. A codon consists of:
    a) 2 nucleotides  b) 3 nucleotides  c) 4 nucleotides  d) 5 nucleotides

53. The genetic code is:
    a) Overlapping  b) Non-overlapping  c) Punctuated  d) Discontinuous

54. The start codon is:
    a) UAA  b) UAG  c) UGA  d) AUG

55. How many stop codons are there?
    a) 1  b) 2  c) 3  d) 4

56. The genetic code is described as:
    a) Specific  b) Universal  c) Degenerate  d) All of the above

57. UAA is a:
    a) Start codon  b) Stop codon  c) Sense codon  d) Regulatory codon

58. The initiator tRNA in prokaryotes carries:
    a) Methionine  b) Formylmethionine  c) Alanine  d) Glycine

59. The A-site in ribosome is for:
    a) Peptidyl tRNA  b) Aminoacyl tRNA  c) Exit of tRNA  d) mRNA binding

60. Peptide bond formation occurs in:
    a) A-site  b) P-site  c) E-site  d) Between A and P sites

61. The activity of peptide bond formation is called:
    a) Transaminase  b) Peptidyl transferase  c) Aminoacyl synthetase  d) Ligase

62. Release factors bind to:
    a) Start codon  b) Stop codon  c) Sense codon  d) Regulatory sequences

63. Lac operon is an example of:
    a) Repressible operon  b) Inducible operon  c) Constitutive operon  d) Negative operon

64. The inducer of lac operon is:
    a) Glucose  b) Lactose  c) Galactose  d) Fructose

65. β-galactosidase is coded by:
    a) lacZ  b) lacY  c) lacA  d) lacI

66. Permease is coded by:
    a) lacZ  b) lacY  c) lacA  d) lacI

67. The repressor protein of lac operon is coded by:
    a) lacZ  b) lacY  c) lacA  d) lacI

68. When lactose is absent, the lac operon is:
    a) ON  b) OFF  c) Partially active  d) Unaffected

69. The Human Genome Project aimed to sequence:
    a) 1 billion base pairs  b) 2 billion base pairs  c) 3 billion base pairs  d) 4 billion base pairs

70. The number of genes in human genome is approximately:
    a) 10,000-15,000  b) 20,000-25,000  c) 30,000-35,000  d) 40,000-45,000

71. EST stands for:
    a) Expressed Sequence Tags  b) Essential Sequence Tags  c) Estimated Sequence Tags  d) Extended Sequence Tags

72. DNA fingerprinting is based on:
    a) SNPs  b) VNTRs  c) STRs  d) All of the above

73. VNTR stands for:
    a) Variable Number Tandem Repeats  b) Variable Nucleotide Tandem Repeats  c) Variant Number Tandem Repeats  d) Verified Number Tandem Repeats

74. Southern blotting is used for:
    a) DNA analysis  b) RNA analysis  c) Protein analysis  d) Lipid analysis

75. Gel electrophoresis separates DNA fragments based on:
    a) Size  b) Charge  c) Shape  d) All of the above

76. DNA fingerprinting was first developed by:
    a) Watson  b) Crick  c) Alec Jeffreys  d) Kary Mullis

77. PCR stands for:
    a) Polymerase Chain Reaction  b) Protein Chain Reaction  c) Peptide Chain Reaction  d) Phosphate Chain Reaction

78. Taq polymerase is used in:
    a) DNA fingerprinting  b) PCR  c) Gel electrophoresis  d) Southern blotting

79. The melting temperature of DNA depends on:
    a) A-T content  b) G-C content  c) Length  d) All of the above

80. Hyperchromicity refers to:
    a) Increase in DNA absorption  b) Decrease in DNA absorption  c) No change in absorption  d) Random absorption

81. Tm (melting temperature) is the temperature at which:
    a) 25% DNA is denatured  b) 50% DNA is denatured  c) 75% DNA is denatured  d) 100% DNA is denatured

82. C0t½ value is related to:
    a) DNA renaturation  b) DNA denaturation  c) DNA replication  d) DNA transcription

83. The genetic code was deciphered by:
    a) Nirenberg and Matthaei  b) Khorana  c) Crick  d) All of the above

84. Wobble base pairing occurs at:
    a) First position  b) Second position  c) Third position  d) All positions

85. Nonsense mutations result in:
    a) Missense codons  b) Stop codons  c) Start codons  d) Regulatory codons

86. Frameshift mutations are caused by:
    a) Substitution  b) Insertion  c) Deletion  d) Both b and c

87. Shine-Dalgarno sequence is found in:
    a) Eukaryotes  b) Prokaryotes  c) Both  d) Neither

88. Kozak sequence is found in:
    a) Prokaryotes  b) Eukaryotes  c) Both  d) Neither

89. The codon AUG codes for:
    a) Methionine  b) Valine  c) Leucine  d) Isoleucine

90. Aminoacyl tRNA synthetase catalyzes:
    a) Peptide bond formation  b) tRNA charging  c) Translation initiation  d) mRNA processing

91. Polyribosomes are:
    a) Multiple ribosomes on single mRNA  b) Multiple mRNAs on single ribosome  c) Multiple tRNAs  d) Multiple amino acids

92. Rho protein is involved in:
    a) Transcription initiation  b) Transcription termination  c) Translation initiation  d) Translation termination

93. Riboswitches are:
    a) DNA regulatory elements  b) RNA regulatory elements  c) Protein regulatory elements  d) Lipid regulatory elements

94. The promoter sequence in prokaryotes is:
    a) TATA box  b) Pribnow box  c) CAAT box  d) GC box

95. Enhancers are:
    a) Negative regulatory elements  b) Positive regulatory elements  c) Neutral elements  d) Structural elements

96. Catabolite repression is mediated by:
    a) CAP-cAMP  b) Lac repressor  c) Trp repressor  d) Ara operon

97. Attenuation is a mechanism of control in:
    a) Lac operon  b) Trp operon  c) Ara operon  d) Gal operon

98. Alternative splicing results in:
     a) Different mRNAs from same gene  b) Same mRNA from different genes  c) No mRNA  d) Multiple genes

99. Griffith's transforming principle experiment was conducted in which year?
   a) 1925  b) 1928  c) 1930  d) 1935

100. Which bacterial strains were used in Griffith's experiment?
   a) R and S strains  b) A and B strains  c) X and Y strains  d) P and Q strains

## Section B: Very Short Answer Questions (100 × 1 = 100 Marks)

1. What was the conclusion of Griffith’s experiment?
2. Name the bacteria used by Griffith.
3. What does DNase do in transformation experiments?
4. Name the radioactive isotope used to label DNA in Hershey-Chase experiment.
5. Which molecule did Hershey-Chase conclude to be the genetic material?
6. Who proposed the double helix model of DNA?
7. What is Chargaff’s rule?
8. Which base pairs with adenine in DNA?
9. Which sugar is present in RNA?
10. Which type of RNA has an anticodon?
11. Name the enzyme that unwinds DNA.
12. What is the function of DNA ligase?
13. Which enzyme synthesizes new DNA strands?
14. What is the direction of DNA synthesis?
15. Which enzyme synthesizes RNA primers?
16. Which enzyme removes supercoiling during replication?
17. What is meant by semi-conservative replication?
18. What is the pitch of the DNA double helix?
19. How many base pairs are present per turn of DNA?
20. What is the distance between two base pairs in DNA?
21. Name the scientist who demonstrated semi-conservative replication.
22. Name one core histone protein in a nucleosome.
23. What is the role of histone H1?
24. What is chromatin?
25. Which nitrogen base is absent in RNA?
26. Name the enzyme responsible for transcription.
27. Which RNA polymerase transcribes mRNA in eukaryotes?
28. What is hnRNA?
29. What is splicing?
30. What is 5’ capping?
31. What is polyadenylation?
32. What is the full form of mRNA?
33. Which RNA is most abundant in the cell?
34. What is a codon?
35. Name the start codon.
36. Write any one stop codon.
37. What is the role of tRNA?
38. What is the shape of tRNA?
39. What is meant by degenerate code?
40. Which base is unique to RNA?
41. What is meant by non-overlapping genetic code?
42. What is the function of ribosomes?
43. What is translocation in translation?
44. What is the A-site of a ribosome?
45. What is the function of peptidyl transferase?
46. What does the lacZ gene code for?
47. What is the inducer molecule in lac operon?
48. Name the components of a bacterial operon.
49. What is the role of repressor protein in lac operon?
50. Which organism was used to study the lac operon?
51. Define genetic material.
52. What are nucleotides?
53. What is a purine?
54. What is a pyrimidine?
55. What bond joins nucleotides in a strand?
56. Name the sugar found in DNA.
57. Define the central dogma.
58. Name a retrovirus.
59. Name the enzyme responsible for reverse transcription.
60. Which technique separates DNA fragments by size?
61. What are VNTRs?
62. What is DNA fingerprinting?
63. What are introns?
64. What are exons?
65. What is the function of snRNPs?
66. What is the purpose of post-transcriptional modification?
67. What is a polynucleotide?
68. What is meant by antiparallel strands?
69. What is euchromatin?
70. What is heterochromatin?
71. What is the full form of HGP?
72. How many base pairs are there in the human genome?
73. What is pharmacogenomics?
74. What is sequence annotation?
75. What are expressed sequence tags (ESTs)?
76. What is the application of gene therapy?
77. Name one disease where gene therapy is used.
78. What is the principle of DNA fingerprinting?
79. Name any one ethical concern of DNA profiling.
80. What is meant by molecular biology?
81. Name the scientist who coined “central dogma.”
82. What is the function of permease?
83. Name the three structural genes of the lac operon.
84. What is the function of β-galactosidase?
85. What is the use of transacetylase?
86. Define replication.
87. Name one organism with circular DNA.
88. Which RNA is involved in ribosome formation?
89. What does AUG code for?
90. What are Okazaki fragments?
91. What is the template strand?
92. What is a coding strand?
93. Which RNA polymerase synthesizes tRNA?
94. What is gene expression?
95. Define transcription.
96. Define translation.
97. Name the scientist who used radioactive isotopes in viral infection.
98. What is meant by a triplet codon?
99. Define genome.
100. Define gene.

## Section C: Short Answer Questions (100 × 2 = 200 Marks)

1. Compare the experiments of Griffith and Avery with respect to their conclusions.
2. Explain the role of DNase in identifying DNA as genetic material.
3. Write any two differences between DNA and RNA.
4. How did Hershey and Chase confirm that DNA is the genetic material?
5. State two features of the Watson and Crick model of DNA.
6. Explain Chargaff’s rule with an example.
7. Why is DNA more stable than RNA? Give two reasons.
8. Mention two structural differences between purines and pyrimidines.
9. What are histones? What role do they play in packaging DNA?
10. Distinguish between euchromatin and heterochromatin.
11. Why is the genetic code said to be degenerate and universal?
12. Write the role of mRNA and tRNA in protein synthesis.
13. Describe any two features of the genetic code.
14. Mention any two functions of ribosomal RNA.
15. Why is the genetic code described as ‘non-overlapping’ and ‘comma-less’?
16. Distinguish between codon and anticodon.
17. Mention two types of post-transcriptional modifications in eukaryotes.
18. What are introns and exons? State one difference.
19. Write the function of RNA polymerase I and RNA polymerase II.
20. What is the function of spliceosomes? What are they composed of?
21. Define nucleosome. What is the role of H1 histone?
22. Describe the structure of tRNA in two points.
23. Explain the polarity of DNA strands.
24. Write any two enzymes involved in DNA replication and their roles.
25. Compare continuous and discontinuous DNA synthesis.
26. Why is DNA replication considered semi-conservative?
27. Differentiate between leading and lagging strands.
28. What is meant by Okazaki fragments? Where are they found?
29. What is a primer in DNA replication? Who synthesizes it?
30. State the roles of helicase and topoisomerase in DNA replication.
31. Explain the role of promoter and terminator sequences in transcription.
32. How is transcription in eukaryotes different from prokaryotes? (any two points)
33. What is meant by template strand and coding strand in transcription?
34. Explain briefly the process of capping and tailing.
35. Mention two differences between mRNA and hnRNA.
36. What are the three major types of RNA and their functions?
37. How does the tRNA molecule act as an adaptor?
38. Describe the cloverleaf structure of tRNA.
39. What is the significance of AUG codon in protein synthesis?
40. Write any two functions of the small ribosomal subunit.
41. What are the A, P, and E sites of a ribosome?
42. Explain the elongation stage of translation in two steps.
43. How is the translation process terminated?
44. Describe the function of peptidyl transferase during translation.
45. What is a polysome? State its significance.
46. Write the names and functions of the three structural genes in the lac operon.
47. How does the repressor protein regulate the lac operon?
48. What is an operon? Give an example.
49. Explain the function of the inducer in lac operon regulation.
50. Describe any two applications of the lac operon model.
51. What is gene regulation? Why is it important?
52. Describe the role of allolactose in the lac operon.
53. Write a brief note on inducible operon with example.
54. Explain the term “reverse transcription.”
55. Name two organisms in which reverse transcription is observed.
56. Define central dogma. Explain with a diagram.
57. Mention two applications of recombinant DNA technology.
58. What is DNA fingerprinting? Mention one application.
59. What are VNTRs? Why are they important in forensic science?
60. Explain the significance of HGP in human medicine.
61. Write any two goals of the Human Genome Project.
62. What are expressed sequence tags? Mention one use.
63. Define sequence annotation and mention its role in genomics.
64. What are the ethical concerns associated with DNA fingerprinting?
65. How are DNA fragments separated by gel electrophoresis?
66. Differentiate between coding and non-coding DNA.
67. Why is DNA considered a better genetic material than RNA?
68. How does the presence of a 2’-OH group make RNA less stable?
69. Write any two characteristics of genetic code.
70. How are histone octamers formed?
71. Explain the structure of chromatin with the beads-on-string model.
72. What are tandem repeats? Where are they found?
73. Mention two medical applications of Human Genome Project.
74. Define gene therapy. Mention one disease treated using this method.
75. Write a short note on pharmacogenomics.
76. Differentiate between template strand and coding strand.
77. What is an exonuclease? How is it useful?
78. Define mutation. Mention one cause.
79. What is wobble hypothesis?
80. What is a pseudogene?
81. How do prokaryotes couple transcription and translation?
82. What is the function of the operator gene?
83. Differentiate between regulatory and structural genes.
84. Describe the importance of DNA packaging.
85. Write any two differences between snRNA and hnRNA.
86. Why is DNA negatively charged?
87. What is meant by gene expression?
88. What is a repressible operon? Give an example.
89. What is the function of a repressor molecule?
90. Define upstream and downstream in genetic context.
91. Differentiate between DNA replication and transcription.
92. Write two differences between translation in prokaryotes and eukaryotes.
93. What is the function of the terminator sequence?
94. Mention any two tools required for gene cloning.
95. Define recombinant DNA.
96. State any two ethical issues in genome sequencing.
97. What is chromosomal DNA?
98. Mention two uses of DNA polymerase.
99. How is the double helix structure stabilized?
100. Why is the G≡C pair stronger than the A=T pair?

## Section D: Broad Answer Questions (100 × 3 = 300 Marks)

1. Describe the Hershey and Chase experiment and its conclusion.
2. Explain Avery, MacLeod, and McCarty’s experiment that proved DNA is the genetic material.
3. Compare the structures of DNA and RNA with respect to three major features.
4. Describe the Watson and Crick double helix model of DNA.
5. Write the role and significance of Chargaff’s rules in DNA structure.
6. Explain the semi-conservative nature of DNA replication with Meselson and Stahl’s experiment.
7. Describe the roles of DNA helicase, ligase, and polymerase in replication.
8. Differentiate between leading and lagging strand with a diagram.
9. Explain the mechanism of DNA replication in eukaryotes.
10. Write the structure and function of a nucleosome.
11. Compare euchromatin and heterochromatin with examples.
12. Describe the different types of RNA and their functions.
13. Describe the detailed structure of tRNA and its role in protein synthesis.
14. Explain the process of transcription in eukaryotes.
15. Describe post-transcriptional modifications of hnRNA.
16. Write a note on the three RNA polymerases in eukaryotes and their functions.
17. Explain the initiation, elongation, and termination stages of translation.
18. Describe the genetic code and its main characteristics with examples.
19. Discuss the significance of degenerate, universal, and non-overlapping nature of the genetic code.
20. Explain the lac operon model in detail.
21. Differentiate between inducible and repressible operons.
22. Write the function of all structural genes of lac operon.
23. Describe gene regulation in prokaryotes using an operon system.
24. How does allolactose regulate gene expression in lac operon?
25. Describe the structure of chromatin and the role of histones.
26. Explain the steps of post-transcriptional processing in mRNA formation.
27. Explain in detail the role of ribosomes in translation.
28. Write a comparative note on transcription in prokaryotes and eukaryotes.
29. Explain the central dogma of molecular biology with diagram.
30. Describe reverse transcription with a suitable example.
31. Describe the process of protein synthesis from transcription to translation.
32. What are Okazaki fragments? Explain their formation and significance.
33. Explain the concept of anticodon and its role in translation.
34. What are codons? Mention types and their role in translation.
35. Describe the structure of a ribosome and explain its role in translation.
36. Describe the function and importance of peptidyl transferase activity in ribosomes.
37. Explain the function of promoter, operator, and structural genes in an operon.
38. Discuss the steps involved in translation termination.
39. Describe DNA packaging in eukaryotic cells with a labeled diagram.
40. Explain the concept of chromatin remodeling and its biological significance.
41. Discuss the methods and ethical issues in DNA fingerprinting.
42. Write a short note on the Human Genome Project—goals, methods, and benefits.
43. What are the applications of HGP in medicine and forensics?
44. Explain the structure of DNA with reference to hydrogen bonding and base pairing.
45. Write a note on transcription factors and enhancers.
46. What is gene expression? Describe the regulation in eukaryotic cells.
47. Differentiate between structural and regulatory genes.
48. Explain the importance of histone modifications in gene regulation.
49. Describe the three sites of a ribosome and their roles during protein synthesis.
50. Write a note on the wobble hypothesis and its role in codon recognition.
51. Explain how gene expression is controlled at the transcriptional level.
52. Describe the experimental setup and results of Griffith’s experiment.
53. Compare the roles of helicase, topoisomerase, and ligase in DNA replication.
54. Describe how the template strand guides mRNA synthesis.
55. Explain how RNA processing increases the efficiency of gene expression.
56. What are VNTRs and how are they used in forensic science?
57. Describe the principle and procedure of gel electrophoresis.
58. Explain how DNA fingerprinting is used to determine paternity.
59. What is gene therapy? Explain with examples.
60. Write a note on the role of bioinformatics in genome annotation.
61. Explain the difference between exons and introns and their fate during RNA splicing.
62. Describe any three major tools required in recombinant DNA technology.
63. Explain the steps of Southern blotting.
64. Discuss the role of small nuclear RNA in mRNA processing.
65. What are the molecular differences between DNA replication and transcription?
66. How does RNA polymerase recognize the promoter and initiate transcription?
67. Write a note on the concept and functions of a gene.
68. Describe three types of point mutations and their effects.
69. Explain the concept of genetic code degeneracy and silent mutations.
70. Compare and contrast DNA and chromosomal mutations.
71. Write a note on regulatory sequences in prokaryotic genomes.
72. Describe how DNA methylation affects gene expression.
73. Explain the molecular mechanism of antisense RNA technology.
74. Discuss the importance of coding vs. non-coding DNA.
75. Describe three common applications of genetic engineering.
76. Write about any three medical applications of recombinant DNA technology.
77. Explain the role of ESTs and sequence annotation in HGP.
78. Discuss the ethical, legal, and social issues of genetic information.
79. Explain how transcription and translation are coupled in prokaryotes.
80. Describe the experimental method used to prove that DNA replicates semi-conservatively.
81. What is the molecular structure of a nucleosome? Draw and label.
82. How do ribosomes facilitate polypeptide elongation?
83. Describe three differences between prokaryotic and eukaryotic translation.
84. Explain how alternative splicing leads to protein diversity.
85. Describe the molecular mechanism of action of restriction enzymes.
86. Explain how genes are mapped using sequencing techniques.
87. Describe any three key bioinformatics databases used in genomics.
88. Compare the characteristics of mitochondrial and nuclear DNA.
89. Explain how transcription is terminated in prokaryotes.
90. Describe the method of creating a cDNA library.
91. Explain the concept of split genes and its discovery.
92. Describe how polymerase chain reaction (PCR) works.
93. Discuss the applications and importance of PCR in biology.
94. Describe the entire mechanism of lac operon regulation with diagrams.
95. How are DNA-binding motifs important for gene regulation?
96. What are pseudogenes? How do they arise?
97. Explain the differences in DNA packaging between prokaryotes and eukaryotes.
98. Write a short note on the ribozyme activity of RNA.
99. Describe how stem-loop structures are formed in RNA.
100. Summarize the journey from gene to functional protein in molecular terms.

## Answer Key

### Section A (MCQ Answers):
1. (d)
2. (a)
3. (c)
4. (b)
5. (c)
6. (b)
7. (a)
8. (b)
9. (b)
10. (c)
11. (b)
12. (c)
13. (c)
14. (c)
15. (c)
16. (c)
17. (a)
18. (b)
19. (b)
20. (c)
21. (b)
22. (b)
23. (c)
24. (c)
25. (b)
26. (b)
27. (b)
28. (b)
29. (b)
30. (b)
31. (b)
32. (c)
33. (c)
34. (b)
35. (b)
36. (c)
37. (b)
38. (b)
39. (b)
40. (c)
41. (c)
42. (b)
43. (b)
44. (c)
45. (c)
46. (d)
47. (b)
48. (b)
49. (a)
50. (b)
51. (b)
52. (b)
53. (b)
54. (d)
55. (c)
56. (d)
57. (b)
58. (b)
59. (b)
60. (d)
61. (b)
62. (b)
63. (b)
64. (b)
65. (a)
66. (b)
67. (d)
68. (b)
69. (c)
70. (b)
71. (a)
72. (b)
73. (a)
74. (a)
75. (a)
76. (c)
77. (a)
78. (b)
79. (b)
80. (a)
81. (b)
82. (a)
83. (d)
84. (c)
85. (b)
86. (d)
87. (b)
88. (b)
89. (a)
90. (b)
91. (a)
92. (b)
93. (b)
94. (b)
95. (b)
96. (a)
97. (b)
98. (a)
99. (b)
100. (a)

### Section B (Very Short Answer Questions):
1. Griffith concluded that a "transforming principle" from heat-killed S strain bacteria could transform live R strain bacteria into virulent S strain.
2. Streptococcus pneumoniae.
3. DNase degrades DNA, preventing transformation, thus proving DNA is the genetic material.
4. Radioactive phosphorus (³²P).
5. DNA.
6. James Watson and Francis Crick.
7. Chargaff's rule states that in DNA, the amount of adenine (A) equals thymine (T), and guanine (G) equals cytosine (C).
8. Thymine (T).
9. Ribose.
10. tRNA (transfer RNA).
11. Helicase.
12. DNA ligase joins Okazaki fragments and other DNA breaks.
13. DNA polymerase.
14. 5' to 3'.
15. Primase.
16. Topoisomerase.
17. Semi-conservative replication means each new DNA molecule consists of one original (parental) strand and one newly synthesized strand.
18. 3.4 nm.
19. 10 base pairs.
20. 0.34 nm.
21. Meselson and Stahl.
22. H2A, H2B, H3, or H4 (any one).
23. Histone H1 acts as a linker histone, binding DNA where it enters and leaves the nucleosome.
24. Chromatin is the complex of DNA and proteins (histones) found in eukaryotic cells.
25. Thymine (T).
26. RNA polymerase.
27. RNA polymerase II.
28. hnRNA (heterogeneous nuclear RNA) is the primary transcript in eukaryotes, containing both introns and exons.
29. Splicing is the process of removing introns and joining exons in hnRNA to form mature mRNA.
30. 5' capping is the addition of a 7-methylguanosine cap to the 5' end of hnRNA.
31. Polyadenylation is the addition of a poly-A tail (adenine nucleotides) to the 3' end of hnRNA.
32. Messenger RNA.
33. rRNA (ribosomal RNA).
34. A codon is a sequence of three nucleotides in mRNA that codes for a specific amino acid or a stop signal.
35. AUG.
36. UAA, UAG, or UGA (any one).
37. tRNA carries specific amino acids to the ribosome during protein synthesis.
38. Cloverleaf shape (2D) or L-shape (3D).
39. Degenerate code means most amino acids are coded by more than one codon.
40. Uracil (U).
41. Non-overlapping genetic code means each nucleotide is part of only one codon.
42. Ribosomes are the sites of protein synthesis (translation).
43. Translocation is the movement of the ribosome along the mRNA during translation.
44. The A-site (aminoacyl site) is where incoming aminoacyl-tRNAs bind.
45. Peptidyl transferase catalyzes the formation of peptide bonds between amino acids during translation.
46. The lacZ gene codes for β-galactosidase.
47. Lactose (or allolactose).
48. Promoter, operator, and structural genes (and a regulatory gene).
49. The repressor protein binds to the operator, preventing transcription of structural genes.
50. Escherichia coli (E. coli).
51. Genetic material is the substance that carries genetic information in living organisms and is passed from parent to offspring.
52. Nucleotides are the basic building blocks of nucleic acids (DNA and RNA), consisting of a sugar, a phosphate group, and a nitrogenous base.
53. A purine is a type of nitrogenous base with a double-ring structure (Adenine and Guanine).
54. A pyrimidine is a type of nitrogenous base with a single-ring structure (Cytosine, Thymine, and Uracil).
55. Phosphodiester bond.
56. Deoxyribose.
57. The central dogma describes the flow of genetic information from DNA to RNA to protein.
58. HIV (Human Immunodeficiency Virus).
59. Reverse transcriptase.
60. Gel electrophoresis.
61. VNTRs (Variable Number Tandem Repeats) are short, repetitive DNA sequences that vary in number among individuals.
62. DNA fingerprinting is a technique used to identify individuals based on their unique DNA patterns.
63. Introns are non-coding regions within a gene that are removed during RNA processing.
64. Exons are coding regions within a gene that are expressed and remain in mature mRNA.
65. snRNPs (small nuclear ribonucleoproteins) are involved in splicing hnRNA.
66. Post-transcriptional modification processes hnRNA into mature mRNA, ensuring its stability, transport, and translation.
67. A polynucleotide is a polymer made of many nucleotide monomers linked together (e.g., DNA or RNA).
68. Antiparallel strands means the two strands of DNA run in opposite 5' to 3' directions.
69. Euchromatin is loosely packed chromatin that is transcriptionally active.
70. Heterochromatin is densely packed chromatin that is transcriptionally inactive.
71. Human Genome Project.
72. Approximately 3 billion base pairs.
73. Pharmacogenomics is the study of how an individual's genetic makeup affects their response to drugs.
74. Sequence annotation is the process of identifying and assigning functions to different regions of a genome sequence.
75. ESTs (Expressed Sequence Tags) are short DNA sequences derived from mRNA, representing expressed genes.
76. Gene therapy is the introduction of genes into a patient's cells to treat or cure a disease.
77. SCID (Severe Combined Immunodeficiency) or Cystic Fibrosis (any one).
78. DNA fingerprinting is based on the uniqueness of VNTR patterns among individuals.
79. Privacy concerns or potential for discrimination (any one).
80. Molecular biology is the study of biological processes at the molecular level, focusing on DNA, RNA, and proteins.
81. Francis Crick.
82. Permease increases the permeability of the bacterial cell to lactose.
83. lacZ, lacY, lacA.
84. β-galactosidase hydrolyzes lactose into glucose and galactose.
85. Transacetylase is involved in the detoxification of non-metabolizable β-galactosides.
86. Replication is the process by which a DNA molecule makes an exact copy of itself.
87. Bacteria or plasmids (any one).
88. rRNA (ribosomal RNA).
89. Methionine (and acts as a start codon).
90. Okazaki fragments are short, newly synthesized DNA fragments on the lagging strand during replication.
91. The template strand is the DNA strand that serves as a template for RNA synthesis during transcription.
92. The coding strand is the non-template DNA strand that has the same sequence as the mRNA (except T instead of U).
93. RNA polymerase III.
94. Gene expression is the process by which information from a gene is used in the synthesis of a functional gene product.
95. Transcription is the process of synthesizing RNA from a DNA template.
96. Translation is the process of synthesizing protein from an mRNA template.
97. Hershey and Chase.
98. A triplet codon is a sequence of three nucleotides that codes for a specific amino acid.
99. A genome is the complete set of genetic material (DNA or RNA) present in an organism.
100. A gene is a unit of heredity that is transferred from a parent to offspring and is held to determine some characteristic of the offspring.

### Section C: Short Answer Questions (100 × 2 = 200 Marks)

1. Compare the experiments of Griffith and Avery with respect to their conclusions.
2. Explain the role of DNase in identifying DNA as genetic material.
3. Write any two differences between DNA and RNA.
4. How did Hershey and Chase confirm that DNA is the genetic material?
5. State two features of the Watson and Crick model of DNA.
6. Explain Chargaff’s rule with an example.
7. Why is DNA more stable than RNA? Give two reasons.
8. Mention two structural differences between purines and pyrimidines.
9. What are histones? What role do they play in packaging DNA?
10. Distinguish between euchromatin and heterochromatin.
11. Why is the genetic code said to be degenerate and universal?
12. Write the role of mRNA and tRNA in protein synthesis.
13. Describe any two features of the genetic code.
14. Mention any two functions of ribosomal RNA.
15. Why is the genetic code described as ‘non-overlapping’ and ‘comma-less’?
16. Distinguish between codon and anticodon.
17. Mention two types of post-transcriptional modifications in eukaryotes.
18. What are introns and exons? State one difference.
19. Write the function of RNA polymerase I and RNA polymerase II.
20. What is the function of spliceosomes? What are they composed of?
21. Define nucleosome. What is the role of H1 histone?
22. Describe the structure of tRNA in two points.
23. Explain the polarity of DNA strands.
24. Write any two enzymes involved in DNA replication and their roles.
25. Compare continuous and discontinuous DNA synthesis.
26. Why is DNA replication considered semi-conservative?
27. Differentiate between leading and lagging strands.
28. What is meant by Okazaki fragments? Where are they found?
29. What is a primer in DNA replication? Who synthesizes it?
30. State the roles of helicase and topoisomerase in DNA replication.
31. Explain the role of promoter and terminator sequences in transcription.
32. How is transcription in eukaryotes different from prokaryotes? (any two points)
33. What is meant by template strand and coding strand in transcription?
34. Explain briefly the process of capping and tailing.
35. Mention two differences between mRNA and hnRNA.
36. What are the three major types of RNA and their functions?
37. How does the tRNA molecule act as an adaptor?
38. Describe the cloverleaf structure of tRNA.
39. What is the significance of AUG codon in protein synthesis?
40. Write any two functions of the small ribosomal subunit.
41. What are the A, P, and E sites of a ribosome?
42. Explain the elongation stage of translation in two steps.
43. How is the translation process terminated?
44. Describe the function of peptidyl transferase during translation.
45. What is a polysome? State its significance.
46. Write the names and functions of the three structural genes in the lac operon.
47. How does the repressor protein regulate the lac operon?
48. What is an operon? Give an example.
49. Explain the function of the inducer in lac operon regulation.
50. Describe any two applications of the lac operon model.
51. What is gene regulation? Why is it important?
52. Describe the role of allolactose in the lac operon.
53. Write a brief note on inducible operon with example.
54. Explain the term “reverse transcription.”
55. Name two organisms in which reverse transcription is observed.
56. Define central dogma. Explain with a diagram.
57. Mention two applications of recombinant DNA technology.
58. What is DNA fingerprinting? Mention one application.
59. What are VNTRs? Why are they important in forensic science?
60. Explain the significance of HGP in human medicine.
61. Write any two goals of the Human Genome Project.
62. What are expressed sequence tags? Mention one use.
63. Define sequence annotation and mention its role in genomics.
64. What are the ethical concerns associated with DNA fingerprinting?
65. How are DNA fragments separated by gel electrophoresis?
66. Differentiate between coding and non-coding DNA.
67. Why is DNA considered a better genetic material than RNA?
68. How does the presence of a 2’-OH group make RNA less stable?
69. Write any two characteristics of genetic code.
70. How are histone octamers formed?
71. Explain the structure of chromatin with the beads-on-string model.
72. What are tandem repeats? Where are they found?
73. Mention two medical applications of Human Genome Project.
74. Define gene therapy. Mention one disease treated using this method.
75. Write a short note on pharmacogenomics.
76. Differentiate between template strand and coding strand.
77. What is an exonuclease? How is it useful?
78. Define mutation. Mention one cause.
79. What is wobble hypothesis?
80. What is a pseudogene?
81. How do prokaryotes couple transcription and translation?
82. What is the function of the operator gene?
83. Differentiate between regulatory and structural genes.
84. Describe the importance of DNA packaging.
85. Write any two differences between snRNA and hnRNA.
86. Why is DNA negatively charged?
87. What is meant by gene expression?
88. What is a repressible operon? Give an example.
89. What is the function of a repressor molecule?
90. Define upstream and downstream in genetic context.
91. Differentiate between DNA replication and transcription.
92. Write two differences between translation in prokaryotes and eukaryotes.
93. What is the function of the terminator sequence?
94. Mention any two tools required for gene cloning.
95. Define recombinant DNA.
96. State any two ethical issues in genome sequencing.
97. What is chromosomal DNA?
98. Mention two uses of DNA polymerase.
99. How is the double helix structure stabilized?
100. Why is the G≡C pair stronger than the A=T pair?

## Section D: Broad Answer Questions (100 × 3 = 300 Marks)

1. Describe the Hershey and Chase experiment and its conclusion.
2. Explain Avery, MacLeod, and McCarty’s experiment that proved DNA is the genetic material.
3. Compare the structures of DNA and RNA with respect to three major features.
4. Describe the Watson and Crick double helix model of DNA.
5. Write the role and significance of Chargaff’s rules in DNA structure.
6. Explain the semi-conservative nature of DNA replication with Meselson and Stahl’s experiment.
7. Describe the roles of DNA helicase, ligase, and polymerase in replication.
8. Differentiate between leading and lagging strand with a diagram.
9. Explain the mechanism of DNA replication in eukaryotes.
10. Write the structure and function of a nucleosome.
11. Compare euchromatin and heterochromatin with examples.
12. Describe the different types of RNA and their functions.
13. Describe the detailed structure of tRNA and its role in protein synthesis.
14. Explain the process of transcription in eukaryotes.
15. Describe post-transcriptional modifications of hnRNA.
16. Write a note on the three RNA polymerases in eukaryotes and their functions.
17. Explain the initiation, elongation, and termination stages of translation.
18. Describe the genetic code and its main characteristics with examples.
19. Discuss the significance of degenerate, universal, and non-overlapping nature of the genetic code.
20. Explain the lac operon model in detail.
21. Differentiate between inducible and repressible operons.
22. Write the function of all structural genes of lac operon.
23. Describe gene regulation in prokaryotes using an operon system.
24. How does allolactose regulate gene expression in lac operon?
25. Describe the structure of chromatin and the role of histones.
26. Explain the steps of post-transcriptional processing in mRNA formation.
27. Explain in detail the role of ribosomes in translation.
28. Write a comparative note on transcription in prokaryotes and eukaryotes.
29. Explain the central dogma of molecular biology with diagram.
30. Describe reverse transcription with a suitable example.
31. Describe the process of protein synthesis from transcription to translation.
32. What are Okazaki fragments? Explain their formation and significance.
33. Explain the concept of anticodon and its role in translation.
34. What are codons? Mention types and their role in translation.
35. Describe the structure of a ribosome and explain its role in translation.
36. Describe the function and importance of peptidyl transferase activity in ribosomes.
37. Explain the function of promoter, operator, and structural genes in an operon.
38. Discuss the steps involved in translation termination.
39. Describe DNA packaging in eukaryotic cells with a labeled diagram.
40. Explain the concept of chromatin remodeling and its biological significance.
41. Discuss the methods and ethical issues in DNA fingerprinting.
42. Write a short note on the Human Genome Project—goals, methods, and benefits.
43. What are the applications of HGP in medicine and forensics?
44. Explain the structure of DNA with reference to hydrogen bonding and base pairing.
45. Write a note on transcription factors and enhancers.
46. What is gene expression? Describe the regulation in eukaryotic cells.
47. Differentiate between structural and regulatory genes.
48. Explain the importance of histone modifications in gene regulation.
49. Describe the three sites of a ribosome and their roles during protein synthesis.
50. Write a note on the wobble hypothesis and its role in codon recognition.
51. Explain how gene expression is controlled at the transcriptional level.
52. Describe the experimental setup and results of Griffith’s experiment.
53. Compare the roles of helicase, topoisomerase, and ligase in DNA replication.
54. Describe how the template strand guides mRNA synthesis.
55. Explain how RNA processing increases the efficiency of gene expression.
56. What are VNTRs and how are they used in forensic science?
57. Describe the principle and procedure of gel electrophoresis.
58. Explain how DNA fingerprinting is used to determine paternity.
59. What is gene therapy? Explain with examples.
60. Write a note on the role of bioinformatics in genome annotation.
61. Explain the difference between exons and introns and their fate during RNA splicing.
62. Describe any three major tools required in recombinant DNA technology.
63. Explain the steps of Southern blotting.
64. Discuss the role of small nuclear RNA in mRNA processing.
65. What are the molecular differences between DNA replication and transcription?
66. How does RNA polymerase recognize the promoter and initiate transcription?
67. Write a note on the concept and functions of a gene.
68. Describe three types of point mutations and their effects.
69. Explain the concept of genetic code degeneracy and silent mutations.
70. Compare and contrast DNA and chromosomal mutations.
71. Write a note on regulatory sequences in prokaryotic genomes.
72. Describe how DNA methylation affects gene expression.
73. Explain the molecular mechanism of antisense RNA technology.
74. Discuss the importance of coding vs. non-coding DNA.
75. Describe three common applications of genetic engineering.
76. Write about any three medical applications of recombinant DNA technology.
77. Explain the role of ESTs and sequence annotation in HGP.
78. Discuss the ethical, legal, and social issues of genetic information.
79. Explain how transcription and translation are coupled in prokaryotes.
80. Describe the experimental method used to prove that DNA replicates semi-conservatively.
81. What is the molecular structure of a nucleosome? Draw and label.
82. How do ribosomes facilitate polypeptide elongation?
83. Describe three differences between prokaryotic and eukaryotic translation.
84. Explain how alternative splicing leads to protein diversity.
85. Describe the molecular mechanism of action of restriction enzymes.
86. Explain how genes are mapped using sequencing techniques.
87. Describe any three key bioinformatics databases used in genomics.
88. Compare the characteristics of mitochondrial and nuclear DNA.
89. Explain how transcription is terminated in prokaryotes.
90. Describe the method of creating a cDNA library.
91. Explain the concept of split genes and its discovery.
92. Describe how polymerase chain reaction (PCR) works.
93. Discuss the applications and importance of PCR in biology.
94. Describe the entire mechanism of lac operon regulation with diagrams.
95. How are DNA-binding motifs important for gene regulation?
96. What are pseudogenes? How do they arise?
97. Explain the differences in DNA packaging between prokaryotes and eukaryotes.
98. Write a short note on the ribozyme activity of RNA.
99. Describe how stem-loop structures are formed in RNA.
100. Summarize the journey from gene to functional protein in molecular terms.

## Answer Key

### Section A (MCQ Answers):
1. (d)
2. (a)
3. (c)
4. (b)
5. (c)
6. (b)
7. (a)
8. (b)
9. (b)
10. (c)
11. (b)
12. (c)
13. (c)
14. (c)
15. (c)
16. (c)
17. (a)
18. (b)
19. (b)
20. (c)
21. (b)
22. (b)
23. (c)
24. (c)
25. (b)
26. (b)
27. (b)
28. (b)
29. (b)
30. (b)
31. (b)
32. (c)
33. (c)
34. (b)
35. (b)
36. (c)
37. (b)
38. (b)
39. (b)
40. (c)
41. (c)
42. (b)
43. (b)
44. (c)
45. (c)
46. (d)
47. (b)
48. (b)
49. (a)
50. (b)
51. (b)
52. (b)
53. (b)
54. (d)
55. (c)
56. (d)
57. (b)
58. (b)
59. (b)
60. (d)
61. (b)
62. (b)
63. (b)
64. (b)
65. (a)
66. (b)
67. (d)
68. (b)
69. (c)
70. (b)
71. (a)
72. (b)
73. (a)
74. (a)
75. (a)
76. (c)
77. (a)
78. (b)
79. (b)
80. (a)
81. (b)
82. (a)
83. (d)
84. (c)
85. (b)
86. (d)
87. (b)
88. (b)
89. (a)
90. (b)
91. (a)
92. (b)
93. (b)
94. (b)
95. (b)
96. (a)
97. (b)
98. (a)
99. (b)
100. (a)

### Section B (Very Short Answer Questions):
1. Griffith concluded that a "transforming principle" from heat-killed S strain bacteria could transform live R strain bacteria into virulent S strain.
2. Streptococcus pneumoniae.
3. DNase degrades DNA, preventing transformation, thus proving DNA is the genetic material.
4. Radioactive phosphorus (³²P).
5. DNA.
6. James Watson and Francis Crick.
7. Chargaff's rule states that in DNA, the amount of adenine (A) equals thymine (T), and guanine (G) equals cytosine (C).
8. Thymine (T).
9. Ribose.
10. tRNA (transfer RNA).
11. Helicase.
12. DNA ligase joins Okazaki fragments and other DNA breaks.
13. DNA polymerase.
14. 5' to 3'.
15. Primase.
16. Topoisomerase.
17. Semi-conservative replication means each new DNA molecule consists of one original (parental) strand and one newly synthesized strand.
18. 3.4 nm.
19. 10 base pairs.
20. 0.34 nm.
21. Meselson and Stahl.
22. H2A, H2B, H3, or H4 (any one).
23. Histone H1 acts as a linker histone, binding DNA where it enters and leaves the nucleosome.
24. Chromatin is the complex of DNA and proteins (histones) found in eukaryotic cells.
25. Thymine (T).
26. RNA polymerase.
27. RNA polymerase II.
28. hnRNA (heterogeneous nuclear RNA) is the primary transcript in eukaryotes, containing both introns and exons.
29. Splicing is the process of removing introns and joining exons in hnRNA to form mature mRNA.
30. 5' capping is the addition of a 7-methylguanosine cap to the 5' end of hnRNA.
31. Polyadenylation is the addition of a poly-A tail (adenine nucleotides) to the 3' end of hnRNA.
32. Messenger RNA.
33. rRNA (ribosomal RNA).
34. A codon is a sequence of three nucleotides in mRNA that codes for a specific amino acid or a stop signal.
35. AUG.
36. UAA, UAG, or UGA (any one).
37. tRNA carries specific amino acids to the ribosome during protein synthesis.
38. Cloverleaf shape (2D) or L-shape (3D).
39. Degenerate code means most amino acids are coded by more than one codon.
40. Uracil (U).
41. Non-overlapping genetic code means each nucleotide is part of only one codon.
42. Ribosomes are the sites of protein synthesis (translation).
43. Translocation is the movement of the ribosome along the mRNA during translation.
44. The A-site (aminoacyl site) is where incoming aminoacyl-tRNAs bind.
45. Peptidyl transferase catalyzes the formation of peptide bonds between amino acids during translation.
46. The lacZ gene codes for β-galactosidase.
47. Lactose (or allolactose).
48. Promoter, operator, and structural genes (and a regulatory gene).
49. The repressor protein binds to the operator, preventing transcription of structural genes.
50. Escherichia coli (E. coli).
51. Genetic material is the substance that carries genetic information in living organisms and is passed from parent to offspring.
52. Nucleotides are the basic building blocks of nucleic acids (DNA and RNA), consisting of a sugar, a phosphate group, and a nitrogenous base.
53. A purine is a type of nitrogenous base with a double-ring structure (Adenine and Guanine).
54. A pyrimidine is a type of nitrogenous base with a single-ring structure (Cytosine, Thymine, and Uracil).
55. Phosphodiester bond.
56. Deoxyribose.
57. The central dogma describes the flow of genetic information from DNA to RNA to protein.
58. HIV (Human Immunodeficiency Virus).
59. Reverse transcriptase.
60. Gel electrophoresis.
61. VNTRs (Variable Number Tandem Repeats) are short, repetitive DNA sequences that vary in number among individuals.
62. DNA fingerprinting is a technique used to identify individuals based on their unique DNA patterns.
63. Introns are non-coding regions within a gene that are removed during RNA processing.
64. Exons are coding regions within a gene that are expressed and remain in mature mRNA.
65. snRNPs (small nuclear ribonucleoproteins) are involved in splicing hnRNA.
66. Post-transcriptional modification processes hnRNA into mature mRNA, ensuring its stability, transport, and translation.
67. A polynucleotide is a polymer made of many nucleotide monomers linked together (e.g., DNA or RNA).
68. Antiparallel strands means the two strands of DNA run in opposite 5' to 3' directions.
69. Euchromatin is loosely packed chromatin that is transcriptionally active.
70. Heterochromatin is densely packed chromatin that is transcriptionally inactive.
71. Human Genome Project.
72. Approximately 3 billion base pairs.
73. Pharmacogenomics is the study of how an individual's genetic makeup affects their response to drugs.
74. Sequence annotation is the process of identifying and assigning functions to different regions of a genome sequence.
75. ESTs (Expressed Sequence Tags) are short DNA sequences derived from mRNA, representing expressed genes.
76. Gene therapy is the introduction of genes into a patient's cells to treat or cure a disease.
77. SCID (Severe Combined Immunodeficiency) or Cystic Fibrosis (any one).
78. DNA fingerprinting is based on the uniqueness of VNTR patterns among individuals.
79. Privacy concerns or potential for discrimination (any one).
80. Molecular biology is the study of biological processes at the molecular level, focusing on DNA, RNA, and proteins.
81. Francis Crick.
82. Permease increases the permeability of the bacterial cell to lactose.
83. lacZ, lacY, lacA.
84. β-galactosidase hydrolyzes lactose into glucose and galactose.
85. Transacetylase is involved in the detoxification of non-metabolizable β-galactosides.
86. Replication is the process by which a DNA molecule makes an exact copy of itself.
87. Bacteria or plasmids (any one).
88. rRNA (ribosomal RNA).
89. Methionine (and acts as a start codon).
90. Okazaki fragments are short, newly synthesized DNA fragments on the lagging strand during replication.
91. The template strand is the DNA strand that serves as a template for RNA synthesis during transcription.
92. The coding strand is the non-template DNA strand that has the same sequence as the mRNA (except T instead of U).
93. RNA polymerase III.
94. Gene expression is the process by which information from a gene is used in the synthesis of a functional gene product.
95. Transcription is the process of synthesizing RNA from a DNA template.
96. Translation is the process of synthesizing protein from an mRNA template.
97. Hershey and Chase.
98. A triplet codon is a sequence of three nucleotides that codes for a specific amino acid.
99. A genome is the complete set of genetic material (DNA or RNA) present in an organism.
100. A gene is a unit of heredity that is transferred from a parent to offspring and is held to determine some characteristic of the offspring.

### Section C: Short Answer Questions (100 × 2 = 200 Marks)

1. Compare the experiments of Griffith and Avery with respect to their conclusions.
2. Explain the role of DNase in identifying DNA as genetic material.
3. Write any two differences between DNA and RNA.
4. How did Hershey and Chase confirm that DNA is the genetic material?
5. State two features of the Watson and Crick model of DNA.
6. Explain Chargaff’s rule with an example.
7. Why is DNA more stable than RNA? Give two reasons.
8. Mention two structural differences between purines and pyrimidines.
9. What are histones? What role do they play in packaging DNA?
10. Distinguish between euchromatin and heterochromatin.
11. Why is the genetic code said to be degenerate and universal?
12. Write the role of mRNA and tRNA in protein synthesis.
13. Describe any two features of the genetic code.
14. Mention any two functions of ribosomal RNA.
15. Why is the genetic code described as ‘non-overlapping’ and ‘comma-less’?
16. Distinguish between codon and anticodon.
17. Mention two types of post-transcriptional modifications in eukaryotes.
18. What are introns and exons? State one difference.
19. Write the function of RNA polymerase I and RNA polymerase II.
20. What is the function of spliceosomes? What are they composed of?
21. Define nucleosome. What is the role of H1 histone?
22. Describe the structure of tRNA in two points.
23. Explain the polarity of DNA strands.
24. Write any two enzymes involved in DNA replication and their roles.
25. Compare continuous and discontinuous DNA synthesis.
26. Why is DNA replication considered semi-conservative?
27. Differentiate between leading and lagging strands.
28. What is meant by Okazaki fragments? Where are they found?
29. What is a primer in DNA replication? Who synthesizes it?
30. State the roles of helicase and topoisomerase in DNA replication.
31. Explain the role of promoter and terminator sequences in transcription.
32. How is transcription in eukaryotes different from prokaryotes? (any two points)
33. What is meant by template strand and coding strand in transcription?
34. Explain briefly the process of capping and tailing.
35. Mention two differences between mRNA and hnRNA.
36. What are the three major types of RNA and their functions?
37. How does the tRNA molecule act as an adaptor?
38. Describe the cloverleaf structure of tRNA.
39. What is the significance of AUG codon in protein synthesis?
40. Write any two functions of the small ribosomal subunit.
41. What are the A, P, and E sites of a ribosome?
42. Explain the elongation stage of translation in two steps.
43. How is the translation process terminated?
44. Describe the function of peptidyl transferase during translation.
45. What is a polysome? State its significance.
46. Write the names and functions of the three structural genes in the lac operon.
47. How does the repressor protein regulate the lac operon?
48. What is an operon? Give an example.
49. Explain the function of the inducer in lac operon regulation.
50. Describe any two applications of the lac operon model.
51. What is gene regulation? Why is it important?
52. Describe the role of allolactose in the lac operon.
53. Write a brief note on inducible operon with example.
54. Explain the term “reverse transcription.”
55. Name two organisms in which reverse transcription is observed.
56. Define central dogma. Explain with a diagram.
57. Mention two applications of recombinant DNA technology.
58. What is DNA fingerprinting? Mention one application.
59. What are VNTRs? Why are they important in forensic science?
60. Explain the significance of HGP in human medicine.
61. Write any two goals of the Human Genome Project.
62. What are expressed sequence tags? Mention one use.
63. Define sequence annotation and mention its role in genomics.
64. What are the ethical concerns associated with DNA fingerprinting?
65. How are DNA fragments separated by gel electrophoresis?
66. Differentiate between coding and non-coding DNA.
67. Why is DNA considered a better genetic material than RNA?
68. How does the presence of a 2’-OH group make RNA less stable?
69. Write any two characteristics of genetic code.
70. How are histone octamers formed?
71. Explain the structure of chromatin with the beads-on-string model.
72. What are tandem repeats? Where are they found?
73. Mention two medical applications of Human Genome Project.
74. Define gene therapy. Mention one disease treated using this method.
75. Write a short note on pharmacogenomics.
76. Differentiate between template strand and coding strand.
77. What is an exonuclease? How is it useful?
78. Define mutation. Mention one cause.
79. What is wobble hypothesis?
80. What is a pseudogene?
81. How do prokaryotes couple transcription and translation?
82. What is the function of the operator gene?
83. Differentiate between regulatory and structural genes.
84. Describe the importance of DNA packaging.
85. Write any two differences between snRNA and hnRNA.
86. Why is DNA negatively charged?
87. What is meant by gene expression?
88. What is a repressible operon? Give an example.
89. What is the function of a repressor molecule?
90. Define upstream and downstream in genetic context.
91. Differentiate between DNA replication and transcription.
92. Write two differences between translation in prokaryotes and eukaryotes.
93. What is the function of the terminator sequence?
94. Mention any two tools required for gene cloning.
95. Define recombinant DNA.
96. State any two ethical issues in genome sequencing.
97. What is chromosomal DNA?
98. Mention two uses of DNA polymerase.
99. How is the double helix structure stabilized?
100. Why is the G≡C pair stronger than the A=T pair?

## Section D: Broad Answer Questions (100 × 3 = 300 Marks)

1. Describe the Hershey and Chase experiment and its conclusion.
2. Explain Avery, MacLeod, and McCarty’s experiment that proved DNA is the genetic material.
3. Compare the structures of DNA and RNA with respect to three major features.
4. Describe the Watson and Crick double helix model of DNA.
5. Write the role and significance of Chargaff’s rules in DNA structure.
6. Explain the semi-conservative nature of DNA replication with Meselson and Stahl’s experiment.
7. Describe the roles of DNA helicase, ligase, and polymerase in replication.
8. Differentiate between leading and lagging strand with a diagram.
9. Explain the mechanism of DNA replication in eukaryotes.
10. Write the structure and function of a nucleosome.
11. Compare euchromatin and heterochromatin with examples.
12. Describe the different types of RNA and their functions.
13. Describe the detailed structure of tRNA and its role in protein synthesis.
14. Explain the process of transcription in eukaryotes.
15. Describe post-transcriptional modifications of hnRNA.
16. Write a note on the three RNA polymerases in eukaryotes and their functions.
17. Explain the initiation, elongation, and termination stages of translation.
18. Describe the genetic code and its main characteristics with examples.
19. Discuss the significance of degenerate, universal, and non-overlapping nature of the genetic code.
20. Explain the lac operon model in detail.
21. Differentiate between inducible and repressible operons.
22. Write the function of all structural genes of lac operon.
23. Describe gene regulation in prokaryotes using an operon system.
24. How does allolactose regulate gene expression in lac operon?
25. Describe the structure of chromatin and the role of histones.
26. Explain the steps of post-transcriptional processing in mRNA formation.
27. Explain in detail the role of ribosomes in translation.
28. Write a comparative note on transcription in prokaryotes and eukaryotes.
29. Explain the central dogma of molecular biology with diagram.
30. Describe reverse transcription with a suitable example.
31. Describe the process of protein synthesis from transcription to translation.
32. What are Okazaki fragments? Explain their formation and significance.
33. Explain the concept of anticodon and its role in translation.
34. What are codons? Mention types and their role in translation.
35. Describe the structure of a ribosome and explain its role in translation.
36. Describe the function and importance of peptidyl transferase activity in ribosomes.
37. Explain the function of promoter, operator, and structural genes in an operon.
38. Discuss the steps involved in translation termination.
39. Describe DNA packaging in eukaryotic cells with a labeled diagram.
40. Explain the concept of chromatin remodeling and its biological significance.
41. Discuss the methods and ethical issues in DNA fingerprinting.
42. Write a short note on the Human Genome Project—goals, methods, and benefits.
43. What are the applications of HGP in medicine and forensics?
44. Explain the structure of DNA with reference to hydrogen bonding and base pairing.
45. Write a note on transcription factors and enhancers.
46. What is gene expression? Describe the regulation in eukaryotic cells.
47. Differentiate between structural and regulatory genes.
48. Explain the importance of histone modifications in gene regulation.
49. Describe the three sites of a ribosome and their roles during protein synthesis.
50. Write a note on the wobble hypothesis and its role in codon recognition.
51. Explain how gene expression is controlled at the transcriptional level.
52. Describe the experimental setup and results of Griffith’s experiment.
53. Compare the roles of helicase, topoisomerase, and ligase in DNA replication.
54. Describe how the template strand guides mRNA synthesis.
55. Explain how RNA processing increases the efficiency of gene expression.
56. What are VNTRs and how are they used in forensic science?
57. Describe the principle and procedure of gel electrophoresis.
58. Explain how DNA fingerprinting is used to determine paternity.
59. What is gene therapy? Explain with examples.
60. Write a note on the role of bioinformatics in genome annotation.
61. Explain the difference between exons and introns and their fate during RNA splicing.
62. Describe any three major tools required in recombinant DNA technology.
63. Explain the steps of Southern blotting.
64. Discuss the role of small nuclear RNA in mRNA processing.
65. What are the molecular differences between DNA replication and transcription?
66. How does RNA polymerase recognize the promoter and initiate transcription?
67. Write a note on the concept and functions of a gene.
68. Describe three types of point mutations and their effects.
69. Explain the concept of genetic code degeneracy and silent mutations.
70. Compare and contrast DNA and chromosomal mutations.
71. Write a note on regulatory sequences in prokaryotic genomes.
72. Describe how DNA methylation affects gene expression.
73. Explain the molecular mechanism of antisense RNA technology.
74. Discuss the importance of coding vs. non-coding DNA.
75. Describe three common applications of genetic engineering.
76. Write about any three medical applications of recombinant DNA technology.
77. Explain the role of ESTs and sequence annotation in HGP.
78. Discuss the ethical, legal, and social issues of genetic information.
79. Explain how transcription and translation are coupled in prokaryotes.
80. Describe the experimental method used to prove that DNA replicates semi-conservatively.
81. What is the molecular structure of a nucleosome? Draw and label.
82. How do ribosomes facilitate polypeptide elongation?
83. Describe three differences between prokaryotic and eukaryotic translation.
84. Explain how alternative splicing leads to protein diversity.
85. Describe the molecular mechanism of action of restriction enzymes.
86. Explain how genes are mapped using sequencing techniques.
87. Describe any three key bioinformatics databases used in genomics.
88. Compare the characteristics of mitochondrial and nuclear DNA.
89. Explain how transcription is terminated in prokaryotes.
90. Describe the method of creating a cDNA library.
91. Explain the concept of split genes and its discovery.
92. Describe how polymerase chain reaction (PCR) works.
93. Discuss the applications and importance of PCR in biology.
94. Describe the entire mechanism of lac operon regulation with diagrams.
95. How are DNA-binding motifs important for gene regulation?
96. What are pseudogenes? How do they arise?
97. Explain the differences in DNA packaging between prokaryotes and eukaryotes.
98. Write a short note on the ribozyme activity of RNA.
99. Describe how stem-loop structures are formed in RNA.
100. Summarize the journey from gene to functional protein in molecular terms.

## Answer Key

### Section A (MCQ Answers):
1. (d)
2. (a)
3. (c)
4. (b)
5. (c)
6. (b)
7. (a)
8. (b)
9. (b)
10. (c)
11. (b)
12. (c)
13. (c)
14. (c)
15. (c)
16. (c)
17. (a)
18. (b)
19. (b)
20. (c)
21. (b)
22. (b)
23. (c)
24. (c)
25. (b)
26. (b)
27. (b)
28. (b)
29. (b)
30. (b)
31. (b)
32. (c)
33. (c)
34. (b)
35. (b)
36. (c)
37. (b)
38. (b)
39. (b)
40. (c)
41. (c)
42. (b)
43. (b)
44. (c)
45. (c)
46. (d)
47. (b)
48. (b)
49. (a)
50. (b)
51. (b)
52. (b)
53. (b)
54. (d)
55. (c)
56. (d)
57. (b)
58. (b)
59. (b)
60. (d)
61. (b)
62. (b)
63. (b)
64. (b)
65. (a)
66. (b)
67. (d)
68. (b)
69. (c)
70. (b)
71. (a)
72. (b)
73. (a)
74. (a)
75. (a)
76. (c)
77. (a)
78. (b)
79. (b)
80. (a)
81. (b)
82. (a)
83. (d)
84. (c)
85. (b)
86. (d)
87. (b)
88. (b)
89. (a)
90. (b)
91. (a)
92. (b)
93. (b)
94. (b)
95. (b)
96. (a)
97. (b)
98. (a)
99. (b)
100. (a)

### Section B (Very Short Answer Questions):
1. Griffith concluded that a "transforming principle" from heat-killed S strain bacteria could transform live R strain bacteria into virulent S strain.
2. Streptococcus pneumoniae.
3. DNase degrades DNA, preventing transformation, thus proving DNA is the genetic material.
4. Radioactive phosphorus (³²P).
5. DNA.
6. James Watson and Francis Crick.
7. Chargaff's rule states that in DNA, the amount of adenine (A) equals thymine (T), and guanine (G) equals cytosine (C).
8. Thymine (T).
9. Ribose.
10. tRNA (transfer RNA).
11. Helicase.
12. DNA ligase joins Okazaki fragments and other DNA breaks.
13. DNA polymerase.
14. 5' to 3'.
15. Primase.
16. Topoisomerase.
17. Semi-conservative replication means each new DNA molecule consists of one original (parental) strand and one newly synthesized strand.
18. 3.4 nm.
19. 10 base pairs.
20. 0.34 nm.
21. Meselson and Stahl.
22. H2A, H2B, H3, or H4 (any one).
23. Histone H1 acts as a linker histone, binding DNA where it enters and leaves the nucleosome.
24. Chromatin is the complex of DNA and proteins (histones) found in eukaryotic cells.
25. Thymine (T).
26. RNA polymerase.
27. RNA polymerase II.
28. hnRNA (heterogeneous nuclear RNA) is the primary transcript in eukaryotes, containing both introns and exons.
29. Splicing is the process of removing introns and joining exons in hnRNA to form mature mRNA.
30. 5' capping is the addition of a 7-methylguanosine cap to the 5' end of hnRNA.
31. Polyadenylation is the addition of a poly-A tail (adenine nucleotides) to the 3' end of hnRNA.
32. Messenger RNA.
33. rRNA (ribosomal RNA).
34. A codon is a sequence of three nucleotides in mRNA that codes for a specific amino acid or a stop signal.
35. AUG.
36. UAA, UAG, or UGA (any one).
37. tRNA carries specific amino acids to the ribosome during protein synthesis.
38. Cloverleaf shape (2D) or L-shape (3D).
39. Degenerate code means most amino acids are coded by more than one codon.
40. Uracil (U).
41. Non-overlapping genetic code means each nucleotide is part of only one codon.
42. Ribosomes are the sites of protein synthesis (translation).
43. Translocation is the movement of the ribosome along the mRNA during translation.
44. The A-site (aminoacyl site) is where incoming aminoacyl-tRNAs bind.
45. Peptidyl transferase catalyzes the formation of peptide bonds between amino acids during translation.
46. The lacZ gene codes for β-galactosidase.
47. Lactose (or allolactose).
48. Promoter, operator, and structural genes (and a regulatory gene).
49. The repressor protein binds to the operator, preventing transcription of structural genes.
50. Escherichia coli (E. coli).
51. Genetic material is the substance that carries genetic information in living organisms and is passed from parent to offspring.
52. Nucleotides are the basic building blocks of nucleic acids (DNA and RNA), consisting of a sugar, a phosphate group, and a nitrogenous base.
53. A purine is a type of nitrogenous base with a double-ring structure (Adenine and Guanine).
54. A pyrimidine is a type of nitrogenous base with a single-ring structure (Cytosine, Thymine, and Uracil).
55. Phosphodiester bond.
56. Deoxyribose.
57. The central dogma describes the flow of genetic information from DNA to RNA to protein.
58. HIV (Human Immunodeficiency Virus).
59. Reverse transcriptase.
60. Gel electrophoresis.
61. VNTRs (Variable Number Tandem Repeats) are short, repetitive DNA sequences that vary in number among individuals.
62. DNA fingerprinting is a technique used to identify individuals based on their unique DNA patterns.
63. Introns are non-coding regions within a gene that are removed during RNA processing.
64. Exons are coding regions within a gene that are expressed and remain in mature mRNA.
65. snRNPs (small nuclear ribonucleoproteins) are involved in splicing hnRNA.
66. Post-transcriptional modification processes hnRNA into mature mRNA, ensuring its stability, transport, and translation.
67. A polynucleotide is a polymer made of many nucleotide monomers linked together (e.g., DNA or RNA).
68. Antiparallel strands means the two strands of DNA run in opposite 5' to 3' directions.
69. Euchromatin is loosely packed chromatin that is transcriptionally active.
70. Heterochromatin is densely packed chromatin that is transcriptionally inactive.
71. Human Genome Project.
72. Approximately 3 billion base pairs.
73. Pharmacogenomics is the study of how an individual's genetic makeup affects their response to drugs.
74. Sequence annotation is the process of identifying and assigning functions to different regions of a genome sequence.
75. ESTs (Expressed Sequence Tags) are short DNA sequences derived from mRNA, representing expressed genes.
76. Gene therapy is the introduction of genes into a patient's cells to treat or cure a disease.
77. SCID (Severe Combined Immunodeficiency) or Cystic Fibrosis (any one).
78. DNA fingerprinting is based on the uniqueness of VNTR patterns among individuals.
79. Privacy concerns or potential for discrimination (any one).
80. Molecular biology is the study of biological processes at the molecular level, focusing on DNA, RNA, and proteins.
81. Francis Crick.
82. Permease increases the permeability of the bacterial cell to lactose.
83. lacZ, lacY, lacA.
84. β-galactosidase hydrolyzes lactose into glucose and galactose.
85. Transacetylase is involved in the detoxification of non-metabolizable β-galactosides.
86. Replication is the process by which a DNA molecule makes an exact copy of itself.
87. Bacteria or plasmids (any one).
88. rRNA (ribosomal RNA).
89. Methionine (and acts as a start codon).
90. Okazaki fragments are short, newly synthesized DNA fragments on the lagging strand during replication.
91. The template strand is the DNA strand that serves as a template for RNA synthesis during transcription.
92. The coding strand is the non-template DNA strand that has the same sequence as the mRNA (except T instead of U).
93. RNA polymerase III.
94. Gene expression is the process by which information from a gene is used in the synthesis of a functional gene product.
95. Transcription is the process of synthesizing RNA from a DNA template.
96. Translation is the process of synthesizing protein from an mRNA template.
97. Hershey and Chase.
98. A triplet codon is a sequence of three nucleotides that codes for a specific amino acid.
99. A genome is the complete set of genetic material (DNA or RNA) present in an organism.
100. A gene is a unit of heredity that is transferred from a parent to offspring and is held to determine some characteristic of the offspring.

### Section C: Short Answer Questions (100 × 2 = 200 Marks)

1.  **Compare the experiments of Griffith and Avery with respect to their conclusions.**
    Griffith's experiment concluded that a "transforming principle" from heat-killed S strain bacteria could transform R strain bacteria, but he did not identify the biochemical nature of this principle. Avery, MacLeod, and McCarty's experiment built upon Griffith's work and concluded that DNA was the genetic material responsible for the transformation.

2.  **Explain the role of DNase in identifying DNA as genetic material.**
    In the Avery, MacLeod, and McCarty experiment, DNase was used to digest DNA from the heat-killed S strain extract. When DNase was added, transformation of the R strain bacteria did not occur, meaning no live S strain bacteria were recovered. This demonstrated that DNA was essential for transformation and thus identified DNA as the genetic material.

3.  **Write any two differences between DNA and RNA.**
    Two differences between DNA and RNA are:
    1.  **Sugar:** DNA contains deoxyribose sugar, while RNA contains ribose sugar.
    2.  **Bases:** DNA contains Thymine (T), while RNA contains Uracil (U) instead of Thymine.

4.  **How did Hershey and Chase confirm that DNA is the genetic material?**
    Hershey and Chase confirmed DNA as the genetic material by labeling bacteriophage DNA with radioactive phosphorus (³²P) and proteins with radioactive sulfur (³⁵S). They found that only ³²P entered the bacterial cells during infection, and this radioactivity was passed on to progeny phages, indicating DNA, not protein, was the genetic material.

5.  **State two features of the Watson and Crick model of DNA.**
    Two features of the Watson and Crick model of DNA are:
    1.  DNA is made of two polynucleotide chains coiled in a right-handed helix.
    2.  The two chains have anti-parallel polarity (one runs 5'→3', the other 3'→5').

6.  **Explain Chargaff’s rule with an example.**
    Chargaff's rule states that in DNA, the amount of Adenine (A) is always equal to the amount of Thymine (T) (A=T), and the amount of Guanine (G) is always equal to the amount of Cytosine (C) (G=C). For example, if a DNA molecule has 20% Adenine, it will also have 20% Thymine.

7.  **Why is DNA more stable than RNA? Give two reasons.**
    DNA is more stable than RNA for two reasons:
    1.  DNA has a deoxyribose sugar, which lacks a 2'-OH group, making it less reactive than the ribose sugar in RNA.
    2.  DNA is double-stranded, providing more stability and protection against degradation compared to the generally single-stranded RNA.

8.  **Mention two structural differences between purines and pyrimidines.**
    Two structural differences between purines and pyrimidines are:
    1.  Purines (Adenine and Guanine) are larger, double-ringed nitrogenous bases.
    2.  Pyrimidines (Cytosine, Thymine, and Uracil) are smaller, single-ringed nitrogenous bases.

9.  **What are histones? What role do they play in packaging DNA?**
    Histones are a group of positively charged proteins. They play a crucial role in packaging DNA in eukaryotes by forming a histone octamer around which the negatively charged DNA wraps, forming nucleosomes, the basic unit of DNA packaging.

10. **Distinguish between euchromatin and heterochromatin.**
    The provided text does not explicitly distinguish between euchromatin and heterochromatin. However, based on general biological knowledge, euchromatin is loosely packed, transcriptionally active chromatin, while heterochromatin is densely packed, transcriptionally inactive chromatin.

11. **Why is the genetic code said to be degenerate and universal?**
    The genetic code is degenerate because most amino acids are coded by more than one codon. It is universal because the same codon codes for the same amino acid in almost all organisms, with only a few exceptions.

12. **Write the role of mRNA and tRNA in protein synthesis.**
    mRNA (messenger RNA) carries the genetic information from DNA in the nucleus to the ribosomes in the cytoplasm, serving as the template for protein synthesis. tRNA (transfer RNA) acts as an adaptor molecule, bringing specific amino acids to the ribosome according to the codons on the mRNA.

13. **Describe any two features of the genetic code.**
    Two features of the genetic code are:
    1.  **Triplet Code:** Each codon consists of three nucleotides.
    2.  **Non-overlapping:** No base is shared between adjacent codons.

14. **Mention any two functions of ribosomal RNA.**
    Two functions of ribosomal RNA (rRNA) are:
    1.  It is a structural component of ribosomes.
    2.  It has catalytic activity (peptidyl transferase) during protein synthesis, forming peptide bonds.

15. **Why is the genetic code described as ‘non-overlapping’ and ‘comma-less’?**
    The genetic code is described as 'non-overlapping' because no base is shared between adjacent codons. It is 'comma-less' because there are no intervening nucleotides or punctuation between codons; they are read consecutively.

16. **Distinguish between codon and anticodon.**
    A codon is a sequence of three consecutive nucleotides on mRNA that codes for a specific amino acid or a stop signal. An anticodon is a sequence of three nucleotides on a tRNA molecule that is complementary to a specific mRNA codon, allowing the tRNA to bring the correct amino acid.

17. **Mention two types of post-transcriptional modifications in eukaryotes.**
    Two types of post-transcriptional modifications in eukaryotes are:
    1.  **Splicing:** Removal of introns and ligation of exons.
    2.  **Capping:** Addition of methyl guanosine triphosphate to the 5'-end.

18. **What are introns and exons? State one difference.**
    Introns are non-coding regions within a gene that are removed during RNA processing. Exons are coding regions that are retained and expressed. One difference is that introns are removed during splicing, while exons are ligated together to form the mature mRNA.

19. **Write the function of RNA polymerase I and RNA polymerase II.**
    RNA polymerase I (RNA Pol I) transcribes ribosomal RNA (rRNA). RNA polymerase II (RNA Pol II) transcribes messenger RNA precursors (hnRNA).

20. **What is the function of spliceosomes? What are they composed of?**
    Spliceosomes are responsible for splicing, the process of removing introns and ligating exons in hnRNA. They are composed of small nuclear ribonucleoproteins (snRNPs).

21. **Define nucleosome. What is the role of H1 histone?**
    A nucleosome is the basic unit of DNA packaging in eukaryotes, consisting of a segment of DNA wound around a core of eight histone proteins (histone octamer). The H1 histone acts as a linker histone, binding the DNA where it enters and leaves the nucleosome, helping to stabilize the chromatin structure.

22. **Describe the structure of tRNA in two points.**
    Two points describing the structure of tRNA are:
    1.  It has a clover-leaf like 2D structure.
    2.  It has an L-shaped 3D structure.

23. **Explain the polarity of DNA strands.**
    The two polynucleotide chains of DNA have anti-parallel polarity, meaning one strand runs in the 5'→3' direction, and the complementary strand runs in the 3'→5' direction. This refers to the orientation of the sugar-phosphate backbone.

24. **Write any two enzymes involved in DNA replication and their roles.**
    Two enzymes involved in DNA replication and their roles are:
    1.  **DNA Helicase:** Unwinds the DNA double helix.
    2.  **DNA Polymerase:** Synthesizes new DNA strands by adding nucleotides complementary to the template strand.

25. **Compare continuous and discontinuous DNA synthesis.**
    The provided text does not explicitly compare continuous and discontinuous DNA synthesis. However, it mentions leading and lagging strands. Continuous synthesis occurs on the leading strand, which is synthesized continuously in the 5'→3' direction towards the replication fork. Discontinuous synthesis occurs on the lagging strand, which is synthesized in short fragments (Okazaki fragments) away from the replication fork.

26. **Why is DNA replication considered semi-conservative?**
    DNA replication is considered semi-conservative because each new DNA molecule consists of one original (parental) strand and one newly synthesized strand. This was experimentally proven by Meselson and Stahl.

27. **Differentiate between leading and lagging strands.**
    The leading strand is synthesized continuously in the 5'→3' direction towards the replication fork. The lagging strand is synthesized discontinuously in short fragments (Okazaki fragments) in the 5'→3' direction, but overall away from the replication fork.

28. **What is meant by Okazaki fragments? Where are they found?**
    Okazaki fragments are short DNA segments synthesized discontinuously on the lagging strand during DNA replication. They are found on the lagging strand of the replication fork.

29. **What is a primer in DNA replication? Who synthesizes it?**
    A primer in DNA replication is a short RNA segment that provides a free 3'-OH group for DNA polymerase to start synthesizing the new DNA strand. It is synthesized by the enzyme Primase.

30. **State the roles of helicase and topoisomerase in DNA replication.**
    **Helicase** unwinds the DNA double helix at the replication fork. **Topoisomerase** relieves the supercoiling tension that builds up ahead of the replication fork due to unwinding.

31. **Explain the role of promoter and terminator sequences in transcription.**
    The **promoter sequence** is the binding site for RNA polymerase, signaling the start point for transcription. The **terminator sequence** signals the end of transcription, causing RNA polymerase to stop and release the newly synthesized RNA.

32. **How is transcription in eukaryotes different from prokaryotes? (any two points)**
    Two differences in transcription between eukaryotes and prokaryotes are:
    1.  In eukaryotes, transcription occurs in the nucleus, while in prokaryotes, it occurs in the cytoplasm.
    2.  In eukaryotes, the primary transcript (hnRNA) undergoes extensive post-transcriptional processing (splicing, capping, tailing), which is generally absent in prokaryotes.

33. **What is meant by template strand and coding strand in transcription?**
    In transcription, the **template strand** (also known as the antisense strand) is the DNA strand that serves as the template for RNA synthesis. The **coding strand** (also known as the sense strand) is the non-template DNA strand that has a sequence identical to the newly synthesized mRNA (except for T in DNA being U in RNA).

34. **Explain briefly the process of capping and tailing.**
    **Capping** is the addition of an unusual nucleotide, methyl guanosine triphosphate, to the 5'-end of hnRNA. **Tailing** is the addition of adenylate residues (200-300) to the 3'-end of hnRNA in a template-independent manner. Both are post-transcriptional modifications in eukaryotes.

35. **Mention two differences between mRNA and hnRNA.**
    Two differences between mRNA and hnRNA are:
    1.  hnRNA is the primary transcript in eukaryotes and contains both introns and exons, while mRNA is the fully processed form of hnRNA with introns removed and is ready for translation.
    2.  hnRNA undergoes capping and tailing, which are modifications that result in the mature mRNA.

36. **What are the three major types of RNA and their functions?**
    The three major types of RNA and their functions are:
    1.  **mRNA (messenger RNA):** Carries genetic information from DNA to ribosomes for protein synthesis.
    2.  **tRNA (transfer RNA):** Acts as an adaptor, bringing specific amino acids to the ribosome during protein synthesis.
    3.  **rRNA (ribosomal RNA):** A structural and catalytic component of ribosomes, providing the site for protein synthesis.

37. **How does the tRNA molecule act as an adaptor?**
    The tRNA molecule acts as an adaptor because it has an anticodon loop that binds to a specific mRNA codon, and an amino acid acceptor end that binds to a specific amino acid. This allows it to "adapt" the genetic information from the mRNA codon into the corresponding amino acid in the polypeptide chain.

38. **Describe the cloverleaf structure of tRNA.**
    The cloverleaf structure describes the two-dimensional structure of tRNA, characterized by several loops and stems formed by base pairing within the single RNA strand. It typically includes an acceptor stem, a D loop, an anticodon loop, and a TψC loop.

39. **What is the significance of AUG codon in protein synthesis?**
    The AUG codon is significant in protein synthesis because it serves as the start codon, signaling the initiation of translation. It also codes for the amino acid Methionine.

40. **Write any two functions of the small ribosomal subunit.**
    Two functions of the small ribosomal subunit are:
    1.  It binds to the mRNA near the start codon during initiation of translation.
    2.  It plays a role in decoding the mRNA codons by interacting with tRNA anticodons.

41. **What are the A, P, and E sites of a ribosome?**
    The A, P, and E sites are binding sites for tRNA molecules on the ribosome during translation:
    *   **A-site (Aminoacyl site):** Where incoming aminoacyl-tRNAs bind.
    *   **P-site (Peptidyl site):** Where the tRNA carrying the growing polypeptide chain is located.
    *   **E-site (Exit site):** Where uncharged tRNAs exit the ribosome.

42. **Explain the elongation stage of translation in two steps.**
    Two steps in the elongation stage of translation are:
    1.  A tRNA carrying the next amino acid binds to the A-site of the ribosome, complementary to the mRNA codon.
    2.  A peptide bond is formed between the amino acid in the A-site and the growing polypeptide chain in the P-site, catalyzed by peptidyl transferase.

43. **How is the translation process terminated?**
    The translation process is terminated when a stop codon (UAA, UAG, or UGA) arrives at the A-site of the ribosome. Since no tRNA binds to these codons, release factors bind to the stop codon, causing the release of the polypeptide chain and dissociation of the ribosomal subunits.

44. **Describe the function of peptidyl transferase during translation.**
    Peptidyl transferase is an enzymatic activity of the ribosomal RNA (rRNA) within the large ribosomal subunit. Its function during translation is to catalyze the formation of peptide bonds between the amino acid in the A-site and the growing polypeptide chain in the P-site.

45. **What is a polysome? State its significance.**
    A polysome (or polyribosome) is a complex formed when multiple ribosomes are attached to a single mRNA molecule, simultaneously translating it into multiple polypeptide chains. Its significance is that it allows for the efficient and rapid synthesis of many copies of the same protein from a single mRNA template.

46. **Write the names and functions of the three structural genes in the lac operon.**
    The three structural genes in the lac operon and their functions are:
    1.  ***lacZ*:** Codes for β-galactosidase, which hydrolyzes lactose into glucose and galactose.
    2.  ***lacY*:** Codes for permease, which increases the permeability of the cell to β-galactosides (lactose).
    3.  ***lacA*:** Codes for transacetylase.

47. **How does the repressor protein regulate the lac operon?**
    When lactose is absent, the repressor protein (synthesized by the i gene) binds to the operator region of the lac operon, physically blocking RNA polymerase from transcribing the structural genes, thus switching the operon OFF.

48. **What is an operon? Give an example.**
    An operon is a functional unit of DNA containing a cluster of genes under the control of a single promoter and operator, which are transcribed together as a single mRNA. An example is the lac operon in *E. coli*.

49. **Explain the function of the inducer in lac operon regulation.**
    In lac operon regulation, lactose (or its isomer allolactose) acts as an inducer. It binds to the repressor protein, causing a conformational change that prevents the repressor from binding to the operator. This allows RNA polymerase to transcribe the structural genes, switching the operon ON.

50. **Describe any two applications of the lac operon model.**
    The provided text does not explicitly describe applications of the lac operon model beyond its role in gene regulation. However, the lac operon is a fundamental model for understanding gene regulation and has been widely used in molecular biology research and biotechnology, for example, in developing inducible expression systems for recombinant protein production.

51. **What is gene regulation? Why is it important?**
    Gene regulation is the process by which cells control the expression of their genes, determining which genes are turned on or off and to what extent. It is important because it allows cells to adapt to changing environmental conditions, differentiate into specialized cell types, and maintain proper cellular function by producing specific proteins only when and where they are needed.

52. **Describe the role of allolactose in the lac operon.**
    Allolactose, an isomer of lactose, acts as the inducer in the lac operon. When present, it binds to the repressor protein, causing a conformational change that prevents the repressor from binding to the operator. This allows transcription of the structural genes to proceed.

53. **Write a brief note on inducible operon with example.**
    An inducible operon is a type of operon where the expression of genes is normally turned off but can be turned on in the presence of a specific molecule called an inducer. The lac operon is a classic example of an inducible operon, where the presence of lactose induces the transcription of genes involved in lactose metabolism.

54. **Explain the term “reverse transcription.”**
    Reverse transcription is a process in which genetic information flows from RNA to DNA, which is the reverse of the typical central dogma. In this process, an RNA molecule acts as a template to synthesize a complementary DNA strand.

55. **Name two organisms in which reverse transcription is observed.**
    Reverse transcription is observed in some viruses, specifically retroviruses, such as HIV.

56. **Define central dogma. Explain with a diagram.**
    The Central Dogma of Molecular Biology, proposed by Francis Crick, states that genetic information flows from DNA to RNA to Protein.
    Diagram: `DNA (Replication) → DNA (Transcription) → RNA (Translation) → Protein`

57. **Mention two applications of recombinant DNA technology.**
    The provided text does not explicitly mention applications of recombinant DNA technology. However, it is a broad field with applications such as:
    1.  Production of therapeutic proteins (e.g., insulin, growth hormone).
    2.  Development of genetically modified organisms (GMOs) for agriculture or research.

58. **What is DNA fingerprinting? Mention one application.**
    DNA fingerprinting (or DNA profiling) is a technique used to identify individuals based on their unique DNA patterns. One application is in forensic science for identifying criminals or victims.

59. **What are VNTRs? Why are they important in forensic science?**
    VNTRs (Variable Number Tandem Repeats) are short, repetitive DNA sequences that vary in number from person to person. They are important in forensic science because their unique variations among individuals form the basis of DNA fingerprinting, allowing for individual identification in criminal investigations and paternity disputes.

60. **Explain the significance of HGP in human medicine.**
    The Human Genome Project (HGP) is significant in human medicine because it has provided a comprehensive map of the human genome, leading to a better understanding of human biology, health, and disease. This knowledge aids in the diagnosis, treatment, and prevention of genetic disorders, and facilitates personalized medicine (pharmacogenomics).

61. **Write any two goals of the Human Genome Project.**
    Two goals of the Human Genome Project were:
    1.  To identify all the approximately 20,000-25,000 genes in human DNA.
    2.  To determine the sequences of the 3 billion chemical base pairs that make up human DNA.

62. **What are expressed sequence tags? Mention one use.**
    Expressed Sequence Tags (ESTs) are short, single-pass sequences from cDNA that represent genes that are actively expressed as RNA. One use is to identify and map genes within the human genome.

63. **Define sequence annotation and mention its role in genomics.**
    Sequence annotation is the process of identifying and assigning functions to different regions of a sequenced genome (both coding and non-coding). Its role in genomics is to make sense of the vast amount of raw sequence data by identifying genes, regulatory elements, and other features, which is crucial for understanding genome function and evolution.

64. **What are the ethical concerns associated with DNA fingerprinting?**
    Ethical concerns associated with DNA fingerprinting include privacy concerns (storage and use of DNA data), potential for discrimination (e.g., in employment or insurance), misinterpretation of results, and issues related to consent.

65. **How are DNA fragments separated by gel electrophoresis?**
    In gel electrophoresis, DNA fragments are separated based on their size and charge. DNA is negatively charged, so when an electric current is applied, the fragments migrate towards the positive electrode. Smaller fragments move faster and further through the gel matrix than larger fragments, resulting in separation by size.

66. **Differentiate between coding and non-coding DNA.**
    Coding DNA refers to the regions of DNA that contain instructions for making proteins (exons). Non-coding DNA refers to the regions of DNA that do not directly code for proteins (e.g., introns, regulatory sequences, repetitive DNA).

67. **Why is DNA considered a better genetic material than RNA?**
    DNA is considered a better genetic material than RNA primarily due to its greater stability. DNA's deoxyribose sugar lacks a 2'-OH group, making it less reactive, and its double-stranded structure provides more protection against degradation and allows for more accurate replication and repair mechanisms.

68. **How does the presence of a 2’-OH group make RNA less stable?**
    The presence of a 2'-OH (hydroxyl) group in the ribose sugar of RNA makes it more reactive and susceptible to hydrolysis (breakdown) compared to the deoxyribose sugar in DNA, which lacks this group. This increased reactivity contributes to RNA's lower stability.

69. **Write any two characteristics of genetic code.**
    Two characteristics of the genetic code are:
    1.  **Specific and Unambiguous:** Each codon codes for only one specific amino acid.
    2.  **Start and Stop Codons:** It has specific start (AUG) and stop (UAA, UAG, UGA) signals.

70. **How are histone octamers formed?**
    Histone octamers are formed by two molecules each of four core histone proteins: H2A, H2B, H3, and H4. These eight histone proteins assemble to form the core around which DNA is wrapped to form a nucleosome.

71. **Explain the structure of chromatin with the beads-on-string model.**
    The beads-on-string model describes the basic structure of chromatin. It depicts DNA as being wound around histone octamers, forming repeating units called nucleosomes. These nucleosomes appear like "beads" on a "string" (the linker DNA) when viewed under an electron microscope.

72. **What are tandem repeats? Where are they found?**
    Tandem repeats are short, repetitive DNA sequences that are arranged consecutively (one after another) in the genome. They are found throughout the genome, and a specific type, Variable Number Tandem Repeats (VNTRs), are particularly important in DNA fingerprinting.

73. **Mention two medical applications of Human Genome Project.**
    Two medical applications of the Human Genome Project are:
    1.  Diagnosis, treatment, and prevention of genetic disorders.
    2.  Pharmacogenomics (personalized medicine), which involves tailoring drug treatments based on an individual's genetic makeup.

74. **Define gene therapy. Mention one disease treated using this method.**
    Gene therapy is a technique that involves introducing, removing, or changing genetic material in a person's cells to treat or prevent disease. The provided text does not mention a specific disease treated using this method. However, gene therapy has been explored for diseases like severe combined immunodeficiency (SCID).

75. **Write a short note on pharmacogenomics.**
    Pharmacogenomics is a field that studies how an individual's genetic makeup influences their response to drugs. It aims to develop personalized medicine approaches by using genetic information to predict how a patient will react to a particular medication, allowing for more effective and safer drug treatments.

76. **Differentiate between template strand and coding strand.**
    In transcription, the template strand is the DNA strand that serves as the guide for RNA synthesis, with RNA polymerase reading its sequence to build the complementary RNA molecule. The coding strand is the non-template DNA strand, which has a sequence identical to the newly synthesized mRNA (except for T being replaced by U).

77. **What is an exonuclease? How is it useful?**
    The provided text does not explicitly define exonuclease. However, in the context of DNA replication, DNA polymerase has exonuclease activity, which is useful for proofreading. It allows the enzyme to remove incorrectly paired nucleotides from the end of a growing DNA strand, thereby correcting errors during replication.

78. **Define mutation. Mention one cause.**
    The provided text does not explicitly define mutation or its causes. However, a mutation is a change in the DNA sequence. One cause of mutation can be errors during DNA replication.

79. **What is wobble hypothesis?**
    The provided text does not mention the wobble hypothesis. The wobble hypothesis states that the pairing between the first two bases of the mRNA codon and the tRNA anticodon is strict, but the pairing at the third base is more flexible, allowing a single tRNA to recognize multiple codons.

80. **What is a pseudogene?**
    The provided text does not mention pseudogenes. A pseudogene is a DNA sequence that resembles a functional gene but has lost its protein-coding ability due to mutations.

81. **How do prokaryotes couple transcription and translation?**
    In prokaryotes, transcription and translation can be coupled because both processes occur in the cytoplasm and there is no nuclear membrane to separate them. As mRNA is being transcribed, ribosomes can immediately attach to the nascent mRNA and begin translation, even before transcription is complete.

82. **What is the function of the operator gene?**
    The operator gene (or operator region) in an operon is the binding site for the repressor protein. Its function is to regulate the transcription of the structural genes by either allowing or blocking the movement of RNA polymerase.

83. **Differentiate between regulatory and structural genes.**
    **Regulatory genes** (like the i gene in the lac operon) code for regulatory proteins (e.g., repressors) that control the expression of other genes. **Structural genes** (like *lacZ*, *lacY*, *lacA* in the lac operon) code for proteins that have a direct metabolic or structural function in the cell.

84. **Describe the importance of DNA packaging.**
    DNA packaging is important because it allows the long DNA molecule to be compactly organized and fit within the confined space of the nucleus in eukaryotic cells. It also plays a crucial role in regulating gene expression and protecting the DNA from damage.

85. **Write any two differences between snRNA and hnRNA.**
    Two differences between snRNA and hnRNA are:
    1.  **Function:** hnRNA (heterogeneous nuclear RNA) is the primary transcript that serves as a precursor to mRNA, while snRNA (small nuclear RNA) is involved in the splicing of hnRNA.
    2.  **Size/Processing:** hnRNA is a larger, unprocessed transcript containing introns and exons, whereas snRNA is a smaller RNA molecule that is part of the spliceosome complex.

86. **Why is DNA negatively charged?**
    DNA is negatively charged due to the presence of phosphate groups in its sugar-phosphate backbone. Each phosphate group carries a negative charge, which contributes to the overall negative charge of the DNA molecule.

87. **What is meant by gene expression?**
    Gene expression is the process by which the information encoded in a gene is used to synthesize a functional gene product, such as a protein or an RNA molecule. It involves transcription (DNA to RNA) and, for proteins, translation (RNA to protein).

88. **What is a repressible operon? Give an example.**
    The provided text does not explicitly define repressible operon. However, a repressible operon is a type of operon where the expression of genes is normally turned on but can be turned off (repressed) in the presence of a specific molecule (e.g., the trp operon, which is repressed by tryptophan).

89. **What is the function of a repressor molecule?**
    A repressor molecule (a protein) binds to the operator region of an operon, thereby preventing RNA polymerase from transcribing the structural genes. Its function is to negatively regulate gene expression by switching off the operon.

90. **Define upstream and downstream in genetic context.**
    In a genetic context, **upstream** refers to the region of DNA located towards the 5' end of a gene or sequence, relative to a reference point (e.g., the start of transcription). **Downstream** refers to the region of DNA located towards the 3' end of a gene or sequence, relative to a reference point.

91. **Differentiate between DNA replication and transcription.**
    **DNA replication** is the process of synthesizing a new DNA molecule from an existing DNA template, resulting in two identical DNA molecules. **Transcription** is the process of synthesizing an RNA molecule from a DNA template. Replication copies the entire genome, while transcription copies specific genes.

92. **Write two differences between translation in prokaryotes and eukaryotes.**
    Two differences between translation in prokaryotes and eukaryotes are:
    1.  In prokaryotes, translation can be coupled with transcription (occurs simultaneously), while in eukaryotes, transcription occurs in the nucleus and translation occurs in the cytoplasm, so they are spatially and temporally separated.
    2.  The initiator amino acid in prokaryotes is formylmethionine, while in eukaryotes, it is methionine.

93. **What is the function of the terminator sequence?**
    The terminator sequence is a specific DNA sequence that signals the end of transcription. When RNA polymerase encounters the terminator sequence, it stops transcribing and releases the newly synthesized RNA molecule.

94. **Mention any two tools required for gene cloning.**
    The provided text does not explicitly mention tools for gene cloning. However, common tools include:
    1.  Restriction enzymes (to cut DNA at specific sites).
    2.  DNA ligase (to join DNA fragments).

95. **Define recombinant DNA.**
    The provided text does not explicitly define recombinant DNA. However, recombinant DNA is a DNA molecule formed by laboratory methods of genetic recombination (such as molecular cloning) to bring together genetic material from multiple sources, creating sequences that would not otherwise be found in the genome.

96. **State any two ethical issues in genome sequencing.**
    Two ethical issues in genome sequencing are:
    1.  Privacy concerns regarding the storage, access, and use of an individual's genetic information.
    2.  Potential for discrimination based on genetic predispositions (e.g., in employment or insurance).

97. **What is chromosomal DNA?**
    Chromosomal DNA refers to the DNA that is organized into chromosomes within the nucleus of eukaryotic cells (or in the nucleoid region of prokaryotic cells). It carries the genetic information of the organism.

98. **Mention two uses of DNA polymerase.**
    Two uses of DNA polymerase are:
    1.  Synthesizing new DNA strands during DNA replication.
    2.  Repairing damaged DNA by filling in gaps or replacing incorrect nucleotides.

99. **How is the double helix structure stabilized?**
    The double helix structure of DNA is stabilized primarily by two forces:
    1.  **Hydrogen bonds:** Specific hydrogen bonds form between complementary base pairs (A=T with two H-bonds, G≡C with three H-bonds).
    2.  **Base stacking interactions:** Hydrophobic interactions between the stacked base pairs in the interior of the helix.

100. **Why is the G≡C pair stronger than the A=T pair?**
    The G≡C (Guanine-Cytosine) pair is stronger than the A=T (Adenine-Thymine) pair because Guanine and Cytosine form three hydrogen bonds between them, whereas Adenine and Thymine form only two hydrogen bonds. More hydrogen bonds lead to greater stability and a stronger interaction.

## Section D: Broad Answer Questions (100 × 3 = 300 Marks)

1.  **Describe the Hershey and Chase experiment and its conclusion.**
    *   **Experiment:** Alfred Hershey and Martha Chase used bacteriophages (viruses) to determine whether DNA or protein is the genetic material. They labeled DNA with radioactive phosphorus (³²P) and proteins with radioactive sulfur (³⁵S). These labeled phages were allowed to infect *E. coli* bacteria. After infection, the mixture was blended to separate viral coats from bacteria and then centrifuged.
    *   **Observations:** Bacteria infected with ³²P-labeled phages were radioactive, and ³²P was found in the progeny phages. Bacteria infected with ³⁵S-labeled phages were not radioactive, and ³⁵S remained in the supernatant (viral coats).
    *   **Conclusion:** DNA, not protein, is the genetic material that is passed from phage to bacteria.

2.  **Explain Avery, MacLeod, and McCarty’s experiment that proved DNA is the genetic material.**
    *   **Experiment:** Oswald Avery, Colin MacLeod, and Maclyn McCarty purified biochemicals (proteins, DNA, RNA) from heat-killed virulent (S) *Streptococcus pneumoniae* cells. They treated the extract with proteases (to digest proteins), RNases (to digest RNA), and DNases (to digest DNA), and then mixed each treated extract with live non-virulent (R) bacteria.
    *   **Observations:** Transformation of R cells into S cells occurred when proteins and RNA were digested. However, transformation was inhibited when DNA was digested (i.e., no S strain appeared when DNase was added).
    *   **Conclusion:** They concluded that DNA is the genetic material responsible for transformation.

3.  **Compare the structures of DNA and RNA with respect to three major features.**
    | Feature | DNA | RNA |
    |---|---|---|
    | **Sugar** | Deoxyribose | Ribose |
    | **Bases** | Adenine, Guanine, Cytosine, Thymine | Adenine, Guanine, Cytosine, Uracil |
    | **Strands** | Double-stranded | Single-stranded (mostly) |

4.  **Describe the Watson and Crick double helix model of DNA.**
    *   The Watson and Crick model describes DNA as a double helix composed of two polynucleotide chains.
    *   The backbones are made of sugar-phosphate, and the nitrogenous bases project inwards.
    *   The two chains run in anti-parallel directions (one 5'→3', the other 3'→5').
    *   Bases in the two strands are paired through hydrogen bonds: Adenine (A) pairs with Thymine (T) via two hydrogen bonds, and Guanine (G) pairs with Cytosine (C) via three hydrogen bonds.
    *   The helix is right-handed, with a pitch of 3.4 nm and approximately 10 base pairs per turn. The distance between two base pairs is 0.34 nm.

5.  **Write the role and significance of Chargaff’s rules in DNA structure.**
    *   **Role:** Erwin Chargaff observed that in DNA, the amount of Adenine (A) is always equal to Thymine (T) (A=T), and the amount of Guanine (G) is always equal to Cytosine (C) (G=C). Also, the ratio of (A+G) to (T+C) is always one.
    *   **Significance:** These rules were crucial for Watson and Crick in deducing the double helix structure of DNA, particularly the specific base pairing (A with T, and G with C) that forms the rungs of the DNA ladder.

6.  **Explain the semi-conservative nature of DNA replication with Meselson and Stahl’s experiment.**
    *   **Semi-conservative nature:** Each new DNA molecule produced after replication consists of one original (parental) strand and one newly synthesized strand.
    *   **Meselson and Stahl’s Experiment (1958):**
        *   They grew *E. coli* in a medium containing heavy nitrogen (¹⁵N) for several generations, so all DNA contained ¹⁵N.
        *   They then transferred the bacteria to a medium containing light nitrogen (¹⁴N).
        *   After one generation, the DNA was extracted and centrifuged, revealing a single hybrid band (¹⁵N-¹⁴N), indicating that each new DNA molecule contained one old and one new strand.
        *   After two generations, two bands were observed: one hybrid (¹⁵N-¹⁴N) and one light (¹⁴N-¹⁴N), further confirming the semi-conservative mode of replication.

7.  **Describe the roles of DNA helicase, ligase, and polymerase in replication.**
    *   **DNA Helicase:** Unwinds the DNA double helix by breaking the hydrogen bonds between complementary base pairs, creating a replication fork.
    *   **DNA Ligase:** Joins the Okazaki fragments (short DNA segments synthesized on the lagging strand) together to form a continuous strand.
    *   **DNA Polymerase:** Synthesizes new DNA strands by adding nucleotides complementary to the template strand. It also has proofreading activity to correct errors. It works in a 5'→3' direction.

8.  **Differentiate between leading and lagging strand with a diagram.**
    *   **Leading Strand:** Synthesized continuously in the 5'→3' direction, moving towards the replication fork. It requires only one primer.
    *   **Lagging Strand:** Synthesized discontinuously in short segments called Okazaki fragments, also in the 5'→3' direction, but moving away from the replication fork. Each Okazaki fragment requires a new primer. These fragments are later joined by DNA ligase.
    *   *(Diagram cannot be provided in text format, but conceptually, the leading strand extends smoothly from the replication fork, while the lagging strand is synthesized in short, backward segments.)*

9.  **Explain the mechanism of DNA replication in eukaryotes.**
    *   Eukaryotic DNA replication is semi-conservative and occurs during the S phase of the cell cycle.
    *   It initiates at multiple origins of replication along each chromosome, forming multiple replication bubbles.
    *   DNA helicases unwind the DNA, and single-strand binding proteins stabilize the separated strands.
    *   Primase synthesizes RNA primers, which DNA polymerase then extends.
    *   DNA polymerase synthesizes the leading strand continuously and the lagging strand discontinuously (forming Okazaki fragments).
    *   RNA primers are removed by RNase H and FEN1, and the gaps are filled by DNA polymerase.
    *   DNA ligase joins the Okazaki fragments.
    *   Topoisomerases relieve supercoiling ahead of the replication fork.
    *   Telomerase replicates the ends of linear chromosomes (telomeres).

10. **Write the structure and function of a nucleosome.**
    *   **Structure:** A nucleosome is the basic structural unit of DNA packaging in eukaryotes. It consists of a segment of DNA (approximately 146 base pairs) wound around a core of eight histone proteins, called a histone octamer. The histone octamer is composed of two molecules each of H2A, H2B, H3, and H4 histones. An H1 histone acts as a linker, binding the DNA where it enters and leaves the nucleosome.
    *   **Function:** Nucleosomes help to condense and package the vast amount of eukaryotic DNA into the compact structure of chromatin, which fits within the cell nucleus. They also play a crucial role in regulating gene expression by controlling the accessibility of DNA to transcription machinery.

11. **Compare euchromatin and heterochromatin with examples.**
    *   **Euchromatin:**
        *   **Structure:** Loosely packed, less condensed chromatin.
        *   **Activity:** Transcriptionally active (genes are accessible for expression).
        *   **Staining:** Stains lightly with DNA stains.
        *   **Examples:** Most of the chromatin in active cells, gene-rich regions.
    *   **Heterochromatin:**
        *   **Structure:** Densely packed, highly condensed chromatin.
        *   **Activity:** Transcriptionally inactive or silenced (genes are generally not expressed).
        *   **Staining:** Stains darkly with DNA stains.
        *   **Examples:** Centromeres, telomeres, Barr bodies (inactivated X chromosomes in females).

12. **Describe the different types of RNA and their functions.**
    *   **mRNA (messenger RNA):** Carries genetic information from DNA in the nucleus to the ribosomes in the cytoplasm, serving as a template for protein synthesis.
    *   **tRNA (transfer RNA):** Acts as an adaptor molecule, carrying specific amino acids to the ribosome and recognizing corresponding codons on the mRNA during translation.
    *   **rRNA (ribosomal RNA):** A structural and catalytic component of ribosomes, where protein synthesis occurs. It possesses peptidyl transferase activity.
    *   **hnRNA (heterogeneous nuclear RNA):** The primary transcript in eukaryotes, containing both introns and exons. It undergoes post-transcriptional modifications to become mRNA.
    *   **snRNA (small nuclear RNA):** Involved in the splicing of hnRNA, forming part of the spliceosome complex.

13. **Describe the detailed structure of tRNA and its role in protein synthesis.**
    *   **Structure:** tRNA is a small RNA molecule, typically 70-90 nucleotides long. It has a characteristic clover-leaf like 2D structure due to intramolecular base pairing, which folds into an L-shaped 3D structure. Key features include:
        *   **Anticodon loop:** Contains a three-nucleotide sequence (anticodon) that base-pairs with a complementary codon on the mRNA.
        *   **Amino acid acceptor arm:** The 3' end where a specific amino acid is covalently attached.
        *   **D loop and TψC loop:** Involved in tRNA recognition by aminoacyl-tRNA synthetases and ribosome binding, respectively.
    *   **Role in protein synthesis:** tRNA acts as a molecular bridge between mRNA codons and amino acids. Each tRNA carries a specific amino acid and, via its anticodon, ensures that the correct amino acid is delivered to the ribosome according to the mRNA sequence, thus building the polypeptide chain.

14. **Explain the process of transcription in eukaryotes.**
    *   Transcription in eukaryotes is the process of synthesizing RNA from a DNA template, occurring in the nucleus.
    *   It involves three main RNA polymerases: RNA Pol I (synthesizes rRNA), RNA Pol II (synthesizes mRNA precursors/hnRNA), and RNA Pol III (synthesizes tRNA and 5s rRNA).
    *   RNA polymerase binds to a promoter region on the DNA with the help of transcription factors.
    *   The DNA double helix unwinds, and RNA polymerase synthesizes an RNA strand complementary to the template DNA strand, in a 5'→3' direction.
    *   Transcription continues until a termination signal is reached.
    *   The primary transcript (hnRNA for mRNA) then undergoes post-transcriptional modifications.

15. **Describe post-transcriptional modifications of hnRNA.**
    *   After transcription, heterogeneous nuclear RNA (hnRNA) undergoes several modifications to become mature mRNA:
        1.  **Splicing:** Introns (non-coding regions) are removed, and exons (coding regions) are ligated together. This process is carried out by spliceosomes, complexes of snRNAs and proteins.
        2.  **Capping:** An unusual nucleotide, 7-methylguanosine triphosphate, is added to the 5'-end of the hnRNA. This cap protects the mRNA from degradation and is important for ribosome binding during translation.
        3.  **Tailing:** A poly-A tail (200-300 adenylate residues) is added to the 3'-end of the hnRNA in a template-independent manner. The poly-A tail enhances mRNA stability and aids in its export from the nucleus and translation.

16. **Write a note on the three RNA polymerases in eukaryotes and their functions.**
    *   Eukaryotic cells possess three distinct nuclear RNA polymerases:
        *   **RNA Polymerase I (RNA Pol I):** Located in the nucleolus, it synthesizes most ribosomal RNAs (rRNAs), specifically the precursors for 18S, 5.8S, and 28S rRNAs.
        *   **RNA Polymerase II (RNA Pol II):** Located in the nucleoplasm, it synthesizes messenger RNA precursors (hnRNA), which are then processed into mRNA. It also synthesizes some small RNAs, including snRNAs.
        *   **RNA Polymerase III (RNA Pol III):** Located in the nucleoplasm, it synthesizes transfer RNAs (tRNAs), 5S rRNA, and other small RNAs.

17. **Explain the initiation, elongation, and termination stages of translation.**
    *   **Translation** is the process of protein synthesis from an mRNA template, occurring on ribosomes.
    *   **Initiation:** The small ribosomal subunit binds to the mRNA near the start codon (AUG). The initiator tRNA (carrying methionine) binds to the start codon. The large ribosomal subunit then joins, forming the initiation complex, with the initiator tRNA in the P-site.
    *   **Elongation:** Amino acids are added one by one to the polypeptide chain.
        *   A new aminoacyl-tRNA enters the A-site, matching the mRNA codon.
        *   A peptide bond is formed between the amino acid in the A-site and the growing polypeptide chain in the P-site, catalyzed by peptidyl transferase (an rRNA activity).
        *   The ribosome translocates (moves) along the mRNA, shifting the tRNAs: the A-site tRNA moves to the P-site, the P-site tRNA moves to the E-site (exit site), and the E-site tRNA is released.
    *   **Termination:** Elongation continues until a stop codon (UAA, UAG, or UGA) enters the A-site. There are no tRNAs for stop codons. Release factors bind to the stop codon, causing the hydrolysis of the bond between the polypeptide and the tRNA in the P-site, leading to the release of the polypeptide chain and the dissociation of the ribosomal subunits from the mRNA.

18. **Describe the genetic code and its main characteristics with examples.**
    *   **Genetic Code:** The set of rules by which information encoded in genetic material (DNA or RNA sequences) is translated into proteins (amino acid sequences). It is read in triplets of nucleotides called codons.
    *   **Main Characteristics:**
        *   **Triplet Code:** Each codon consists of three nucleotides (e.g., AUG, GGC).
        *   **Degenerate (Redundant):** Most amino acids are coded by more than one codon (e.g., both UUA and UUG code for Leucine). This provides some protection against point mutations.
        *   **Unambiguous and Specific:** Each codon specifies only one particular amino acid (e.g., AUG always codes for Methionine).
        *   **Universal:** The same codon codes for the same amino acid in almost all organisms, from bacteria to humans (with minor exceptions).
        *   **Non-overlapping:** Codons are read sequentially, one after another, without any overlap (e.g., ABCDEF is read as ABC, DEF, not ABC, BCD, CDE).
        *   **Comma-less:** There are no intervening nucleotides or "commas" between codons.
        *   **Start Codon:** AUG (codes for Methionine and signals the start of translation).
        *   **Stop Codons (Nonsense Codons):** UAA, UAG, UGA (do not code for any amino acid; they signal the termination of translation).

19. **Discuss the significance of degenerate, universal, and non-overlapping nature of the genetic code.**
    *   **Degenerate (Redundant):** The degeneracy of the genetic code means that multiple codons can specify the same amino acid. This is significant because it provides a buffer against point mutations. A single nucleotide change in a codon might still result in the same amino acid being incorporated into the protein, thus preventing a change in protein function.
    *   **Universal:** The near-universality of the genetic code across all forms of life (with a few minor exceptions) is a strong indicator of a common evolutionary origin. This characteristic is also fundamental to genetic engineering, allowing genes from one organism to be expressed and produce functional proteins in another, enabling technologies like the production of human insulin in bacteria.
    *   **Non-overlapping:** The non-overlapping nature ensures that each nucleotide in the mRNA sequence is read only once as part of a single codon. This precise reading frame is critical for maintaining the correct sequence of amino acids in the polypeptide chain. If the code were overlapping, a single base change would affect multiple codons, leading to more drastic changes in the protein.

20. **Explain the lac operon model in detail.**
    *   The **lac operon** is an inducible operon in *E. coli* that regulates the metabolism of lactose. It is a classic example of gene regulation in prokaryotes.
    *   **Components:**
        *   **Promoter (P):** The binding site for RNA polymerase to initiate transcription.
        *   **Operator (O):** A regulatory sequence located between the promoter and structural genes, where the repressor protein binds.
        *   **Structural Genes:**
            *   *lacZ*: Codes for β-galactosidase, which hydrolyzes lactose into glucose and galactose.
            *   *lacY*: Codes for permease, which increases the permeability of the cell membrane to lactose.
            *   *lacA*: Codes for transacetylase, whose exact role in lactose metabolism is less clear but is part of the operon.
        *   **Regulator Gene (i gene):** Located upstream of the operon, it codes for the lac repressor protein.
    *   **Mechanism:**
        *   **When Lactose is Absent:** The repressor protein, continuously synthesized by the *i* gene, binds tightly to the operator region. This binding physically blocks RNA polymerase from moving past the promoter and transcribing the structural genes. Thus, the operon is switched OFF, and lactose-metabolizing enzymes are not produced.
        *   **When Lactose is Present:** Lactose enters the cell and is converted into allolactose (an isomer). Allolactose acts as an **inducer**. It binds to the repressor protein, causing a conformational change that reduces the repressor's affinity for the operator. The repressor detaches from the operator, allowing RNA polymerase to bind to the promoter and transcribe the *lacZ*, *lacY*, and *lacA* genes. The operon is switched ON, and enzymes for lactose metabolism are produced.

21. **Differentiate between inducible and repressible operons.**
    *   **Inducible Operons:**
        *   **Default State:** Usually "off" (transcription is repressed).
        *   **Activation:** Turned "on" by the presence of a specific molecule called an "inducer." The inducer typically binds to a repressor protein, preventing it from binding to the operator, thereby allowing transcription.
        *   **Example:** Lac operon (induced by lactose/allolactose). These operons typically control catabolic pathways (breaking down substances).
    *   **Repressible Operons:**
        *   **Default State:** Usually "on" (transcription is active).
        *   **Repression:** Turned "off" by the presence of a specific molecule called a "corepressor." The corepressor binds to an inactive repressor protein, activating it, and enabling it to bind to the operator, thereby blocking transcription.
        *   **Example:** Tryptophan operon (repressed by tryptophan). These operons typically control anabolic pathways (synthesizing substances).

22. **Write the function of all structural genes of lac operon.**
    *   **lacZ:** Codes for **β-galactosidase**, an enzyme that hydrolyzes (breaks down) lactose into its monosaccharide components, glucose and galactose.
    *   **lacY:** Codes for **lactose permease**, a membrane protein that facilitates the transport of lactose into the bacterial cell.
    *   **lacA:** Codes for **thiogalactoside transacetylase**, an enzyme whose precise physiological role in lactose metabolism is not fully understood, but it is thought to be involved in detoxifying non-metabolizable galactosides.

23. **Describe gene regulation in prokaryotes using an operon system.**
    *   In prokaryotes, gene expression is primarily regulated at the transcriptional level, often through operon systems. An operon is a functional unit of DNA containing a cluster of genes under the control of a single promoter and operator.
    *   **Mechanism:** A regulatory gene produces a repressor protein.
        *   In **inducible operons** (like the lac operon), the repressor normally binds to the operator, blocking RNA polymerase and preventing transcription. An inducer molecule (e.g., lactose) binds to the repressor, causing it to detach from the operator, allowing transcription to proceed.
        *   In **repressible operons** (like the trp operon), the repressor is normally inactive, allowing transcription. A corepressor molecule (e.g., tryptophan) binds to the repressor, activating it, which then binds to the operator and blocks transcription.
    *   This system allows prokaryotes to rapidly adapt to changes in their environment by turning genes on or off as needed for metabolic efficiency.

24. **How does allolactose regulate gene expression in lac operon?**
    *   Allolactose acts as the **inducer** in the lac operon. When lactose is present in the environment, a small amount is converted into allolactose inside the *E. coli* cell.
    *   Allolactose then binds to the lac repressor protein. This binding causes a conformational change in the repressor, reducing its affinity for the operator DNA sequence.
    *   As a result, the repressor detaches from the operator. This unblocks the promoter, allowing RNA polymerase to bind and initiate transcription of the structural genes (*lacZ*, *lacY*, *lacA*), leading to the production of enzymes necessary for lactose metabolism. Thus, allolactose effectively switches the lac operon "on."

25. **Describe the structure of chromatin and the role of histones.**
    *   **Chromatin Structure:** Chromatin is the complex of DNA and proteins (primarily histones) found within the nucleus of eukaryotic cells. Its fundamental repeating unit is the **nucleosome**, which resembles "beads-on-a-string." Each nucleosome consists of approximately 146 base pairs of DNA wrapped around a core of eight histone proteins (a histone octamer). These nucleosomes are further compacted and folded into higher-order structures, eventually forming chromosomes.
    *   **Role of Histones:** Histones are small, positively charged proteins (due to a high content of lysine and arginine) that bind tightly to the negatively charged DNA. Their primary role is to:
        *   **Package DNA:** They act as spools around which DNA winds, enabling the vast length of DNA to be condensed and fit within the small confines of the nucleus.
        *   **Regulate Gene Expression:** The degree of DNA packaging by histones influences gene accessibility. Tightly packed chromatin (heterochromatin) generally restricts gene expression, while loosely packed chromatin (euchromatin) allows for gene transcription. Modifications to histones can alter chromatin structure and thus gene activity.

26. **Explain the steps of post-transcriptional processing in mRNA formation.**
    *   See answer to Question 15.

27. **Explain in detail the role of ribosomes in translation.**
    *   Ribosomes are complex molecular machines, composed of ribosomal RNA (rRNA) and proteins, that serve as the sites for protein synthesis (translation). Their roles include:
        *   **Binding mRNA:** The small ribosomal subunit binds to the mRNA, correctly positioning the start codon (AUG) for translation initiation.
        *   **Providing Binding Sites for tRNA:** Ribosomes have three key sites for tRNA binding:
            *   **A (Aminoacyl) site:** Where incoming aminoacyl-tRNAs (tRNAs carrying an amino acid) bind to the mRNA codon.
            *   **P (Peptidyl) site:** Holds the tRNA carrying the growing polypeptide chain.
            *   **E (Exit) site:** Where spent, uncharged tRNAs exit the ribosome.
        *   **Catalyzing Peptide Bond Formation:** The large ribosomal subunit contains the peptidyl transferase activity (a ribozyme, meaning it's catalyzed by rRNA), which forms peptide bonds between the amino acid in the A-site and the polypeptide chain in the P-site.
        *   **Translocation:** The ribosome moves along the mRNA in a 5'→3' direction, shifting the tRNAs from A to P to E sites, ensuring sequential reading of codons and elongation of the polypeptide.
        *   **Termination:** Ribosomes recognize stop codons, leading to the release of the completed polypeptide chain and dissociation of the ribosomal subunits.

28. **Write a comparative note on transcription in prokaryotes and eukaryotes.**
    | Feature | Prokaryotic Transcription | Eukaryotic Transcription |
    |---|---|---|
    | **Location** | Cytoplasm | Nucleus |
    | **Coupling with Translation** | Coupled (can occur simultaneously) | Not coupled (transcription in nucleus, translation in cytoplasm) |
    | **RNA Polymerases** | One type (synthesizes all RNA types) | Three types (Pol I, II, III for different RNA types) |
    | **Promoter Structure** | Simpler (e.g., -10 and -35 boxes) | More complex (e.g., TATA box, enhancers, multiple transcription factors) |
    | **Primary Transcript** | mRNA is directly functional (no introns) | hnRNA (contains introns and exons) |
    | **Post-transcriptional Modification** | Minimal or none | Extensive (splicing, 5' capping, 3' polyadenylation) |
    | **Initiation Factors** | Simpler sigma factors | Complex array of general and specific transcription factors |

29. **Explain the central dogma of molecular biology with diagram.**
    *   The **Central Dogma of Molecular Biology**, proposed by Francis Crick, describes the flow of genetic information within a biological system. It states that genetic information flows from DNA to RNA to protein.
    *   **Flow:**
        *   **Replication:** DNA makes copies of itself (DNA → DNA).
        *   **Transcription:** Genetic information from DNA is transcribed into RNA (DNA → RNA).
        *   **Translation:** The information in RNA is translated into a sequence of amino acids to form a protein (RNA → Protein).
    *   **Diagrammatic Representation:**
        ```
        DNA --(Replication)--> DNA
         |
         | (Transcription)
         V
        RNA
         |
         | (Translation)
         V
        Protein
        ```
    *   This fundamental concept explains how genetic instructions are used to build and operate an organism.

30. **Describe reverse transcription with a suitable example.**
    *   **Reverse Transcription:** This is a process that deviates from the central dogma, where genetic information flows from RNA back to DNA. In this process, an RNA molecule serves as a template for the synthesis of a complementary DNA strand.
    *   **Enzyme:** This process is catalyzed by the enzyme **reverse transcriptase**.
    *   **Example:** Retroviruses, such as **HIV (Human Immunodeficiency Virus)**, are a prime example. HIV has an RNA genome. Upon infecting a host cell, its reverse transcriptase enzyme uses the viral RNA as a template to synthesize a DNA copy. This viral DNA is then integrated into the host cell's genome, allowing the virus to replicate and produce new viral particles.

31. **Describe the process of protein synthesis from transcription to translation.**
    *   **Protein synthesis** is a two-stage process: transcription (DNA to RNA) and translation (RNA to protein).
    *   **1. Transcription (in the nucleus for eukaryotes, cytoplasm for prokaryotes):**
        *   A segment of DNA (a gene) serves as a template.
        *   RNA polymerase binds to the promoter region of the gene.
        *   The DNA double helix unwinds, and RNA polymerase synthesizes a complementary RNA molecule (pre-mRNA in eukaryotes, mRNA in prokaryotes) using ribonucleotides.
        *   In eukaryotes, the pre-mRNA undergoes **post-transcriptional modifications** (5' capping, 3' polyadenylation, and splicing to remove introns and ligate exons) to become mature mRNA.
    *   **2. Translation (on ribosomes in the cytoplasm):**
        *   The mature mRNA molecule moves to the cytoplasm and binds to a ribosome.
        *   **Initiation:** The ribosome scans the mRNA for the start codon (AUG), and the initiator tRNA (carrying methionine) binds to it.
        *   **Elongation:** tRNAs, each carrying a specific amino acid, enter the ribosome's A-site, matching their anticodons to the mRNA codons. A peptide bond is formed between the incoming amino acid and the growing polypeptide chain in the P-site. The ribosome then translocates, moving the tRNAs and mRNA.
        *   **Termination:** When a stop codon (UAA, UAG, UGA) is reached in the A-site, release factors bind, causing the release of the completed polypeptide chain from the ribosome.
    *   The newly synthesized polypeptide then folds into its functional 3D structure, often with the help of chaperones, and may undergo further post-translational modifications.

32. **What are Okazaki fragments? Explain their formation and significance.**
    *   **Okazaki fragments** are short, newly synthesized DNA segments that are formed on the lagging strand during DNA replication. In eukaryotes, they are typically 100-200 nucleotides long, while in prokaryotes, they can be 1000-2000 nucleotides long.
    *   **Formation:** DNA polymerase can only synthesize DNA in the 5'→3' direction. Since the two strands of the DNA double helix are anti-parallel, and the replication fork unwinds in one direction, one strand (the leading strand) can be synthesized continuously. However, the other strand (the lagging strand) must be synthesized discontinuously. As the replication fork opens, primase synthesizes short RNA primers, and DNA polymerase then extends these primers in the 5'→3' direction, creating the Okazaki fragments.
    *   **Significance:** Okazaki fragments are crucial for the replication of the lagging strand, ensuring that both strands of the DNA molecule can be replicated despite the directional constraint of DNA polymerase. These fragments are later joined together by DNA ligase after the RNA primers are removed and replaced with DNA.

33. **Explain the concept of anticodon and its role in translation.**
    *   **Anticodon:** An anticodon is a three-nucleotide sequence located on one loop of a transfer RNA (tRNA) molecule. It is complementary to a specific codon on the messenger RNA (mRNA) molecule.
    *   **Role in Translation:** During translation, the anticodon plays a critical role in ensuring the accuracy of protein synthesis. As the ribosome moves along the mRNA, each codon is recognized by a specific tRNA carrying its corresponding amino acid. The anticodon on the tRNA base-pairs with the complementary codon on the mRNA. This precise pairing ensures that the correct amino acid is delivered to the ribosome at the appropriate position in the growing polypeptide chain, thereby maintaining the integrity of the genetic code and the resulting protein sequence.

34. **What are codons? Mention types and their role in translation.**
    *   **Codons:** Codons are sequences of three consecutive nucleotides (a triplet) in a messenger RNA (mRNA) molecule that specify a particular amino acid or a signal for termination of protein synthesis.
    *   **Types:**
        *   **Start Codon:** AUG. It codes for the amino acid Methionine (or formylmethionine in prokaryotes) and also serves as the signal to initiate translation.
        *   **Sense Codons:** The 61 codons that specify the 20 standard amino acids.
        *   **Stop Codons (Nonsense Codons):** UAA, UAG, and UGA. These three codons do not code for any amino acid; instead, they signal the termination of translation.
    *   **Role in Translation:** Codons are the fundamental units of the genetic code. During translation, the sequence of codons on the mRNA dictates the precise order in which amino acids are assembled to form a polypeptide chain. Each codon is recognized by a complementary anticodon on a tRNA molecule, which brings the corresponding amino acid to the ribosome. This sequential reading of codons ensures the accurate synthesis of proteins according to the genetic instructions.

35. **Describe the structure of a ribosome and explain its role in translation.**
    *   **Structure of a Ribosome:** Ribosomes are complex cellular organelles composed of ribosomal RNA (rRNA) and numerous ribosomal proteins. They consist of two main subunits: a large subunit and a small subunit. In prokaryotes, these are 50S and 30S subunits, forming a 70S ribosome. In eukaryotes, they are 60S and 40S subunits, forming an 80S ribosome. Each subunit has a distinct shape and contains specific rRNAs and proteins.
    *   **Role in Translation:** Ribosomes are the molecular machines responsible for protein synthesis (translation). Their key roles include:
        *   **mRNA Binding:** The small subunit binds to the mRNA molecule, correctly positioning the start codon.
        *   **tRNA Binding Sites:** The ribosome provides three binding sites for tRNA molecules:
            *   **A (Aminoacyl) site:** Where incoming aminoacyl-tRNAs bind.
            *   **P (Peptidyl) site:** Holds the tRNA carrying the growing polypeptide chain.
            *   **E (Exit) site:** Where deacylated (uncharged) tRNAs exit the ribosome.
        *   **Peptide Bond Formation:** The large subunit contains the peptidyl transferase activity (catalyzed by rRNA), which forms peptide bonds between amino acids, linking them into a polypeptide chain.
        *   **Translocation:** The ribosome moves along the mRNA, facilitating the sequential movement of tRNAs through the A, P, and E sites, ensuring the elongation of the polypeptide.
        *   **Termination:** Ribosomes recognize stop codons, leading to the release of the completed protein and dissociation of the ribosomal subunits.

36. **Describe the function and importance of peptidyl transferase activity in ribosomes.**
    *   **Function:** Peptidyl transferase is the enzymatic activity within the large ribosomal subunit that catalyzes the formation of a peptide bond. Specifically, it forms a covalent bond between the carboxyl group of the amino acid (or polypeptide chain) attached to the tRNA in the P-site and the amino group of the amino acid attached to the tRNA in the A-site.
    *   **Importance:** This activity is absolutely crucial for protein synthesis because it is responsible for linking individual amino acids together to form the polypeptide chain. Without peptidyl transferase, the amino acids would not be able to polymerize, and functional proteins could not be synthesized. Interestingly, this catalytic activity is not carried out by a protein, but by the ribosomal RNA (rRNA) itself, making the ribosome a **ribozyme**. This highlights the catalytic potential of RNA molecules.

37. **Explain the function of promoter, operator, and structural genes in an operon.**
    *   **Promoter:** A specific DNA sequence located upstream of the genes it controls. Its function is to serve as the binding site for RNA polymerase, which initiates the process of transcription. Without a functional promoter, RNA polymerase cannot bind, and the genes cannot be transcribed.
    *   **Operator:** A short DNA segment located within or adjacent to the promoter. Its function is to act as a binding site for a repressor protein. When the repressor binds to the operator, it typically blocks the movement of RNA polymerase, thereby preventing the transcription of the structural genes. This is a key regulatory element in operons.
    *   **Structural Genes:** These are the genes within an operon that code for the actual proteins (enzymes or other functional proteins) involved in a specific metabolic pathway or cellular process. Their function is to be transcribed into mRNA and subsequently translated into proteins that carry out the operon's biological function. For example, in the lac operon, *lacZ*, *lacY*, and *lacA* are structural genes.

38. **Discuss the steps involved in translation termination.**
    *   Translation termination is the final stage of protein synthesis, where the synthesis of the polypeptide chain ceases, and the ribosomal complex disassembles. The steps are:
        1.  **Stop Codon Recognition:** Elongation continues until one of the three stop codons (UAA, UAG, or UGA) on the mRNA enters the A-site of the ribosome. Unlike sense codons, there are no tRNAs with anticodons complementary to stop codons.
        2.  **Release Factor Binding:** Instead of a tRNA, specific protein molecules called **release factors** (RFs) recognize and bind to the stop codon in the A-site. In prokaryotes, RF1 recognizes UAA and UAG, while RF2 recognizes UAA and UGA. In eukaryotes, eRF1 recognizes all three stop codons.
        3.  **Polypeptide Release:** The binding of release factors to the A-site alters the peptidyl transferase activity of the ribosome. This causes the hydrolysis of the bond between the completed polypeptide chain and the tRNA in the P-site. As a result, the newly synthesized polypeptide is released from the ribosome.
        4.  **Ribosome Disassembly:** Following polypeptide release, other factors (e.g., ribosome recycling factor, RF3, IF3 in prokaryotes) facilitate the dissociation of the ribosomal subunits from the mRNA and from each other, making them available for new rounds of translation.

39. **Describe DNA packaging in eukaryotic cells with a labeled diagram.**
    *   In eukaryotic cells, the vast amount of DNA (which can be meters long) must be highly condensed to fit within the micrometer-sized nucleus. This packaging involves several levels of coiling and folding:
        1.  **Nucleosomes (Beads-on-a-string):** The first level of packaging involves DNA wrapping around histone proteins. Approximately 146 base pairs of DNA wrap nearly twice around a core of eight histone proteins (two each of H2A, H2B, H3, and H4), forming a nucleosome. These nucleosomes are connected by linker DNA, giving a "beads-on-a-string" appearance.
        2.  **30-nm Chromatin Fiber:** Nucleosomes are further coiled and compacted into a 30-nanometer (nm) fiber. This involves the H1 histone, which helps to pull the nucleosomes together, and further interactions between nucleosomes.
        3.  **Looped Domains:** The 30-nm fiber then forms large loops, called looped domains, which are anchored to a protein scaffold within the nucleus.
        4.  **Metaphase Chromosome:** During cell division (mitosis), these looped domains are further coiled and condensed to form the highly compact and visible metaphase chromosomes. This is the most condensed form of DNA packaging.
    *   *(A labeled diagram would typically show DNA double helix -> nucleosomes -> 30-nm fiber -> looped domains -> metaphase chromosome, with histones and linker DNA indicated.)*

40. **Explain the concept of chromatin remodeling and its biological significance.**
    *   **Chromatin Remodeling:** This refers to the dynamic modification of chromatin structure to allow or restrict access of regulatory proteins (like transcription factors) to the underlying DNA. It involves ATP-dependent chromatin remodeling complexes that can slide, eject, or restructure nucleosomes, thereby changing the accessibility of DNA.
    *   **Biological Significance:** Chromatin remodeling is crucial for regulating gene expression.
        *   **Gene Activation:** By moving or displacing nucleosomes, remodeling complexes can expose promoter regions or other regulatory sequences, making them accessible to RNA polymerase and transcription factors, thus activating gene transcription.
        *   **Gene Repression:** Conversely, remodeling can lead to the repositioning of nucleosomes to cover regulatory sequences, making them inaccessible and repressing gene expression.
        *   **DNA Replication and Repair:** Chromatin remodeling is also important for allowing access to DNA during replication and repair processes.
    *   It's a key mechanism by which cells control which genes are expressed at specific times and in specific cell types, contributing to cell differentiation and development.

41. **Discuss the methods and ethical issues in DNA fingerprinting.**
    *   **Methods (Steps):**
        1.  **Isolation of DNA:** DNA is extracted from a biological sample (e.g., blood, saliva, hair, semen).
        2.  **DNA Amplification (PCR):** If the DNA sample is small, specific regions (like VNTRs or STRs) are amplified using Polymerase Chain Reaction (PCR).
        3.  **Restriction Digestion (Traditional RFLP):** DNA is cut into fragments using restriction enzymes (less common now).
        4.  **Gel Electrophoresis:** DNA fragments are separated by size on an agarose gel. Smaller fragments move faster and further.
        5.  **Southern Blotting:** The separated DNA fragments are transferred from the gel to a nylon membrane.
        6.  **Hybridization:** The membrane is incubated with labeled DNA probes (complementary to VNTRs or STRs), which bind to specific fragments.
        7.  **Autoradiography/Detection:** The labeled fragments are visualized (e.g., using X-ray film for radioactive labels or fluorescent detectors), producing a unique banding pattern or peak profile – the DNA fingerprint.
    *   **Ethical Issues:**
        *   **Privacy Concerns:** The storage and use of DNA profiles raise significant privacy concerns. Who has access to this highly personal information, and how will it be protected from misuse?
        *   **Discrimination:** There is a risk of genetic discrimination in areas like employment, insurance, or even social interactions if genetic predispositions to diseases or traits become widely known.
        *   **Misinterpretation of Results:** DNA evidence, while powerful, can be misinterpreted or mishandled, leading to wrongful convictions or accusations. Statistical probabilities associated with matches need to be clearly communicated.
        *   **Consent Issues:** Obtaining informed consent for DNA collection and analysis, especially in forensic contexts or for minors, can be complex.
        *   **Familial Searching:** The practice of searching DNA databases for partial matches to identify relatives of a suspect raises ethical questions about privacy for individuals who have not committed a crime.

42. **Write a short note on the Human Genome Project—goals, methods, and benefits.**
    *   **Human Genome Project (HGP):** An ambitious international scientific research project launched in 1990 and completed in 2003, with the primary goal of mapping and sequencing the entire human genome.
    *   **Goals:**
        *   Determine the complete sequence of the approximately 3 billion chemical base pairs that make up human DNA.
        *   Identify all the estimated 20,000-25,000 genes in human DNA.
        *   Store this information in publicly accessible databases.
        *   Develop tools for data analysis.
        *   Address the ethical, legal, and social issues (ELSI) arising from the project.
    *   **Methods:**
        *   **Clone-by-clone approach:** Breaking the genome into manageable pieces, mapping them, and then sequencing each piece.
        *   **Whole-genome shotgun sequencing:** Fragmenting the entire genome randomly, sequencing the fragments, and then assembling them using computational methods.
        *   **Expressed Sequence Tags (ESTs):** Identifying genes that are actively expressed by sequencing their mRNA transcripts.
        *   **Sequence Annotation:** Assigning functions to different regions of the sequenced genome (coding and non-coding).
    *   **Benefits (Applications):**
        *   Revolutionized biomedical research, leading to a deeper understanding of human biology and disease.
        *   Improved diagnosis, prevention, and treatment of genetic disorders.
        *   Facilitated the development of gene therapy and personalized medicine (pharmacogenomics).
        *   Advanced forensic science and anthropology.
        *   Spurred the growth of the biotechnology industry and bioinformatics.

43. **What are the applications of HGP in medicine and forensics?**
    *   **Applications in Medicine:**
        *   **Disease Diagnosis and Prevention:** Identifying genetic predispositions to diseases (e.g., cancer, Alzheimer's) and developing early diagnostic tests.
        *   **Personalized Medicine (Pharmacogenomics):** Tailoring drug treatments based on an individual's genetic makeup to optimize efficacy and minimize side effects.
        *   **Gene Therapy:** Developing therapies to correct defective genes responsible for genetic disorders.
        *   **Drug Discovery:** Identifying new drug targets and designing more effective pharmaceuticals.
        *   **Understanding Disease Mechanisms:** Elucidating the molecular basis of complex diseases.
    *   **Applications in Forensics:**
        *   **Identification of Criminals:** Matching DNA samples from crime scenes to suspects or databases.
        *   **Paternity Testing:** Determining biological parentage.
        *   **Identification of Human Remains:** Identifying victims in mass disasters or historical cases.
        *   **Wildlife Forensics:** Tracking illegal poaching and trade of endangered species.

44. **Explain the structure of DNA with reference to hydrogen bonding and base pairing.**
    *   The DNA molecule is a double helix, consisting of two polynucleotide strands coiled around a central axis. The backbone of each strand is formed by alternating deoxyribose sugar and phosphate groups.
    *   The nitrogenous bases (Adenine, Guanine, Cytosine, Thymine) project inwards from the sugar-phosphate backbone.
    *   The two strands are held together by **hydrogen bonds** formed between complementary base pairs:
        *   **Adenine (A)** always pairs with **Thymine (T)** via **two hydrogen bonds**.
        *   **Guanine (G)** always pairs with **Cytosine (C)** via **three hydrogen bonds**.
    *   This specific **base pairing** (A-T and G-C) is known as Chargaff's rules and is fundamental to the stability of the double helix, ensuring that the two strands are precisely complementary. The hydrogen bonds, though individually weak, collectively provide significant stability to the DNA molecule.

45. **Write a note on transcription factors and enhancers.**
    *   **Transcription Factors:** These are proteins that bind to specific DNA sequences (or to other proteins) to regulate the rate of gene transcription. They can be broadly categorized into:
        *   **General (Basal) Transcription Factors:** Essential for the transcription of all protein-coding genes, forming the pre-initiation complex at the promoter with RNA polymerase II.
        *   **Specific (Regulatory) Transcription Factors:** Bind to specific DNA sequences (e.g., enhancers, silencers) to either activate (activators) or repress (repressors) the transcription of particular genes. They play a crucial role in tissue-specific and developmental gene expression.
    *   **Enhancers:** These are DNA sequences that can be located far from the gene they regulate (upstream, downstream, or even within an intron). They function by binding specific activator transcription factors. When these activators bind to the enhancer, they can interact with the general transcription factors and RNA polymerase at the promoter, often by forming a DNA loop, to significantly increase the rate of transcription. Enhancers are key elements in the complex regulation of gene expression in eukaryotes.

46. **What is gene expression? Describe the regulation in eukaryotic cells.**
    *   **Gene Expression:** The process by which information encoded in a gene is used to synthesize a functional gene product, such as a protein or an RNA molecule (e.g., tRNA, rRNA). It involves transcription (DNA to RNA) and, for protein-coding genes, translation (RNA to protein).
    *   **Regulation in Eukaryotic Cells:** Gene expression in eukaryotes is highly complex and can be regulated at multiple levels:
        1.  **Chromatin Structure:** The packaging of DNA into chromatin influences gene accessibility. Euchromatin (loosely packed) is generally active, while heterochromatin (densely packed) is repressed. **Chromatin remodeling** (e.g., nucleosome sliding) and **histone modifications** (e.g., acetylation, methylation) can alter chromatin structure to expose or hide genes.
        2.  **Transcriptional Control:** This is a primary level of control. It involves:
            *   **Transcription Factors:** Proteins that bind to DNA (promoters, enhancers) to activate or repress RNA polymerase activity.
            *   **Enhancers and Silencers:** Distant DNA sequences that modulate transcription rates.
            *   **DNA Methylation:** Addition of methyl groups to cytosine bases, often leading to gene silencing.
        3.  **Post-transcriptional Control:**
            *   **RNA Processing:** Alternative splicing can produce different mRNA isoforms from a single gene, leading to protein diversity. Capping and polyadenylation affect mRNA stability and translation.
            *   **mRNA Stability:** The lifespan of an mRNA molecule in the cytoplasm can be regulated, affecting how much protein is produced.
            *   **RNA Interference (RNAi):** Small non-coding RNAs (miRNAs, siRNAs) can bind to mRNA and lead to its degradation or inhibition of translation.
        4.  **Translational Control:** Regulation of the rate at which mRNA is translated into protein (e.g., by regulatory proteins binding to mRNA).
        5.  **Post-translational Control:** Modifications to the protein after synthesis (e.g., phosphorylation, glycosylation, cleavage) that affect its activity, stability, or localization.

47. **Differentiate between structural and regulatory genes.**
    *   **Structural Genes:**
        *   **Function:** These genes code for proteins (or functional RNA molecules like tRNA, rRNA) that are directly involved in cellular structure or metabolic processes. They are the "workhorse" genes that produce the enzymes, transport proteins, structural components, etc., needed for cell function.
        *   **Expression:** Their expression is often regulated by regulatory genes.
        *   **Example:** In the lac operon, *lacZ*, *lacY*, and *lacA* are structural genes that code for enzymes involved in lactose metabolism.
    *   **Regulatory Genes:**
        *   **Function:** These genes code for regulatory proteins (or regulatory RNA molecules) that control the expression of other genes. They do not directly participate in metabolic pathways but rather act as "switches" to turn other genes on or off.
        *   **Product:** Their products are typically transcription factors, repressors, activators, or other signaling molecules.
        *   **Example:** In the lac operon, the *i* gene is a regulatory gene that codes for the lac repressor protein, which controls the transcription of the *lac* structural genes.

48. **Explain the importance of histone modifications in gene regulation.**
    *   Histone modifications are crucial for gene regulation because they alter the structure of chromatin, thereby influencing the accessibility of DNA to the transcriptional machinery. Histones can be modified by the addition or removal of various chemical groups (e.g., acetyl, methyl, phosphate, ubiquitin) to their N-terminal tails.
    *   **Mechanism:**
        *   **Acetylation (e.g., by HATs):** Adding acetyl groups to lysine residues on histones neutralizes their positive charge, weakening their interaction with the negatively charged DNA. This leads to a more open, relaxed chromatin structure (euchromatin), making the DNA more accessible for transcription and thus promoting gene expression.
        *   **Deacetylation (e.g., by HDACs):** Removing acetyl groups restores the positive charge, strengthening DNA-histone interactions, leading to a more condensed chromatin structure (heterochromatin) and gene repression.
        *   **Methylation:** Can have varied effects depending on the specific histone residue and the number of methyl groups. Methylation of H3K4 is often associated with active transcription, while methylation of H3K9 or H3K27 is associated with gene silencing.
    *   **Importance:** These modifications create a "histone code" that is read by other proteins, leading to specific downstream effects on gene expression. They are fundamental to processes like cell differentiation, development, and responses to environmental stimuli, allowing cells to precisely control which genes are turned on or off.

49. **Describe the three sites of a ribosome and their roles during protein synthesis.**
    *   Ribosomes have three distinct binding sites for tRNA molecules during protein synthesis:
        1.  **A (Aminoacyl) site:** This is the entry site for incoming aminoacyl-tRNAs (tRNAs carrying a new amino acid). During elongation, the tRNA with the next amino acid specified by the mRNA codon binds to the A-site.
        2.  **P (Peptidyl) site:** This site holds the tRNA that is attached to the growing polypeptide chain. After the peptide bond is formed, the polypeptide chain is transferred from the A-site tRNA to the P-site tRNA.
        3.  **E (Exit) site:** This is the site from which deacylated (uncharged) tRNAs, which have already delivered their amino acid, exit the ribosome. This site is transiently occupied before the tRNA is released to be recharged with another amino acid.
    *   The sequential movement of tRNAs through these sites (A → P → E) ensures the orderly addition of amino acids to the polypeptide chain according to the mRNA sequence.

50. **Write a note on the wobble hypothesis and its role in codon recognition.**
    *   **Wobble Hypothesis:** Proposed by Francis Crick, the wobble hypothesis explains why a single tRNA molecule can recognize more than one codon, and why there are fewer tRNA molecules than there are codons. It states that the base pairing between the third nucleotide of the mRNA codon and the first nucleotide of the tRNA anticodon is less stringent (more flexible or "wobble") than the first two base pairings.
    *   **Role in Codon Recognition:** This flexibility at the wobble position allows for non-Watson-Crick base pairings (e.g., G with U, or Inosine with U, C, or A). This means that a single tRNA can "wobble" to recognize multiple codons that differ only in their third base. For example, a tRNA with the anticodon 3'-GGU-5' can recognize both 5'-CCA-3' and 5'-CCG-3' mRNA codons (both coding for Proline).
    *   **Significance:** The wobble hypothesis increases the efficiency of protein synthesis by reducing the number of different tRNA molecules required in the cell, while still ensuring accurate translation of the genetic code. It also contributes to the degeneracy of the genetic code.

51. **Explain how gene expression is controlled at the transcriptional level.**
    *   Transcriptional control is the most common and often the most important level of gene regulation, determining whether and how much a gene is transcribed into RNA.
    *   **In Prokaryotes (e.g., Operons):**
        *   **Repressors:** Proteins that bind to operator sequences, blocking RNA polymerase from initiating transcription (e.g., lac repressor).
        *   **Activators:** Proteins that bind to specific DNA sites, enhancing RNA polymerase binding and transcription.
        *   **Inducers/Corepressors:** Small molecules that bind to repressors or activators, altering their DNA-binding affinity and thus regulating transcription (e.g., allolactose in lac operon).
    *   **In Eukaryotes:** Transcriptional control is more complex:
        *   **Chromatin Structure:** The accessibility of DNA is regulated by chromatin remodeling and histone modifications. Condensed chromatin (heterochromatin) generally represses transcription, while open chromatin (euchromatin) promotes it.
        *   **Promoters:** Core promoters (e.g., TATA box) are binding sites for general transcription factors and RNA polymerase.
        *   **Enhancers and Silencers:** Distant DNA sequences that bind specific activator or repressor proteins, respectively, to modulate transcription rates. These can be thousands of base pairs away from the gene.
        *   **Transcription Factors:** A vast array of proteins that bind to specific DNA sequences (or to other transcription factors) to either activate or repress transcription. They integrate signals from various pathways to fine-tune gene expression.
        *   **DNA Methylation:** Addition of methyl groups to cytosine bases in CpG islands often leads to stable gene silencing by altering chromatin structure or blocking transcription factor binding.

52. **Describe the experimental setup and results of Griffith’s experiment.**
    *   **Experimental Setup:** Frederick Griffith (1928) worked with two strains of *Streptococcus pneumoniae* bacteria:
        *   **S strain (smooth):** Virulent (disease-causing), with a polysaccharide capsule.
        *   **R strain (rough):** Non-virulent (non-disease-causing), lacking a capsule.
    *   He performed four experiments involving injecting these bacteria into mice:
        1.  **Live R strain injected into mice:**
        2.  **Live S strain injected into mice:**
        3.  **Heat-killed S strain injected into mice:**
        4.  **Heat-killed S strain mixed with live R strain, then injected into mice:**
    *   **Results:**
        1.  **Live R strain:** Mice lived. (No disease)
        2.  **Live S strain:** Mice died. (Developed pneumonia)
        3.  **Heat-killed S strain:** Mice lived. (No disease, bacteria were dead)
        4.  **Heat-killed S strain + Live R strain:** Mice died. (Developed pneumonia, and live S strain bacteria were recovered from the dead mice).
    *   **Conclusion:** Griffith concluded that some "transforming principle" from the heat-killed S strain had transformed the live R strain into virulent S strain bacteria. He did not identify the chemical nature of this principle, but his work laid the foundation for the discovery of DNA as the genetic material.

53. **Compare the roles of helicase, topoisomerase, and ligase in DNA replication.**
    *   **Helicase:**
        *   **Role:** Unwinds the DNA double helix at the replication fork by breaking the hydrogen bonds between complementary base pairs. This separates the two strands, making them available as templates.
        *   **Analogy:** Like a zipper unzipping the DNA.
    *   **Topoisomerase (e.g., DNA gyrase in prokaryotes):**
        *   **Role:** Relieves the torsional stress (supercoiling) that builds up in the DNA ahead of the replication fork as helicase unwinds the helix. It does this by cutting, unwinding, and rejoining DNA strands.
        *   **Analogy:** Like untangling a twisted rope ahead of the unzipping.
    *   **Ligase (DNA Ligase):**
        *   **Role:** Joins DNA fragments together by forming phosphodiester bonds. Its primary role in replication is to seal the nicks between Okazaki fragments on the lagging strand, creating a continuous DNA strand.
        *   **Analogy:** Like a molecular glue that connects DNA pieces.

54. **Describe how the template strand guides mRNA synthesis.**
    *   During transcription, one of the two DNA strands serves as the **template strand** (also known as the antisense or non-coding strand) for mRNA synthesis.
    *   RNA polymerase moves along this template strand in the 3'→5' direction.
    *   The synthesis of the mRNA molecule occurs in the 5'→3' direction.
    *   The template strand guides mRNA synthesis through the principle of **complementary base pairing**. RNA polymerase adds ribonucleotides to the growing mRNA chain according to the following rules:
        *   If the template DNA has an Adenine (A), a Uracil (U) is added to the mRNA.
        *   If the template DNA has a Thymine (T), an Adenine (A) is added to the mRNA.
        *   If the template DNA has a Guanine (G), a Cytosine (C) is added to the mRNA.
        *   If the template DNA has a Cytosine (C), a Guanine (G) is added to the mRNA.
    *   This ensures that the mRNA sequence is a faithful, complementary copy of the template DNA strand (with U replacing T), carrying the genetic information for protein synthesis.

55. **Explain how RNA processing increases the efficiency of gene expression.**
    *   RNA processing, particularly in eukaryotes, significantly increases the efficiency and complexity of gene expression in several ways:
        1.  **Stability and Protection (Capping and Tailing):** The 5' cap and 3' poly-A tail added to mRNA protect it from degradation by exonucleases, increasing its lifespan in the cytoplasm. A more stable mRNA molecule can be translated multiple times, leading to the production of more protein from a single transcript.
        2.  **Nuclear Export:** The 5' cap and poly-A tail are recognized by transport proteins that facilitate the efficient export of mature mRNA from the nucleus to the cytoplasm, where translation occurs.
        3.  **Translation Initiation:** The 5' cap is crucial for the efficient binding of ribosomes and initiation factors, ensuring that translation begins correctly.
        4.  **Alternative Splicing:** This process allows different combinations of exons from a single pre-mRNA to be joined together, producing multiple distinct mRNA molecules. Each mRNA can then be translated into a different protein isoform. This dramatically increases the protein diversity that can be generated from a limited number of genes, enhancing the functional repertoire of the cell without increasing genome size.
        5.  **Quality Control:** Splicing helps ensure that only functional, intron-free mRNA molecules are translated, preventing the production of truncated or non-functional proteins.

56. **What are VNTRs and how are they used in forensic science?**
    *   **VNTRs (Variable Number Tandem Repeats):** These are short nucleotide sequences (typically 10-100 base pairs long) that are repeated in tandem (one after another) at specific locations in the genome. The number of times these sequences are repeated varies significantly among individuals, making them highly polymorphic.
    *   **Use in Forensic Science:** VNTRs are a cornerstone of traditional DNA fingerprinting (RFLP-based) and are still conceptually relevant for modern methods using STRs (Short Tandem Repeats, which are shorter versions of VNTRs).
        *   **Individual Identification:** Because the number of repeats at various VNTR loci is unique to almost every individual (except identical twins), VNTR analysis creates a distinctive DNA profile.
        *   **Crime Scene Analysis:** DNA samples from a crime scene (e.g., blood, semen, hair) can be analyzed for their VNTR patterns and compared to those of suspects. A match provides strong evidence linking a suspect to the crime.
        *   **Paternity Testing:** By comparing the VNTR patterns of a child, mother, and alleged father, it can be determined if the alleged father is the biological father, as the child inherits half of its VNTR alleles from each parent.
        *   **Identification of Human Remains:** VNTR analysis can be used to identify deceased individuals by comparing their DNA to that of known relatives.

57. **Describe the principle and procedure of gel electrophoresis.**
    *   **Principle:** Gel electrophoresis is a laboratory technique used to separate macromolecules (like DNA, RNA, or proteins) based on their size, charge, and shape. The principle relies on the fact that charged molecules will migrate through a porous gel matrix when an electric field is applied. Nucleic acids (DNA and RNA) are negatively charged due to their phosphate backbone, so they migrate towards the positive electrode. Smaller molecules move more easily through the gel pores and thus travel faster and further than larger molecules.
    *   **Procedure (for DNA):**
        1.  **Gel Preparation:** An agarose gel (a porous matrix) is prepared and cast with wells at one end.
        2.  **Sample Loading:** DNA samples (often pre-cut with restriction enzymes or amplified by PCR) are loaded into the wells. A loading dye is added to track migration.
        3.  **Electrophoresis:** The gel is placed in a buffer solution, and an electric current is applied across the gel. The negative electrode is placed near the wells, and the positive electrode at the opposite end.
        4.  **Migration:** DNA fragments migrate through the gel towards the positive electrode. Smaller fragments move faster and further down the gel.
        5.  **Visualization:** After electrophoresis, the gel is stained with a DNA-binding dye (e.g., ethidium bromide, SYBR Green) that fluoresces under UV light. This allows the separated DNA bands to be visualized and photographed. A DNA ladder (fragments of known sizes) is run alongside the samples to estimate the size of the unknown fragments.

58. **Explain how DNA fingerprinting is used to determine paternity.**
    *   DNA fingerprinting (or DNA profiling) is a highly accurate method for determining paternity based on the principle that a child inherits half of its genetic material from its biological mother and half from its biological father.
    *   **Procedure:**
        1.  **DNA Collection:** DNA samples are collected from the child, the biological mother, and the alleged father(s).
        2.  **DNA Analysis:** Specific regions of the DNA, typically highly variable short tandem repeats (STRs), are amplified using PCR. These STRs are polymorphic, meaning the number of repeats varies greatly among individuals.
        3.  **Comparison of Profiles:** The DNA profiles (patterns of STR alleles) of the child, mother, and alleged father are compared.
        *   Every STR allele in the child's profile must be accounted for by either the mother or the father.
        *   If an allele in the child's profile is not present in the mother's profile, it *must* have been inherited from the biological father.
        *   If all of the child's paternal alleles match those of the alleged father, then paternity is confirmed with a very high probability.
        *   If there are mismatches (i.e., the child has paternal alleles not found in the alleged father), then the alleged father is excluded as the biological father.

59. **What is gene therapy? Explain with examples.**
    *   **Gene Therapy:** Gene therapy is a medical approach that involves modifying a person's genes to treat or cure a disease. It typically works by introducing a new, functional gene into a patient's cells to replace a mutated or missing gene, or to provide a new function that helps fight disease.
    *   **Mechanism:** Genes are usually delivered into cells using a "vector," most commonly a modified virus (e.g., adeno-associated virus, lentivirus) that has been engineered to carry the therapeutic gene but is rendered harmless. The vector delivers the gene into the target cells, where it can then be expressed to produce the desired protein or RNA.
    *   **Examples:**
        *   **SCID (Severe Combined Immunodeficiency):** One of the earliest successes. Gene therapy has been used to introduce a functional gene (e.g., ADA gene) into the immune cells of children with SCID, allowing them to develop a functional immune system.
        *   **Cystic Fibrosis:** Research is ongoing to deliver a functional CFTR gene to the lung cells of patients to correct the defective protein responsible for the disease.
        *   **Cancer Treatment:** Gene therapy is being explored to enhance the body's immune response against cancer cells (e.g., CAR T-cell therapy) or to introduce genes that make cancer cells more susceptible to chemotherapy.
        *   **Leber Congenital Amaurosis (LCA):** A form of inherited blindness. Gene therapy (e.g., Luxturna) has been approved to deliver a functional RPE65 gene to retinal cells, restoring vision in some patients.

60. **Write a note on the role of bioinformatics in genome annotation.**
    *   **Genome Annotation:** The process of identifying the locations of genes and all of the coding regions in a genome and determining what those genes do. It involves attaching biological information to sequences.
    *   **Role of Bioinformatics:** Bioinformatics plays an indispensable and central role in genome annotation. It provides the computational tools, algorithms, and databases necessary to analyze the vast amounts of raw sequencing data and extract meaningful biological information.
        *   **Sequence Assembly:** Bioinformatics algorithms are used to assemble short DNA reads into contiguous sequences (contigs) and then into complete chromosomes.
        *   **Gene Prediction:** Algorithms identify open reading frames (ORFs), splice sites, promoters, and other regulatory elements to predict the location of protein-coding genes. This often involves comparing sequences to known genes in databases.
        *   **Functional Annotation:** Once genes are identified, bioinformatics tools are used to infer their function. This involves:
            *   **Homology Searches:** Comparing newly identified gene sequences to known genes in public databases (e.g., GenBank, UniProt) using tools like BLAST to find homologous genes with known functions.
            *   **Domain Prediction:** Identifying conserved protein domains or motifs that suggest specific functions.
            *   **Pathway Analysis:** Placing genes within known metabolic or signaling pathways.
        *   **Non-coding RNA Identification:** Bioinformatics tools are also used to identify non-coding RNA genes (tRNA, rRNA, miRNA, etc.) and other regulatory sequences.
        *   **Data Management and Visualization:** Bioinformatics databases store and organize annotated genome data, and visualization tools allow researchers to explore and interpret the complex genomic information.

61. **Explain the difference between exons and introns and their fate during RNA splicing.**
    *   **Exons:**
        *   **Definition:** Exons are the coding regions of a gene that contain the genetic information that will be translated into a protein. They are the "expressed" sequences.
        *   **Fate during Splicing:** During RNA splicing, exons are retained and precisely ligated (joined) together to form the mature messenger RNA (mRNA) molecule.
    *   **Introns:**
        *   **Definition:** Introns are the non-coding, intervening sequences within a gene that are transcribed into pre-mRNA but are not translated into protein. They are the "intervening" sequences.
        *   **Fate during Splicing:** During RNA splicing, introns are precisely removed (excised) from the pre-mRNA molecule.
    *   **RNA Splicing:** This is a crucial post-transcriptional modification process in eukaryotes. It is carried out by a complex molecular machine called the spliceosome (composed of snRNAs and proteins). The spliceosome recognizes specific sequences at the exon-intron boundaries, cuts out the introns, and ligates the adjacent exons, producing a continuous coding sequence in the mature mRNA. This process ensures that only the protein-coding information is present in the final mRNA that is exported to the cytoplasm for translation.

62. **Describe any three major tools required in recombinant DNA technology.**
    *   **Recombinant DNA technology** involves combining DNA from different sources to create new genetic combinations. Key tools include:
        1.  **Restriction Enzymes (Restriction Endonucleases):** These are enzymes that recognize and cut DNA at specific nucleotide sequences (recognition sites). They act like "molecular scissors," allowing scientists to precisely cut DNA at desired locations. Different restriction enzymes recognize different sequences, producing either "sticky ends" (overhangs) or "blunt ends," which are crucial for joining DNA fragments.
        2.  **DNA Ligase:** This enzyme acts as "molecular glue." It catalyzes the formation of phosphodiester bonds between the sugar-phosphate backbones of DNA fragments, effectively joining them together. After restriction enzymes cut DNA, DNA ligase is used to insert a desired gene into a vector (e.g., a plasmid) by joining the cut ends.
        3.  **Vectors (e.g., Plasmids, Viruses):** Vectors are DNA molecules (often plasmids from bacteria or modified viruses) that can carry foreign DNA into a host cell and allow it to be replicated. They act as "delivery vehicles" for genes. Plasmids are commonly used because they are small, circular, can replicate independently, and often contain selectable markers (e.g., antibiotic resistance genes) to identify cells that have taken up the recombinant DNA. Viruses can also be engineered to deliver genes into eukaryotic cells.

63. **Explain the steps of Southern blotting.**
    *   Southern blotting is a molecular biology technique used to detect specific DNA sequences in a DNA sample. It involves:
        1.  **DNA Isolation and Restriction Digestion:** DNA is extracted from cells and then cut into smaller fragments using restriction enzymes.
        2.  **Gel Electrophoresis:** The DNA fragments are separated by size using agarose gel electrophoresis. The fragments migrate through the gel, with smaller fragments moving faster and further.
        3.  **Denaturation:** The double-stranded DNA fragments in the gel are denatured (separated into single strands) by treating the gel with an alkaline solution. This is necessary for the probe to bind.
        4.  **Blotting (Transfer):** The single-stranded DNA fragments are transferred from the fragile gel to a more stable solid support membrane, typically a nylon or nitrocellulose membrane. This is usually done by capillary action, where buffer draws the DNA from the gel onto the membrane.
        5.  **Hybridization:** The membrane is incubated with a labeled **probe**. The probe is a single-stranded DNA (or RNA) molecule with a sequence complementary to the target DNA sequence of interest. The probe is labeled with a radioactive isotope or a fluorescent tag. The probe will bind (hybridize) only to the complementary DNA fragments on the membrane.
        6.  **Washing:** The membrane is washed to remove any unbound or non-specifically bound probes.
        7.  **Detection (Autoradiography/Chemiluminescence):** The hybridized probe is detected. If a radioactive probe was used, the membrane is exposed to X-ray film (autoradiography), which reveals bands where the probe hybridized. If a fluorescent or chemiluminescent probe was used, a detector captures the signal. The resulting pattern of bands indicates the presence and size of the target DNA sequence.

64. **Discuss the role of small nuclear RNA in mRNA processing.**
    *   Small nuclear RNAs (snRNAs) play a critical and central role in the processing of messenger RNA (mRNA) in eukaryotic cells, specifically in the process of **splicing**.
    *   **Splicing:** This is the process where non-coding intron sequences are removed from the primary RNA transcript (hnRNA or pre-mRNA), and the coding exon sequences are precisely ligated together to form mature mRNA.
    *   **Role of snRNAs:** snRNAs (e.g., U1, U2, U4, U5, U6) associate with specific proteins to form small nuclear ribonucleoproteins (snRNPs, often pronounced "snurps"). These snRNPs are the key components of the **spliceosome**, a large and dynamic molecular machine responsible for splicing.
        *   **Recognition of Splice Sites:** snRNAs within the spliceosome recognize and bind to specific consensus sequences at the 5' and 3' ends of introns, as well as an internal branch point sequence. This recognition occurs through complementary base pairing between the snRNA and the pre-mRNA.
        *   **Catalysis:** The snRNAs also have catalytic activity (acting as ribozymes) within the spliceosome, facilitating the two transesterification reactions that precisely cut out the intron and join the exons.
        *   **Spliceosome Assembly and Rearrangement:** snRNAs guide the assembly and conformational changes of the spliceosome, ensuring that the splicing reactions occur accurately and efficiently.
    *   Without snRNAs, proper splicing would not occur, leading to the production of non-functional proteins from eukaryotic genes.

65. **What are the molecular differences between DNA replication and transcription?**
    | Feature | DNA Replication | Transcription |
    |---|---|---|
    | **Purpose** | To make an identical copy of the entire DNA genome for cell division. | To synthesize an RNA molecule from a specific gene (DNA template). |
    | **Template** | Both strands of the DNA double helix serve as templates. | Only one strand of the DNA (the template strand) serves as a template. |
    | **Product** | A new double-stranded DNA molecule. | A single-stranded RNA molecule (mRNA, tRNA, rRNA, etc.). |
    | **Enzyme** | DNA Polymerase (and many other accessory enzymes like helicase, ligase, primase). | RNA Polymerase. |
    | **Nucleotides Used** | Deoxyribonucleotides (dATP, dCTP, dGTP, dTTP). | Ribonucleotides (ATP, CTP, GTP, UTP). |
    | **Base Pairing** | A-T, G-C (in new DNA strand). | A-U, T-A, G-C, C-G (in RNA strand, U replaces T). |
    | **Extent** | Replicates the entire genome (or large sections). | Transcribes specific genes or operons. |
    | **Primers** | Requires RNA primers to initiate synthesis. | Does not require a primer; RNA polymerase can initiate synthesis *de novo*. |
    | **Proofreading** | High proofreading activity (by DNA polymerase) for high fidelity. | Lower proofreading activity (by RNA polymerase), as RNA is transient. |

66. **How does RNA polymerase recognize the promoter and initiate transcription?**
    *   **Promoter Recognition:**
        *   **In Prokaryotes:** RNA polymerase, along with a sigma (σ) factor, forms a holoenzyme. The sigma factor is crucial for recognizing and binding to specific DNA sequences within the promoter region (e.g., the -10 box or Pribnow box, and the -35 box). This binding positions the RNA polymerase correctly at the transcription start site.
        *   **In Eukaryotes:** RNA polymerase (especially Pol II for mRNA) does not directly recognize the promoter. Instead, it relies on a set of **general transcription factors (GTFs)**. These GTFs bind to specific sequences in the core promoter (e.g., the TATA box) and recruit RNA polymerase to form a pre-initiation complex.
    *   **Initiation:**
        *   Once RNA polymerase is correctly positioned at the promoter, it unwinds a short segment of the DNA double helix, forming a transcription bubble.
        *   RNA polymerase then begins to synthesize a new RNA strand by adding complementary ribonucleotides to the template DNA strand, starting at the transcription start site.
        *   In prokaryotes, the sigma factor typically dissociates after initiation. In eukaryotes, the RNA polymerase undergoes phosphorylation of its C-terminal domain (CTD), which signals the transition from initiation to elongation.

67. **Write a note on the concept and functions of a gene.**
    *   **Concept of a Gene:** A gene is the fundamental unit of heredity in an organism. Classically, it is defined as a segment of DNA (or RNA in some viruses) that contains the instructions for making a specific functional product, which can be a protein or a functional RNA molecule (like tRNA or rRNA). Modern definitions acknowledge that genes include not only coding sequences but also regulatory sequences (promoters, enhancers) that control their expression.
    *   **Functions of a Gene:**
        1.  **Heredity:** Genes are passed from parents to offspring, transmitting genetic traits and characteristics across generations.
        2.  **Information Storage:** Genes store the blueprint for all cellular structures and activities in the sequence of their nucleotides.
        3.  **Protein Synthesis:** Protein-coding genes provide the instructions for synthesizing proteins, which perform a vast array of functions (enzymes, structural components, transport, signaling, etc.).
        4.  **RNA Production:** Some genes code directly for functional RNA molecules (e.g., ribosomal RNA, transfer RNA, small nuclear RNA, microRNA) that play crucial roles in gene expression and regulation.
        5.  **Regulation of Gene Expression:** Genes contain regulatory sequences that control when, where, and how much of a gene product is made, allowing cells to adapt to different conditions and specialize.
        6.  **Mutation and Evolution:** Genes can undergo mutations, which are changes in their nucleotide sequence. These mutations can lead to new traits and are the raw material for evolution by natural selection.

68. **Describe three types of point mutations and their effects.**
    *   A **point mutation** is a gene mutation where a single nucleotide base is changed, inserted, or deleted from a DNA or RNA sequence.
    *   1.  **Silent Mutation:**
        *   **Description:** A base substitution that changes a single nucleotide, but does not change the amino acid sequence of the protein. This is possible due to the degeneracy of the genetic code (multiple codons can code for the same amino acid).
        *   **Effect:** No observable effect on the protein's function or the organism's phenotype.
        *   **Example:** If a codon changes from GGU to GGC, both still code for Glycine.
    *   2.  **Missense Mutation:**
        *   **Description:** A base substitution that results in a codon that codes for a different amino acid.
        *   **Effect:** Can have varying effects. If the new amino acid is chemically similar or located in a non-critical region, the effect might be minor. However, if it's in a critical region or significantly changes the protein's properties, it can lead to a non-functional or altered protein.
        *   **Example:** Sickle cell anemia is caused by a missense mutation where a single base change in the beta-globin gene leads to a substitution of Valine for Glutamic acid, altering hemoglobin structure.
    *   3.  **Nonsense Mutation:**
        *   **Description:** A base substitution that changes a codon for an amino acid into a stop codon (UAA, UAG, or UGA).
        *   **Effect:** Results in premature termination of protein synthesis, leading to a truncated (shortened) and usually non-functional protein. This often has severe consequences for the cell or organism.
        *   **Example:** If a codon for Tryptophan (UGG) mutates to a stop codon (UGA), protein synthesis will stop prematurely.

69. **Explain the concept of genetic code degeneracy and silent mutations.**
    *   **Genetic Code Degeneracy:** The genetic code is said to be degenerate (or redundant) because most amino acids are specified by more than one codon. There are 64 possible codons (4³), but only 20 standard amino acids (plus stop signals). This means that multiple codons can code for the same amino acid. For example, both UUA and UUG code for Leucine, and all four codons starting with CC (CCA, CCC, CCG, CCU) code for Proline. This degeneracy often involves the third nucleotide of the codon (the "wobble" position).
    *   **Silent Mutations:** A silent mutation is a type of point mutation (a single base change in DNA) that, despite altering the DNA sequence, does not change the amino acid sequence of the protein. This occurs precisely because of the degeneracy of the genetic code. If a base substitution results in a new codon that still codes for the same amino acid as the original codon, the mutation is "silent" at the protein level.
    *   **Relationship:** Genetic code degeneracy is the underlying reason why silent mutations can occur. It provides a protective mechanism, meaning that not every single nucleotide change in a gene will necessarily lead to a change in the resulting protein, thus buffering the effects of random mutations.

70. **Compare and contrast DNA and chromosomal mutations.**
    *   **DNA Mutations (Gene Mutations):**
        *   **Definition:** Changes that occur within the nucleotide sequence of a single gene. They typically involve a small number of nucleotides.
        *   **Types:** Point mutations (substitutions, insertions, deletions of one or a few bases) and small insertions/deletions (indels).
        *   **Effect:** Primarily affect the sequence of a single gene's product (protein or RNA), potentially altering its function or expression.
        *   **Detection:** Often require DNA sequencing to detect.
        *   **Examples:** Sickle cell anemia (point mutation), cystic fibrosis (small deletion).
    *   **Chromosomal Mutations:**
        *   **Definition:** Large-scale changes that alter the structure or number of entire chromosomes. They involve segments of chromosomes or whole chromosomes.
        *   **Types:**
            *   **Structural Changes:** Deletions (loss of a segment), duplications (repetition of a segment), inversions (reversal of a segment), translocations (movement of a segment to a non-homologous chromosome).
            *   **Numerical Changes (Aneuploidy/Polyploidy):** Gain or loss of entire chromosomes (e.g., Trisomy 21 in Down syndrome) or entire sets of chromosomes.
        *   **Effect:** Often have significant and widespread effects on the organism's phenotype due to changes in gene dosage or disruption of many genes. Can be lethal or cause severe developmental disorders.
        *   **Detection:** Often detectable by karyotyping (microscopic examination of chromosomes).
        *   **Examples:** Down syndrome (Trisomy 21), Cri-du-chat syndrome (chromosomal deletion).
    *   **Contrast:** DNA mutations are small-scale changes within a gene, affecting its specific product. Chromosomal mutations are large-scale changes affecting chromosome structure or number, impacting many genes and often having more severe phenotypic consequences.

71. **Write a note on regulatory sequences in prokaryotic genomes.**
    *   Prokaryotic genomes contain various regulatory sequences that control gene expression, primarily at the transcriptional level. These sequences are typically located in the non-coding regions adjacent to the genes they regulate.
    *   **Promoter:** A DNA sequence upstream of a gene (or operon) that serves as the binding site for RNA polymerase. It contains specific consensus sequences (e.g., the -10 Pribnow box and the -35 sequence) that are recognized by the sigma subunit of RNA polymerase, positioning the enzyme correctly for transcription initiation.
    *   **Operator:** A DNA sequence located within or overlapping the promoter region. It is the binding site for repressor proteins. When a repressor binds to the operator, it physically blocks RNA polymerase from transcribing the downstream genes, thereby switching off gene expression.
    *   **Terminator:** A DNA sequence at the end of a gene or operon that signals the RNA polymerase to stop transcription and release the newly synthesized RNA molecule.
    *   **Ribosome Binding Site (Shine-Dalgarno sequence):** A sequence in the mRNA (not DNA) upstream of the start codon that helps recruit the ribosome for translation initiation in prokaryotes. While not a DNA regulatory sequence itself, its presence in the mRNA is determined by the DNA sequence.
    *   These sequences, often organized into operons, allow prokaryotes to rapidly and efficiently respond to environmental changes by regulating the expression of genes involved in metabolic pathways.

72. **Describe how DNA methylation affects gene expression.**
    *   **DNA methylation** is an epigenetic mechanism that plays a crucial role in gene regulation, primarily by repressing gene expression. It involves the addition of a methyl group (CH₃) to the cytosine base, typically at CpG dinucleotides (a cytosine followed by a guanine nucleotide) in the DNA sequence.
    *   **Mechanism of Gene Repression:**
        1.  **Direct Interference:** Methylation in promoter regions can directly interfere with the binding of transcription factors and RNA polymerase, thereby preventing transcription initiation.
        2.  **Chromatin Remodeling:** Methylated DNA can recruit specific proteins called methyl-CpG-binding proteins (MBPs). These MBPs, in turn, recruit histone deacetylases (HDACs) and other chromatin remodeling complexes. HDACs remove acetyl groups from histones, leading to a more condensed chromatin structure (heterochromatin), which makes the DNA less accessible to transcription machinery and thus silences gene expression.
    *   **Biological Significance:** DNA methylation is involved in:
        *   **Gene Silencing:** Stable silencing of genes, particularly in development and differentiation.
        *   **X-chromosome Inactivation:** In female mammals, one of the two X chromosomes is largely inactivated through extensive methylation.
        *   **Genomic Imprinting:** Differential expression of genes depending on whether they are inherited from the mother or father.
        *   **Suppression of Transposable Elements:** Methylation helps to silence parasitic DNA sequences (transposons) to maintain genome stability.
    *   Changes in DNA methylation patterns are also implicated in various diseases, including cancer.

73. **Explain the molecular mechanism of antisense RNA technology.**
    *   **Antisense RNA technology** is a gene regulation strategy that uses a single-stranded RNA molecule (the "antisense RNA") that is complementary to a target messenger RNA (mRNA) molecule. The binding of the antisense RNA to the target mRNA interferes with gene expression, typically by blocking translation or promoting mRNA degradation.
    *   **Molecular Mechanism:**
        1.  **Complementary Binding:** An engineered antisense RNA molecule is introduced into a cell. This antisense RNA has a sequence that is precisely complementary to a specific region of a target mRNA (e.g., the start codon region, a splice site, or a coding region).
        2.  **Formation of Double-Stranded RNA:** The antisense RNA binds to its target mRNA through complementary base pairing, forming a double-stranded RNA (dsRNA) hybrid.
        3.  **Interference with Gene Expression:** The formation of this dsRNA hybrid can interfere with gene expression in several ways:
            *   **Blocking Translation:** The dsRNA structure can physically block the ribosome from binding to the mRNA or moving along it, thereby preventing translation into protein.
            *   **mRNA Degradation:** The dsRNA can be recognized by cellular machinery (e.g., RNase H or components of the RNA interference pathway like Dicer and RISC), leading to the degradation of the target mRNA. This effectively removes the template for protein synthesis.
            *   **Blocking Splicing:** If the antisense RNA targets a splice site, it can prevent proper splicing of the pre-mRNA, leading to a non-functional mRNA.
    *   **Applications:** Antisense technology has potential therapeutic applications for diseases caused by overexpression of a gene or by the production of a harmful protein, such as viral infections (e.g., HIV) and certain cancers. It is also a valuable research tool for studying gene function.

74. **Discuss the importance of coding vs. non-coding DNA.**
    *   The human genome, and indeed most eukaryotic genomes, consists of both coding and non-coding DNA sequences, both of which are important for cellular function and organismal complexity.
    *   **Coding DNA (Exons):**
        *   **Definition:** These are the sequences within genes that are transcribed into mRNA and subsequently translated into proteins. They contain the instructions for building the functional proteins that carry out most cellular processes.
        *   **Importance:** Essential for producing all the proteins required for life, including enzymes, structural components, transport proteins, signaling molecules, etc. Mutations in coding regions often have direct and significant impacts on protein function and phenotype.
    *   **Non-coding DNA:**
        *   **Definition:** These are DNA sequences that do not code for proteins. Historically, much of this was considered "junk DNA," but its importance is increasingly recognized. It includes introns, regulatory sequences, repetitive DNA, and genes for non-coding RNAs.
        *   **Importance:**
            *   **Gene Regulation:** Regulatory sequences (promoters, enhancers, silencers, insulators) control when, where, and how much a gene is expressed. They are crucial for cell differentiation, development, and responses to environmental stimuli.
            *   **Non-coding RNAs (ncRNAs):** Many non-coding DNA regions are transcribed into functional RNA molecules that are not translated into protein. These include:
                *   **tRNA and rRNA:** Essential for protein synthesis.
                *   **snRNA:** Involved in mRNA splicing.
                *   **miRNA and siRNA:** Play critical roles in gene silencing (RNA interference).
                *   **lncRNA (long non-coding RNA):** Involved in various regulatory processes, including chromatin remodeling and gene expression.
            *   **Chromatin Structure and Genome Organization:** Repetitive DNA sequences (e.g., centromeres, telomeres) are important for chromosome stability, segregation during cell division, and overall genome organization.
            *   **Evolutionary Significance:** Non-coding regions can serve as reservoirs for genetic variation and provide raw material for evolutionary innovation.
    *   In summary, while coding DNA provides the blueprints for proteins, non-coding DNA provides the crucial regulatory framework and other functional RNA molecules that orchestrate gene expression and maintain genome integrity, contributing significantly to the complexity of higher organisms.

75. **Describe three common applications of genetic engineering.**
    *   **Genetic engineering** involves the direct manipulation of an organism's genes using biotechnology.
    *   1.  **Production of Recombinant Proteins (Biopharmaceuticals):**
        *   **Application:** Genetically engineered microorganisms (like bacteria or yeast) or cell lines are used as "factories" to produce valuable human proteins.
        *   **Example:** The production of human insulin for diabetes treatment. The gene for human insulin is inserted into bacteria, which then synthesize large quantities of functional human insulin, replacing the need for animal-derived insulin. Other examples include human growth hormone, clotting factors, and vaccines.
    *   2.  **Development of Genetically Modified Organisms (GMOs) in Agriculture:**
        *   **Application:** Introducing genes into crops or livestock to enhance desirable traits, such as increased yield, pest resistance, herbicide tolerance, or improved nutritional value.
        *   **Example:** "Bt corn" contains a gene from the bacterium *Bacillus thuringiensis* (Bt) that produces a protein toxic to certain insect pests, reducing the need for chemical pesticides. "Golden Rice" is engineered to produce beta-carotene (a precursor to Vitamin A) to combat Vitamin A deficiency.
    *   3.  **Gene Therapy:**
        *   **Application:** Introducing functional genes into a patient's cells to treat or cure genetic diseases caused by defective or missing genes.
        *   **Example:** Treating Severe Combined Immunodeficiency (SCID) by introducing a functional gene for adenosine deaminase (ADA) into the patient's immune cells, allowing them to develop a normal immune system. (See Q59 for more details).

76. **Write about any three medical applications of recombinant DNA technology.**
    *   **Recombinant DNA technology** involves combining DNA from different sources to create new genetic combinations, with numerous applications in medicine:
    *   1.  **Production of Therapeutic Proteins (Biopharmaceuticals):**
        *   **Application:** Using genetically engineered microorganisms or cell cultures to produce large quantities of human proteins that can be used as drugs.
        *   **Examples:**
            *   **Human Insulin:** Produced in *E. coli* to treat diabetes.
            *   **Human Growth Hormone:** Used to treat growth deficiencies.
            *   **Erythropoietin (EPO):** Used to treat anemia.
            *   **Clotting Factors (e.g., Factor VIII):** Used to treat hemophilia.
    *   2.  **Vaccine Development:**
        *   **Application:** Creating safer and more effective vaccines by producing specific viral or bacterial antigens using recombinant DNA techniques, rather than using attenuated or killed pathogens.
        *   **Example:** The Hepatitis B vaccine is produced by inserting the gene for the Hepatitis B surface antigen into yeast cells, which then produce the antigen. This antigen is purified and used as the vaccine.
    *   3.  **Gene Therapy:**
        *   **Application:** Introducing functional genes into a patient's cells to correct genetic defects or provide new therapeutic functions.
        *   **Example:** Treating certain forms of inherited blindness (e.g., Leber Congenital Amaurosis) by delivering a functional gene to retinal cells using a viral vector. (See Q59 for more details).

77. **Explain the role of ESTs and sequence annotation in HGP.**
    *   **Human Genome Project (HGP)** aimed to sequence the entire human genome.
    *   **Expressed Sequence Tags (ESTs):**
        *   **Role:** ESTs are short, single-pass sequences obtained from the ends of cDNA clones. Since cDNA is synthesized from mRNA, ESTs represent sequences that are actively transcribed (expressed) in a particular tissue or at a particular developmental stage.
        *   **Significance in HGP:** ESTs were crucial for rapidly identifying protein-coding genes within the vast stretches of genomic DNA. By sequencing ESTs, researchers could quickly pinpoint regions of the genome that were likely to contain genes, helping to focus sequencing efforts and providing a preliminary catalog of human genes. They also helped in understanding gene expression patterns.
    *   **Sequence Annotation:**
        *   **Role:** After the raw DNA sequence of the human genome was determined, sequence annotation was the process of identifying and assigning biological information to different regions of the genome. This includes locating genes, identifying regulatory sequences, repetitive elements, and determining the function of genes and their products.
        *   **Significance in HGP:** Annotation transformed the raw sequence data into a biologically meaningful resource. It involved using computational tools and algorithms to:
            *   Predict gene locations (coding regions, introns, exons).
            *   Identify promoters, enhancers, and other regulatory elements.
            *   Assign putative functions to genes based on homology to known genes in other organisms.
            *   Identify non-coding RNA genes.
        *   Annotation made the human genome sequence interpretable and usable for biomedical research, allowing scientists to understand the genetic basis of diseases and develop new therapies.

78. **Discuss the ethical, legal, and social issues of genetic information.**
    *   The increasing availability of genetic information, largely driven by projects like the Human Genome Project and advancements in sequencing technologies, has raised significant ethical, legal, and social issues (ELSI).
    *   **Ethical Issues:**
        *   **Privacy and Confidentiality:** Who owns genetic information? How should it be stored, accessed, and protected from unauthorized use? The potential for re-identification from anonymized data is a concern.
        *   **Informed Consent:** Ensuring individuals fully understand the implications of genetic testing and research, including potential risks and benefits, before providing consent.
        *   **Reproductive Choices:** Genetic testing can lead to difficult decisions regarding family planning, prenatal diagnosis, and selective abortion.
        *   **Genetic Determinism:** The risk of oversimplifying complex traits and behaviors as solely determined by genes, potentially leading to prejudice or discrimination.
    *   **Legal Issues:**
        *   **Discrimination:** Laws are needed to prevent discrimination based on genetic information in employment (e.g., Genetic Information Nondiscrimination Act - GINA in the US) and insurance.
        *   **Forensic Use:** The use of DNA databases in criminal investigations raises questions about civil liberties, privacy, and the potential for wrongful convictions.
        *   **Ownership of Genetic Material:** Legal frameworks for intellectual property rights related to genes, gene patents, and genetic technologies.
    *   **Social Issues:**
        *   **Stigmatization:** Individuals with genetic predispositions to certain diseases might face social stigma or psychological burden.
        *   **Equity and Access:** Ensuring equitable access to genetic technologies and therapies, avoiding a "genetically privileged" class.
        *   **Societal Impact:** How will genetic information change our understanding of identity, family, and responsibility? The potential for "designer babies" or enhancement technologies raises profound societal questions.
        *   **Misinterpretation:** The public's understanding of genetic risk and probability can be limited, leading to anxiety or false assurances.

79. **Explain how transcription and translation are coupled in prokaryotes.**
    *   In prokaryotic cells (like bacteria), transcription and translation are said to be **coupled**, meaning they occur simultaneously and in the same cellular compartment (the cytoplasm). This is a key difference from eukaryotes, where transcription occurs in the nucleus and translation in the cytoplasm.
    *   **Mechanism of Coupling:**
        1.  As RNA polymerase transcribes a gene, the newly synthesized messenger RNA (mRNA) molecule begins to emerge from the RNA polymerase enzyme.
        2.  Even before transcription of the entire gene is complete, ribosomes can bind to the 5' end of the nascent mRNA.
        3.  Translation then begins on this partially synthesized mRNA. As the RNA polymerase continues to move along the DNA template, elongating the mRNA, ribosomes follow closely behind, translating the mRNA into protein.
        4.  Multiple ribosomes can attach to a single mRNA molecule, forming a **polysome** (or polyribosome), further increasing the efficiency of protein synthesis.
    *   This coupling allows for a very rapid response to environmental changes, as proteins can be synthesized almost immediately after their genes are transcribed, enabling quick adaptation.

80. **Describe the experimental method used to prove that DNA replicates semi-conservatively.**
    *   See answer to Question 6 (Meselson and Stahl's Experiment).

81. **What is the molecular structure of a nucleosome? Draw and label.**
    *   See answer to Question 10. (Cannot draw, but the description is provided).

82. **How do ribosomes facilitate polypeptide elongation?**
    *   Ribosomes facilitate polypeptide elongation, the stage of translation where amino acids are sequentially added to the growing polypeptide chain, through a cyclical process involving their three tRNA binding sites (A, P, E) and catalytic activity:
        1.  **Aminoacyl-tRNA Entry (A-site):** An incoming aminoacyl-tRNA (tRNA carrying the next amino acid) binds to the A-site of the ribosome. Its anticodon base-pairs with the complementary mRNA codon.
        2.  **Peptide Bond Formation:** The peptidyl transferase activity (catalyzed by rRNA in the large ribosomal subunit) forms a peptide bond between the amino acid in the A-site and the growing polypeptide chain attached to the tRNA in the P-site. This transfers the polypeptide chain to the A-site tRNA.
        3.  **Translocation:** The ribosome then moves exactly one codon along the mRNA in the 5'→3' direction. This movement (translocation) shifts the tRNAs:
            *   The tRNA now carrying the polypeptide chain moves from the A-site to the P-site.
            *   The deacylated (empty) tRNA moves from the P-site to the E-site (exit site).
        4.  **tRNA Release:** The empty tRNA in the E-site is released from the ribosome, ready to be recharged with another amino acid.
        5.  **Cycle Repetition:** The A-site is now empty, and the cycle repeats with the binding of the next aminoacyl-tRNA, continuing until a stop codon is reached.

83. **Describe three differences between prokaryotic and eukaryotic translation.**
    *   1.  **Coupling with Transcription:**
        *   **Prokaryotes:** Transcription and translation are coupled; they occur simultaneously in the cytoplasm. Ribosomes can begin translating an mRNA molecule even before its transcription is complete.
        *   **Eukaryotes:** Transcription occurs in the nucleus, and translation occurs in the cytoplasm. These processes are spatially and temporally separated. mRNA must be fully processed and exported from the nucleus before translation can begin.
    *   2.  **Ribosome Size and Structure:**
        *   **Prokaryotes:** Have smaller 70S ribosomes, composed of 50S and 30S subunits.
        *   **Eukaryotes:** Have larger 80S ribosomes, composed of 60S and 40S subunits.
    *   3.  **Initiation of Translation:**
        *   **Prokaryotes:** Ribosomes bind to a specific sequence on the mRNA called the Shine-Dalgarno sequence (ribosome binding site), which is located upstream of the start codon (AUG). The initiator tRNA carries formylmethionine (fMet).
        *   **Eukaryotes:** Ribosomes typically bind to the 5' cap of the mRNA and then scan downstream until they encounter the first AUG start codon (Kozak sequence often surrounds it). The initiator tRNA carries unformylated methionine.

84. **Explain how alternative splicing leads to protein diversity.**
    *   **Alternative splicing** is a crucial post-transcriptional regulatory mechanism in eukaryotes that allows a single gene to produce multiple different protein isoforms.
    *   **Mechanism:** In genes containing multiple exons and introns, alternative splicing involves the selective inclusion or exclusion of certain exons during the splicing process. Instead of always joining all exons in a fixed order, the spliceosome can skip certain exons, or include mutually exclusive exons, or use alternative 5' or 3' splice sites.
    *   **Resulting Diversity:** Each unique combination of exons results in a distinct messenger RNA (mRNA) molecule. Since each mRNA molecule is then translated into a protein, alternative splicing effectively generates a diverse set of proteins (isoforms) from a single gene. These different protein isoforms can have varied functions, cellular locations, binding affinities, or enzymatic activities.
    *   **Significance:** Alternative splicing significantly expands the protein-coding capacity of a genome without increasing the number of genes. It is a major contributor to the complexity and functional diversity of eukaryotic organisms, playing roles in cell differentiation, development, and tissue-specific functions. For example, a single gene might produce different protein isoforms in different cell types or at different developmental stages.

85. **Describe the molecular mechanism of action of restriction enzymes.**
    *   **Restriction enzymes** (also called restriction endonucleases) are bacterial enzymes that play a crucial role in molecular biology and genetic engineering. They act as "molecular scissors" by recognizing and cutting DNA at specific nucleotide sequences.
    *   **Molecular Mechanism:**
        1.  **Recognition Site:** Each restriction enzyme recognizes a specific, short DNA sequence, typically 4 to 8 base pairs long. These recognition sites are often **palindromic**, meaning they read the same forwards and backwards on opposite strands (e.g., GAATTC).
        2.  **Binding:** The restriction enzyme binds to the DNA molecule at its specific recognition site.
        3.  **Cleavage:** The enzyme then catalyzes the hydrolysis (breaking) of the phosphodiester bonds within the DNA backbone at precise locations within or near the recognition site.
        4.  **Types of Cuts:**
            *   **Sticky Ends (Cohesive Ends):** Many restriction enzymes make staggered cuts, leaving short, single-stranded overhangs at the cut ends. These "sticky ends" are complementary to each other and can readily base-pair with other DNA fragments cut by the same enzyme, facilitating the joining of DNA from different sources. (e.g., EcoRI cuts to leave AATT overhangs).
            *   **Blunt Ends:** Some restriction enzymes cut straight across both DNA strands, leaving no overhangs. These "blunt ends" can be joined to any other blunt-ended DNA fragment, though less efficiently than sticky ends. (e.g., SmaI cuts to leave no overhangs).
    *   **Biological Role:** In bacteria, restriction enzymes serve as a defense mechanism against invading bacteriophages (viruses) by cutting up foreign DNA. Bacteria protect their own DNA from being cut by modifying their recognition sites (e.g., by methylation).

86. **Explain how genes are mapped using sequencing techniques.**
    *   Gene mapping, the process of determining the relative locations of genes on a chromosome, has been revolutionized by DNA sequencing techniques.
    *   **Early Mapping (Linkage Mapping):** Historically, genes were mapped based on recombination frequencies (how often genes are inherited together).
    *   **Modern Mapping (Physical Mapping and Sequencing):**
        1.  **Genome Sequencing:** The entire genome of an organism is sequenced, generating a vast amount of raw DNA sequence data.
        2.  **Sequence Assembly:** Bioinformatics tools are used to assemble these short sequence reads into longer contiguous sequences (contigs) and then into complete chromosomes.
        3.  **Gene Prediction (Annotation):** Computational algorithms are applied to the assembled sequence to identify potential gene-coding regions (exons, introns, promoters, terminators). This involves looking for open reading frames (ORFs), splice sites, and comparing sequences to known genes in databases.
        4.  **Marker Identification:** Specific DNA markers (e.g., SNPs - Single Nucleotide Polymorphisms, STRs - Short Tandem Repeats) are identified across the genome. These markers serve as landmarks.
        5.  **Linkage to Phenotypes:** In genetic studies, these sequence-based markers are correlated with specific traits or diseases (phenotypes) in populations or families. If a marker consistently co-segregates with a trait, it indicates that the gene responsible for that trait is located near that marker.
        6.  **Functional Annotation:** Once a gene is identified and its location is known, its function can be inferred by comparing its sequence to genes with known functions in other organisms (homology searches) and by experimental validation.
    *   By combining high-throughput sequencing with computational analysis and genetic studies, researchers can precisely map the location of genes on chromosomes and understand their relationship to traits and diseases.

87. **Describe any three key bioinformatics databases used in genomics.**
    *   Bioinformatics databases are essential repositories for storing, organizing, and making biological data accessible for research.
    *   1.  **GenBank (NCBI Nucleotide Database):**
        *   **Description:** A comprehensive, publicly available database of all publicly available nucleotide sequences (DNA and RNA) and their associated protein translations. It is maintained by the National Center for Biotechnology Information (NCBI) in the USA.
        *   **Use:** Researchers submit their sequencing data to GenBank. It's used for sequence retrieval, homology searches (e.g., using BLAST), gene identification, and comparative genomics.
    *   2.  **UniProt (Universal Protein Resource):**
        *   **Description:** A central, comprehensive, and high-quality resource for protein sequence and annotation data. It consists of two main sections: UniProtKB/Swiss-Prot (manually annotated and reviewed) and UniProtKB/TrEMBL (automatically annotated).
        *   **Use:** Provides detailed information about protein function, domains, post-translational modifications, disease associations, and evolutionary relationships. Crucial for proteomics and functional genomics.
    *   3.  **PDB (Protein Data Bank):**
        *   **Description:** A worldwide repository for the 3D structural data of large biological molecules, such as proteins and nucleic acids, determined by methods like X-ray crystallography, NMR spectroscopy, and cryo-electron microscopy.
        *   **Use:** Essential for structural biology, drug design, and understanding protein function based on its three-dimensional shape. Researchers can visualize and analyze the structures of macromolecules.

88. **Compare the characteristics of mitochondrial and nuclear DNA.**
    | Feature | Mitochondrial DNA (mtDNA) | Nuclear DNA (nDNA) |
    |---|---|---|
    | **Location** | Mitochondria (cytoplasm) | Nucleus |
    | **Structure** | Circular, double-stranded | Linear, double-stranded |
    | **Size** | Much smaller (e.g., ~16.5 kb in humans) | Much larger (e.g., ~3.2 billion bp in humans) |
    | **Number of Copies** | Multiple copies per mitochondrion; hundreds to thousands per cell | Two copies per diploid cell (one from each parent) |
    | **Inheritance** | Maternal (inherited almost exclusively from the mother) | Biparental (inherited equally from both parents) |
    | **Genes** | Codes for a small number of genes (e.g., rRNAs, tRNAs, some proteins for oxidative phosphorylation) | Codes for the vast majority of genes in the organism |
    | **Introns** | Generally lacks introns | Contains numerous introns |
    | **Histones** | Not associated with histones | Associated with histones to form chromatin |
    | **Replication** | Replicates independently of the cell cycle | Replicates during the S phase of the cell cycle |
    | **Mutation Rate** | Higher mutation rate | Lower mutation rate (due to repair mechanisms) |

89. **Explain how transcription is terminated in prokaryotes.**
    *   In prokaryotes, transcription termination occurs when RNA polymerase encounters a specific signal in the DNA template, causing it to stop synthesizing RNA and release the nascent RNA transcript. There are two main mechanisms:
    *   1.  **Rho-Independent (Intrinsic) Termination:**
        *   **Mechanism:** This mechanism relies solely on specific sequences in the DNA template. The transcribed RNA forms a **hairpin loop** structure (a stem-loop) followed by a stretch of several uridine (U) nucleotides. The hairpin loop forms due to complementary base pairing within the RNA molecule.
        *   **Process:** The formation of the stable hairpin loop causes RNA polymerase to pause. The weak A-U base pairs between the RNA and DNA template in the U-rich region downstream of the hairpin are then unable to hold the RNA-DNA hybrid together. This leads to the dissociation of the RNA transcript from the DNA template and the release of RNA polymerase.
    *   2.  **Rho-Dependent Termination:**
        *   **Mechanism:** This mechanism requires the involvement of a protein called the **Rho (ρ) factor**. Rho is an ATP-dependent helicase that binds to a specific Rho utilization (rut) site on the nascent RNA transcript.
        *   **Process:** After binding to the rut site, Rho moves along the RNA transcript towards the RNA polymerase. When RNA polymerase pauses at a Rho-dependent termination sequence, Rho catches up to the polymerase. Rho then unwinds the RNA-DNA hybrid within the transcription bubble, causing the release of the RNA transcript and the dissociation of RNA polymerase from the DNA.

90. **Describe the method of creating a cDNA library.**
    *   A **cDNA (complementary DNA) library** is a collection of cloned DNA sequences that represent all the messenger RNA (mRNA) molecules expressed in a particular cell type or tissue at a specific time. Unlike genomic DNA libraries, cDNA libraries only contain coding sequences (exons) because they are derived from mRNA, which has already undergone splicing.
    *   **Method (Key Steps):**
        1.  **mRNA Isolation:** Total RNA is extracted from the desired cells or tissue. Then, messenger RNA (mRNA) is isolated from the total RNA, typically using oligo-dT cellulose chromatography, which binds to the poly-A tail present on most eukaryotic mRNAs.
        2.  **First-Strand cDNA Synthesis:** Oligo-dT primers (short sequences of thymine nucleotides) are annealed to the poly-A tail of the mRNA. The enzyme **reverse transcriptase** then uses the mRNA as a template and the oligo-dT primer to synthesize a complementary DNA (cDNA) strand. This results in an mRNA-cDNA hybrid.
        3.  **mRNA Degradation:** The mRNA strand in the mRNA-cDNA hybrid is removed. This can be done by alkaline hydrolysis or by using RNase H.
        4.  **Second-Strand cDNA Synthesis:** The first cDNA strand serves as a template for the synthesis of the second DNA strand. This can be initiated by forming a hairpin loop at the 3' end of the first cDNA strand, which acts as a primer, or by adding a new primer. DNA polymerase then synthesizes the second strand, creating a double-stranded cDNA molecule.
        5.  **Ligation into Vector:** The double-stranded cDNA molecules are then ligated (inserted) into a suitable cloning vector (e.g., a plasmid or bacteriophage vector) that has been cut with restriction enzymes. Linkers or adaptors may be added to the cDNA ends to facilitate ligation.
        6.  **Transformation/Transfection:** The recombinant vectors (containing the cDNA inserts) are introduced into host cells (e.g., *E. coli* bacteria).
        7.  **Selection and Amplification:** The host cells are grown on selective media to identify those that have taken up the recombinant vector. As the host cells multiply, they replicate the vector and the inserted cDNA, creating a library of cloned cDNA fragments.

91. **Explain the concept of split genes and its discovery.**
    *   **Split Genes (Interrupted Genes):** The concept of split genes refers to the discovery that in eukaryotic organisms, many genes are not continuous sequences of coding DNA. Instead, they are "split" into coding segments called **exons** (expressed regions) and non-coding intervening segments called **introns** (intervening regions). Both exons and introns are transcribed into a primary RNA transcript (pre-mRNA or hnRNA), but the introns are subsequently removed through a process called RNA splicing, leaving only the exons to be joined together to form the mature mRNA.
    *   **Discovery:** The discovery of split genes was a groundbreaking finding in molecular biology, challenging the earlier assumption (based on prokaryotic gene structure) that genes were continuous.
        *   It was first made in **1977** by independent research groups led by **Richard Roberts** and **Phillip Sharp**.
        *   They were studying the genes of **adenoviruses** (DNA viruses that infect eukaryotic cells).
        *   Using **electron microscopy** and **hybridization techniques**, they observed that when mature viral mRNA was hybridized to its corresponding viral DNA, loops of DNA were formed that did not hybridize to the mRNA. These loops corresponded to the introns, which were present in the DNA but absent from the mature mRNA.
        *   This direct visual evidence demonstrated that the coding sequences (exons) in the DNA were interrupted by non-coding sequences (introns) that were removed during RNA processing.
    *   This discovery fundamentally changed our understanding of gene structure and expression in eukaryotes and led to the concept of RNA splicing.

92. **Describe how polymerase chain reaction (PCR) works.**
    *   **Polymerase Chain Reaction (PCR)** is a powerful molecular biology technique used to amplify (make many copies of) a specific segment of DNA *in vitro* (in a test tube). It can generate millions to billions of copies of a target DNA sequence from a very small initial amount.
    *   **Key Components:**
        *   **DNA Template:** The DNA containing the target sequence to be amplified.
        *   **Primers:** Two short, single-stranded DNA oligonucleotides (typically 18-25 bases long) that are complementary to the sequences flanking the target region. One primer binds to the forward strand, and the other to the reverse strand.
        *   **Taq Polymerase:** A heat-stable DNA polymerase (originally isolated from *Thermus aquaticus* bacterium) that can withstand the high temperatures required for denaturation.
        *   **dNTPs (Deoxynucleotide Triphosphates):** The building blocks (A, T, C, G) for synthesizing new DNA strands.
        *   **Buffer:** Provides optimal conditions for the reaction.
    *   **Procedure (Repeated Cycles):** PCR involves a series of 20-40 repeated cycles, each typically consisting of three temperature-dependent steps:
        1.  **Denaturation (94-98°C):** The reaction mixture is heated to a high temperature to break the hydrogen bonds between the two strands of the DNA template, separating them into single strands.
        2.  **Annealing (50-65°C):** The temperature is lowered, allowing the forward and reverse primers to anneal (bind) to their complementary sequences on the single-stranded DNA templates. The annealing temperature is critical for primer specificity.
        3.  **Extension (72°C):** The temperature is raised to the optimal temperature for Taq polymerase. Taq polymerase then synthesizes new DNA strands by adding dNTPs to the 3' ends of the primers, extending them along the template strands.
    *   **Amplification:** Each cycle doubles the number of DNA copies. After 30-40 cycles, millions to billions of copies of the target DNA sequence are produced exponentially. The newly synthesized DNA strands from one cycle serve as templates for the next, leading to exponential amplification.

93. **Discuss the applications and importance of PCR in biology.**
    *   **Polymerase Chain Reaction (PCR)** has revolutionized molecular biology and has a vast array of applications due to its ability to rapidly amplify specific DNA sequences from minute amounts of starting material.
    *   **Applications:**
        1.  **DNA Fingerprinting/Forensics:** Amplifying DNA from tiny samples (e.g., a single hair, a drop of blood) found at crime scenes for individual identification, paternity testing, and victim identification.
        2.  **Medical Diagnostics:**
            *   **Detection of Pathogens:** Rapidly identifying viral (e.g., HIV, COVID-19, Hepatitis B/C) or bacterial (e.g., *Mycobacterium tuberculosis*) infections by amplifying pathogen-specific DNA/RNA.
            *   **Genetic Disease Diagnosis:** Detecting mutations or genetic predispositions to diseases (e.g., cystic fibrosis, Huntington's disease) in patient samples.
            *   **Cancer Detection:** Identifying cancer-specific mutations or monitoring minimal residual disease.
        3.  **Gene Cloning and Genetic Engineering:** Amplifying specific genes for insertion into vectors for cloning, sequencing, or genetic modification.
        4.  **Evolutionary Biology and Phylogenetics:** Amplifying DNA from ancient samples (e.g., fossil remains, museum specimens) to study evolutionary relationships and population genetics.
        5.  **Research:** Used in countless research applications, including gene expression analysis (RT-PCR), mutagenesis, and DNA sequencing preparation.
    *   **Importance:**
        *   **Sensitivity:** Can amplify DNA from extremely small or degraded samples.
        *   **Speed:** Provides results much faster than traditional cloning methods.
        *   **Specificity:** Primers ensure that only the target DNA sequence is amplified.
        *   **Versatility:** Adaptable to a wide range of applications across various fields of biology and medicine.
        *   **Cost-Effectiveness:** Relatively inexpensive compared to other DNA manipulation techniques.

94. **Describe the entire mechanism of lac operon regulation with diagrams.**
    *   See answer to Question 20. (Cannot provide diagrams, but the detailed mechanism is described).

95. **How are DNA-binding motifs important for gene regulation?**
    *   **DNA-binding motifs** are specific structural patterns or domains within proteins that enable them to recognize and bind to particular DNA sequences. These motifs are crucial for gene regulation because they allow regulatory proteins (like transcription factors, repressors, and activators) to precisely interact with DNA and control gene expression.
    *   **Importance:**
        1.  **Specificity:** Different DNA-binding motifs recognize different DNA sequences. This specificity ensures that a particular regulatory protein binds only to its target genes, allowing for precise control over which genes are turned on or off. Common motifs include helix-turn-helix, zinc fingers, and leucine zippers.
        2.  **Gene Activation/Repression:** By binding to specific regulatory sequences (e.g., promoters, operators, enhancers), these proteins can either:
            *   **Activate Transcription:** Recruit RNA polymerase or other components of the transcription machinery.
            *   **Repress Transcription:** Block RNA polymerase binding or movement.
        3.  **Integration of Signals:** Many regulatory proteins contain multiple domains, including DNA-binding domains and domains that interact with other proteins or signaling molecules. This allows them to integrate various cellular signals and fine-tune gene expression in response to internal and external cues.
        4.  **Chromatin Remodeling:** Some DNA-binding proteins recruit chromatin remodeling complexes or histone-modifying enzymes, thereby altering chromatin structure and gene accessibility.
    *   In essence, DNA-binding motifs are the molecular "keys" that allow regulatory proteins to "read" the DNA sequence and implement the genetic program of the cell.

96. **What are pseudogenes? How do they arise?**
    *   **Pseudogenes:** Pseudogenes are DNA sequences that resemble functional genes but have lost their protein-coding ability due to mutations. They are often referred to as "fossil genes" or "dead genes" because they are non-functional remnants of once-active genes.
    *   **How they arise:** There are two main mechanisms by which pseudogenes are thought to arise:
        1.  **Non-processed (Duplicated) Pseudogenes:**
            *   **Mechanism:** These arise from gene duplication events. During evolution, an existing functional gene is duplicated, creating an extra copy. Over time, one of the copies accumulates mutations (e.g., frameshift mutations, premature stop codons, deletions) that render it non-functional, while the original gene remains active.
            *   **Characteristics:** They typically retain the original gene's exon-intron structure and regulatory sequences, as they are direct copies of genomic DNA.
        2.  **Processed (Retrotransposed) Pseudogenes:**
            *   **Mechanism:** These arise from the reverse transcription of an mRNA molecule back into DNA, followed by its re-insertion into the genome. This process is mediated by reverse transcriptase (often from retrotransposons).
            *   **Characteristics:** Since they are derived from mRNA, they lack introns and often lack the original gene's promoter and other regulatory sequences (unless they are inserted near an active promoter). They frequently have a poly-A tail at their 3' end, reflecting their mRNA origin. They are usually non-functional because they lack proper regulatory elements for transcription.
    *   **Significance:** While generally non-functional, pseudogenes can sometimes acquire new functions or play regulatory roles (e.g., by producing non-coding RNAs). They are also valuable for evolutionary studies, providing insights into gene duplication, genome evolution, and phylogenetic relationships.

97. **Explain the differences in DNA packaging between prokaryotes and eukaryotes.**
    | Feature | Prokaryotic DNA Packaging | Eukaryotic DNA Packaging |
    |---|---|---|
    | **Location** | Nucleoid region (cytoplasm) | Nucleus (and mitochondria/chloroplasts) |
    | **DNA Structure** | Circular, usually a single chromosome | Linear, multiple chromosomes |
    | **Associated Proteins** | **No histones**. Associated with histone-like proteins (e.g., HU, H-NS) that help compact the DNA, but these are not true histones. | **Associated with histones**. DNA wraps around histone proteins to form nucleosomes. |
    | **Basic Unit of Packaging** | Supercoiled loops, often anchored to a protein scaffold. | **Nucleosome** (DNA wrapped around a histone octamer). |
    | **Levels of Compaction** | Primarily supercoiling and looping. | Multiple levels: nucleosomes → 30-nm fiber → looped domains → metaphase chromosome. |
    | **Complexity** | Simpler, less elaborate packaging. | Highly complex and dynamic packaging, crucial for gene regulation. |
    | **Chromatin** | No true chromatin structure. | Forms **chromatin** (DNA + histones), which can be euchromatin or heterochromatin. |

98. **Write a short note on the ribozyme activity of RNA.**
    *   **Ribozyme Activity of RNA:** A ribozyme is an RNA molecule that possesses catalytic activity, meaning it can act as an enzyme to catalyze specific biochemical reactions. This concept challenged the long-held belief that only proteins could function as enzymes.
    *   **Discovery:** The first ribozymes were discovered in the early 1980s by Sidney Altman (RNase P) and Thomas Cech (Tetrahymena self-splicing RNA), for which they shared the Nobel Prize in Chemistry in 1989.
    *   **Examples and Importance:**
        *   **Ribosomal RNA (rRNA):** The most prominent example is the peptidyl transferase activity of the large ribosomal subunit during protein synthesis. It is the rRNA, not a protein, that catalyzes the formation of peptide bonds between amino acids. This highlights RNA's central role in the fundamental process of life.
        *   **Self-splicing Introns:** Some introns (Group I and Group II) can catalyze their own removal from RNA transcripts without the help of proteins.
        *   **RNase P:** An enzyme involved in tRNA processing, where the RNA component is catalytic.
    *   The discovery of ribozymes provided strong evidence for the "RNA world hypothesis," suggesting that early life forms may have used RNA for both genetic information storage and catalysis before the evolution of DNA and proteins.

99. **Describe how stem-loop structures are formed in RNA.**
    *   **Stem-loop structures** (also known as hairpin loops) are common and important secondary structures found in single-stranded RNA molecules. They are formed when a segment of an RNA molecule folds back on itself, allowing complementary base pairing between nucleotides within the same strand.
    *   **Formation Mechanism:**
        1.  **Complementary Sequences:** An RNA molecule contains two short stretches of nucleotides that are complementary to each other (e.g., 5'-GCGC...GCGC-3').
        2.  **Folding:** The RNA strand folds back on itself, bringing these complementary sequences into close proximity.
        3.  **Base Pairing:** Hydrogen bonds form between the complementary bases (A-U, G-C) in these two stretches, creating a double-stranded "stem" region.
        4.  **Loop Formation:** The nucleotides between the two complementary stretches that do not pair up form an unpaired "loop" region at the end of the stem.
    *   **Examples and Importance:**
        *   **tRNA:** The cloverleaf 2D structure of tRNA is characterized by several stem-loop structures (e.g., D loop, anticodon loop, TψC loop).
        *   **rRNA:** Stem-loops contribute to the complex 3D structure of ribosomal RNA, which is crucial for ribosome function.
        *   **mRNA:** Stem-loops in mRNA can act as regulatory elements, affecting mRNA stability, translation initiation, or serving as termination signals in prokaryotic transcription (rho-independent terminators).
        *   **snRNA:** Involved in the structure and function of snRNPs in splicing.
    *   These structures are critical for the function of many RNA molecules, providing specific shapes that allow them to interact with other molecules (proteins, DNA, other RNAs) and perform their diverse roles in the cell.

100. **Summarize the journey from gene to functional protein in molecular terms.**
    *   The journey from a gene to a functional protein, often referred to as the central dogma of molecular biology, involves a precise flow of genetic information through two main processes: transcription and translation.
    *   **1. Transcription (DNA to RNA):**
        *   The process begins in the nucleus (eukaryotes) or cytoplasm (prokaryotes) with a specific gene on a DNA molecule.
        *   **RNA polymerase** binds to the gene's **promoter** region, unwinds the DNA double helix, and uses one DNA strand as a template.
        *   It synthesizes a complementary **RNA molecule** (pre-mRNA in eukaryotes, mRNA in prokaryotes) by adding ribonucleotides according to base-pairing rules (A with U, T with A, G with C, C with G).
        *   In eukaryotes, the pre-mRNA undergoes **post-transcriptional modifications**: a 5' cap is added, a poly-A tail is added to the 3' end, and **introns** (non-coding regions) are removed by **splicing**, leaving only **exons** (coding regions) joined together to form mature mRNA.
       **2. Translation (RNA to Protein):**
        *   The mature mRNA molecule then moves to the cytoplasm and associates with a **ribosome**.
        *   **Initiation:** The ribosome binds to the mRNA and scans for the **start codon (AUG)**. An initiator **tRNA** (carrying methionine) binds to this codon.
        *   **Elongation:** The ribosome moves along the mRNA, reading codons sequentially. For each codon, a specific tRNA molecule (with a complementary **anticodon**) brings the corresponding amino acid to the ribosome's A-site. The **peptidyl transferase** activity of the ribosome (catalyzed by rRNA) forms a **peptide bond** between the incoming amino acid and the growing polypeptide chain. The ribosome then translocates, moving the tRNAs and mRNA.
        *   **Termination:** When a **stop codon (UAA, UAG, UGA)** is reached, release factors bind, causing the release of the completed polypeptide chain from the ribosome.
        **3. Post-translational Modifications and Folding:**
        *   The newly synthesized polypeptide chain then folds into its specific three-dimensional functional structure, often assisted by chaperone proteins.
        *   It may undergo further **post-translational modifications** (e.g., phosphorylation, glycosylation, cleavage) that are essential for its activity, stability, localization, or interaction with other molecules.
       This intricate molecular pathway ensures that the genetic information encoded in DNA is accurately converted into the diverse array of functional proteins that carry out all cellular processes.
